# This Makefile is for the BioPerl extension to perl.
#
# It was generated automatically by MakeMaker version
# 7.0401 (Revision: 70401) from the contents of
# Makefile.PL. Don't edit this file, edit Makefile.PL instead.
#
#       ANY CHANGES MADE HERE WILL BE LOST!
#
#   MakeMaker ARGV: (q[INSTALL_BASE=/bios-store5/lyt/python_demo_try/SeriesAnalysis/perl5])
#

#   MakeMaker Parameters:

#     ABSTRACT => q[Perl modules for biology]
#     AUTHOR => [q[See individual modules]]
#     BUILD_REQUIRES => {  }
#     CONFIGURE_REQUIRES => { ExtUtils::MakeMaker=>q[0] }
#     DISTNAME => q[BioPerl]
#     EXE_FILES => [q[bin/bp_aacomp], q[bin/bp_bioflat_index], q[bin/bp_biogetseq], q[bin/bp_dbsplit], q[bin/bp_extract_feature_seq], q[bin/bp_fastam9_to_table], q[bin/bp_fetch], q[bin/bp_filter_search], q[bin/bp_find-blast-matches], q[bin/bp_gccalc], q[bin/bp_genbank2gff3], q[bin/bp_index], q[bin/bp_local_taxonomydb_query], q[bin/bp_make_mrna_protein], q[bin/bp_mask_by_search], q[bin/bp_mrtrans], q[bin/bp_mutate], q[bin/bp_nexus2nh], q[bin/bp_nrdb], q[bin/bp_oligo_count], q[bin/bp_process_gadfly], q[bin/bp_process_sgd], q[bin/bp_revtrans-motif], q[bin/bp_search2alnblocks], q[bin/bp_search2gff], q[bin/bp_search2table], q[bin/bp_search2tribe], q[bin/bp_seq_length], q[bin/bp_seqconvert], q[bin/bp_seqcut], q[bin/bp_seqpart], q[bin/bp_seqret], q[bin/bp_seqretsplit], q[bin/bp_split_seq], q[bin/bp_sreformat], q[bin/bp_taxid4species], q[bin/bp_taxonomy2tree], q[bin/bp_translate_seq], q[bin/bp_tree2pag], q[bin/bp_unflatten_seq]]
#     LICENSE => q[perl]
#     MIN_PERL_VERSION => q[5.006]
#     NAME => q[BioPerl]
#     PREREQ_PM => { AnyDBM_File=>q[0], Carp=>q[0], Cwd=>q[0], DB_File=>q[0], Data::Dumper=>q[0], Data::Stag=>q[0], Digest::MD5=>q[0], Dumpvalue=>q[0], Error=>q[0], Exporter=>q[0], Fcntl=>q[0], File::Basename=>q[0], File::Copy=>q[0], File::Path=>q[0], File::Spec=>q[0], File::Spec::Functions=>q[0], File::Temp=>q[0], FileHandle=>q[0], FindBin=>q[0], Getopt::Long=>q[0], Graph::Directed=>q[0], HTTP::Request::Common=>q[0], HTTP::Response=>q[0], IO::File=>q[0], IO::Handle=>q[0], IO::Pipe=>q[0], IO::Scalar=>q[0], IO::Socket=>q[0], IO::String=>q[0], IPC::Open3=>q[0], IPC::Run=>q[0], LWP::UserAgent=>q[0], List::MoreUtils=>q[0], List::Util=>q[0], Math::BigFloat=>q[0], Module::Build=>q[0], POSIX=>q[0], Pod::Usage=>q[0], Scalar::Util=>q[0], Set::Scalar=>q[0], Storable=>q[0], Symbol=>q[0], Test::Builder=>q[0], Test::Builder::Module=>q[0], Test::Memory::Cycle=>q[0], Test::More=>q[0], Test::Most=>q[0], Test::RequiresInternet=>q[0], Test::Weaken=>q[0], Text::Balanced=>q[0], Text::Wrap=>q[0], Tie::Handle=>q[0], Tie::RefHash=>q[0], Time::HiRes=>q[0], UNIVERSAL=>q[0], URI=>q[0], XML::DOM=>q[0], XML::LibXML=>q[0], XML::LibXML::Reader=>q[0], XML::Parser::PerlSAX=>q[0], XML::SAX=>q[0], XML::SAX::Base=>q[0], XML::SAX::Writer=>q[0], XML::Twig=>q[0], XML::Writer=>q[0], YAML=>q[0], base=>q[0], constant=>q[0], integer=>q[0], lib=>q[0], overload=>q[0], parent=>q[0], strict=>q[0], utf8=>q[0], vars=>q[0], version=>q[0], warnings=>q[0] }
#     TEST_REQUIRES => { File::Spec=>q[0], FindBin=>q[0], IO::Handle=>q[0], IPC::Open3=>q[0], Test::Memory::Cycle=>q[0], Test::More=>q[0], Test::Weaken=>q[0], lib=>q[0] }
#     VERSION => q[1.7.8]
#     test => { TESTS=>q[t/*.t t/Align/*.t t/AlignIO/*.t t/Annotation/*.t t/LocalDB/*.t t/LocalDB/Index/*.t t/LocalDB/Taxonomy/*.t t/Matrix/*.t t/Matrix/IO/*.t t/Ontology/*.t t/Ontology/IO/*.t t/RemoteDB/*.t t/Root/*.t t/SearchIO/*.t t/SearchIO/Writer/*.t t/Seq/*.t t/SeqFeature/*.t t/SeqIO/*.t t/SeqTools/*.t t/Tools/*.t t/Tools/Alignment/*.t t/Tools/EMBOSS/*.t t/Tools/Phylo/*.t t/Tools/Phylo/Phylip/*.t t/Tools/Signalp/*.t t/Tools/Spidey/*.t t/Tree/*.t t/Tree/TreeIO/*.t] }

# --- MakeMaker post_initialize section:


# --- MakeMaker const_config section:

# These definitions are from config.sh (via /bios-store1/home/yutongli/.conda/envs/try_619_new/lib/perl5/5.22.0/x86_64-linux-thread-multi/Config.pm).
# They may have been overridden via Makefile.PL or on the command line.
AR = ar
CC = cc
CCCDLFLAGS = -fPIC
CCDLFLAGS = -Wl,-E
DLEXT = so
DLSRC = dl_dlopen.xs
EXE_EXT = 
FULL_AR = /opt/rh/devtoolset-2/root/usr/bin/ar
LD = cc
LDDLFLAGS = -shared -O2 -L/usr/local/lib -fstack-protector-strong
LDFLAGS =   -fstack-protector-strong -L/usr/local/lib
LIBC = libc-2.12.so
LIB_EXT = .a
OBJ_EXT = .o
OSNAME = linux
OSVERS = 3.13.0-106-generic
RANLIB = :
SITELIBEXP = /bios-store1/home/yutongli/.conda/envs/try_619_new/lib/perl5/site_perl/5.22.0
SITEARCHEXP = /bios-store1/home/yutongli/.conda/envs/try_619_new/lib/perl5/site_perl/5.22.0/x86_64-linux-thread-multi
SO = so
VENDORARCHEXP = 
VENDORLIBEXP = 


# --- MakeMaker constants section:
AR_STATIC_ARGS = cr
DIRFILESEP = /
DFSEP = $(DIRFILESEP)
NAME = BioPerl
NAME_SYM = BioPerl
VERSION = 1.7.8
VERSION_MACRO = VERSION
VERSION_SYM = 1_7_8
DEFINE_VERSION = -D$(VERSION_MACRO)=\"$(VERSION)\"
XS_VERSION = 1.7.8
XS_VERSION_MACRO = XS_VERSION
XS_DEFINE_VERSION = -D$(XS_VERSION_MACRO)=\"$(XS_VERSION)\"
INST_ARCHLIB = blib/arch
INST_SCRIPT = blib/script
INST_BIN = blib/bin
INST_LIB = blib/lib
INST_MAN1DIR = blib/man1
INST_MAN3DIR = blib/man3
MAN1EXT = 1
MAN3EXT = 3
INSTALLDIRS = site
INSTALL_BASE = /bios-store5/lyt/python_demo_try/SeriesAnalysis/perl5
DESTDIR = 
PREFIX = $(INSTALL_BASE)
INSTALLPRIVLIB = $(INSTALL_BASE)/lib/perl5
DESTINSTALLPRIVLIB = $(DESTDIR)$(INSTALLPRIVLIB)
INSTALLSITELIB = $(INSTALL_BASE)/lib/perl5
DESTINSTALLSITELIB = $(DESTDIR)$(INSTALLSITELIB)
INSTALLVENDORLIB = $(INSTALL_BASE)/lib/perl5
DESTINSTALLVENDORLIB = $(DESTDIR)$(INSTALLVENDORLIB)
INSTALLARCHLIB = $(INSTALL_BASE)/lib/perl5/x86_64-linux-thread-multi
DESTINSTALLARCHLIB = $(DESTDIR)$(INSTALLARCHLIB)
INSTALLSITEARCH = $(INSTALL_BASE)/lib/perl5/x86_64-linux-thread-multi
DESTINSTALLSITEARCH = $(DESTDIR)$(INSTALLSITEARCH)
INSTALLVENDORARCH = $(INSTALL_BASE)/lib/perl5/x86_64-linux-thread-multi
DESTINSTALLVENDORARCH = $(DESTDIR)$(INSTALLVENDORARCH)
INSTALLBIN = $(INSTALL_BASE)/bin
DESTINSTALLBIN = $(DESTDIR)$(INSTALLBIN)
INSTALLSITEBIN = $(INSTALL_BASE)/bin
DESTINSTALLSITEBIN = $(DESTDIR)$(INSTALLSITEBIN)
INSTALLVENDORBIN = $(INSTALL_BASE)/bin
DESTINSTALLVENDORBIN = $(DESTDIR)$(INSTALLVENDORBIN)
INSTALLSCRIPT = $(INSTALL_BASE)/bin
DESTINSTALLSCRIPT = $(DESTDIR)$(INSTALLSCRIPT)
INSTALLSITESCRIPT = $(INSTALL_BASE)/bin
DESTINSTALLSITESCRIPT = $(DESTDIR)$(INSTALLSITESCRIPT)
INSTALLVENDORSCRIPT = $(INSTALL_BASE)/bin
DESTINSTALLVENDORSCRIPT = $(DESTDIR)$(INSTALLVENDORSCRIPT)
INSTALLMAN1DIR = $(INSTALL_BASE)/man/man1
DESTINSTALLMAN1DIR = $(DESTDIR)$(INSTALLMAN1DIR)
INSTALLSITEMAN1DIR = $(INSTALL_BASE)/man/man1
DESTINSTALLSITEMAN1DIR = $(DESTDIR)$(INSTALLSITEMAN1DIR)
INSTALLVENDORMAN1DIR = $(INSTALL_BASE)/man/man1
DESTINSTALLVENDORMAN1DIR = $(DESTDIR)$(INSTALLVENDORMAN1DIR)
INSTALLMAN3DIR = $(INSTALL_BASE)/man/man3
DESTINSTALLMAN3DIR = $(DESTDIR)$(INSTALLMAN3DIR)
INSTALLSITEMAN3DIR = $(INSTALL_BASE)/man/man3
DESTINSTALLSITEMAN3DIR = $(DESTDIR)$(INSTALLSITEMAN3DIR)
INSTALLVENDORMAN3DIR = $(INSTALL_BASE)/man/man3
DESTINSTALLVENDORMAN3DIR = $(DESTDIR)$(INSTALLVENDORMAN3DIR)
PERL_LIB = /bios-store1/home/yutongli/.conda/envs/try_619_new/lib/perl5/5.22.0
PERL_ARCHLIB = /bios-store1/home/yutongli/.conda/envs/try_619_new/lib/perl5/5.22.0/x86_64-linux-thread-multi
PERL_ARCHLIBDEP = /bios-store1/home/yutongli/.conda/envs/try_619_new/lib/perl5/5.22.0/x86_64-linux-thread-multi
LIBPERL_A = libperl.a
FIRST_MAKEFILE = Makefile
MAKEFILE_OLD = Makefile.old
MAKE_APERL_FILE = Makefile.aperl
PERLMAINCC = $(CC)
PERL_INC = /bios-store1/home/yutongli/.conda/envs/try_619_new/lib/perl5/5.22.0/x86_64-linux-thread-multi/CORE
PERL_INCDEP = /bios-store1/home/yutongli/.conda/envs/try_619_new/lib/perl5/5.22.0/x86_64-linux-thread-multi/CORE
PERL = "/bios-store1/home/yutongli/.conda/envs/try_619_new/bin/perl"
FULLPERL = "/bios-store1/home/yutongli/.conda/envs/try_619_new/bin/perl"
ABSPERL = $(PERL)
PERLRUN = $(PERL)
FULLPERLRUN = $(FULLPERL)
ABSPERLRUN = $(ABSPERL)
PERLRUNINST = $(PERLRUN) "-I$(INST_ARCHLIB)" "-I$(INST_LIB)"
FULLPERLRUNINST = $(FULLPERLRUN) "-I$(INST_ARCHLIB)" "-I$(INST_LIB)"
ABSPERLRUNINST = $(ABSPERLRUN) "-I$(INST_ARCHLIB)" "-I$(INST_LIB)"
PERL_CORE = 0
PERM_DIR = 755
PERM_RW = 644
PERM_RWX = 755

MAKEMAKER   = /bios-store1/home/yutongli/.conda/envs/try_619_new/lib/perl5/5.22.0/ExtUtils/MakeMaker.pm
MM_VERSION  = 7.0401
MM_REVISION = 70401

# FULLEXT = Pathname for extension directory (eg Foo/Bar/Oracle).
# BASEEXT = Basename part of FULLEXT. May be just equal FULLEXT. (eg Oracle)
# PARENT_NAME = NAME without BASEEXT and no trailing :: (eg Foo::Bar)
# DLBASE  = Basename part of dynamic library. May be just equal BASEEXT.
MAKE = make
FULLEXT = BioPerl
BASEEXT = BioPerl
PARENT_NAME = 
DLBASE = $(BASEEXT)
VERSION_FROM = 
OBJECT = 
LDFROM = $(OBJECT)
LINKTYPE = dynamic
BOOTDEP = 

# Handy lists of source code files:
XS_FILES = 
C_FILES  = 
O_FILES  = 
H_FILES  = 
MAN1PODS = bin/bp_aacomp \
	bin/bp_bioflat_index \
	bin/bp_biogetseq \
	bin/bp_dbsplit \
	bin/bp_extract_feature_seq \
	bin/bp_fastam9_to_table \
	bin/bp_fetch \
	bin/bp_filter_search \
	bin/bp_find-blast-matches \
	bin/bp_gccalc \
	bin/bp_genbank2gff3 \
	bin/bp_index \
	bin/bp_local_taxonomydb_query \
	bin/bp_make_mrna_protein \
	bin/bp_mask_by_search \
	bin/bp_mrtrans \
	bin/bp_mutate \
	bin/bp_nexus2nh \
	bin/bp_nrdb \
	bin/bp_oligo_count \
	bin/bp_process_gadfly \
	bin/bp_process_sgd \
	bin/bp_revtrans-motif \
	bin/bp_search2alnblocks \
	bin/bp_search2gff \
	bin/bp_search2table \
	bin/bp_search2tribe \
	bin/bp_seq_length \
	bin/bp_seqconvert \
	bin/bp_seqcut \
	bin/bp_seqpart \
	bin/bp_seqret \
	bin/bp_seqretsplit \
	bin/bp_split_seq \
	bin/bp_sreformat \
	bin/bp_taxid4species \
	bin/bp_taxonomy2tree \
	bin/bp_translate_seq \
	bin/bp_tree2pag \
	bin/bp_unflatten_seq
MAN3PODS = lib/Bio/Align/AlignI.pm \
	lib/Bio/Align/DNAStatistics.pm \
	lib/Bio/Align/PairwiseStatistics.pm \
	lib/Bio/Align/ProteinStatistics.pm \
	lib/Bio/Align/StatisticsI.pm \
	lib/Bio/Align/Utilities.pm \
	lib/Bio/AlignIO.pm \
	lib/Bio/AlignIO/Handler/GenericAlignHandler.pm \
	lib/Bio/AlignIO/arp.pm \
	lib/Bio/AlignIO/bl2seq.pm \
	lib/Bio/AlignIO/clustalw.pm \
	lib/Bio/AlignIO/emboss.pm \
	lib/Bio/AlignIO/fasta.pm \
	lib/Bio/AlignIO/largemultifasta.pm \
	lib/Bio/AlignIO/maf.pm \
	lib/Bio/AlignIO/mase.pm \
	lib/Bio/AlignIO/mega.pm \
	lib/Bio/AlignIO/meme.pm \
	lib/Bio/AlignIO/metafasta.pm \
	lib/Bio/AlignIO/msf.pm \
	lib/Bio/AlignIO/nexus.pm \
	lib/Bio/AlignIO/pfam.pm \
	lib/Bio/AlignIO/phylip.pm \
	lib/Bio/AlignIO/po.pm \
	lib/Bio/AlignIO/proda.pm \
	lib/Bio/AlignIO/prodom.pm \
	lib/Bio/AlignIO/psi.pm \
	lib/Bio/AlignIO/selex.pm \
	lib/Bio/AlignIO/xmfa.pm \
	lib/Bio/AnalysisI.pm \
	lib/Bio/AnalysisParserI.pm \
	lib/Bio/AnalysisResultI.pm \
	lib/Bio/AnnotatableI.pm \
	lib/Bio/Annotation/AnnotationFactory.pm \
	lib/Bio/Annotation/Collection.pm \
	lib/Bio/Annotation/Comment.pm \
	lib/Bio/Annotation/DBLink.pm \
	lib/Bio/Annotation/OntologyTerm.pm \
	lib/Bio/Annotation/Reference.pm \
	lib/Bio/Annotation/Relation.pm \
	lib/Bio/Annotation/SimpleValue.pm \
	lib/Bio/Annotation/StructuredValue.pm \
	lib/Bio/Annotation/TagTree.pm \
	lib/Bio/Annotation/Target.pm \
	lib/Bio/Annotation/Tree.pm \
	lib/Bio/Annotation/TypeManager.pm \
	lib/Bio/AnnotationCollectionI.pm \
	lib/Bio/AnnotationI.pm \
	lib/Bio/CodonUsage/IO.pm \
	lib/Bio/CodonUsage/Table.pm \
	lib/Bio/DB/DBFetch.pm \
	lib/Bio/DB/Failover.pm \
	lib/Bio/DB/Fasta.pm \
	lib/Bio/DB/FileCache.pm \
	lib/Bio/DB/Flat.pm \
	lib/Bio/DB/Flat/BDB.pm \
	lib/Bio/DB/Flat/BDB/embl.pm \
	lib/Bio/DB/Flat/BDB/fasta.pm \
	lib/Bio/DB/Flat/BDB/genbank.pm \
	lib/Bio/DB/Flat/BDB/swiss.pm \
	lib/Bio/DB/Flat/BinarySearch.pm \
	lib/Bio/DB/GFF/Util/Binning.pm \
	lib/Bio/DB/GFF/Util/Rearrange.pm \
	lib/Bio/DB/GenericWebAgent.pm \
	lib/Bio/DB/InMemoryCache.pm \
	lib/Bio/DB/IndexedBase.pm \
	lib/Bio/DB/LocationI.pm \
	lib/Bio/DB/Qual.pm \
	lib/Bio/DB/Query/WebQuery.pm \
	lib/Bio/DB/QueryI.pm \
	lib/Bio/DB/RandomAccessI.pm \
	lib/Bio/DB/ReferenceI.pm \
	lib/Bio/DB/Registry.pm \
	lib/Bio/DB/SeqI.pm \
	lib/Bio/DB/Taxonomy.pm \
	lib/Bio/DB/Taxonomy/flatfile.pm \
	lib/Bio/DB/Taxonomy/greengenes.pm \
	lib/Bio/DB/Taxonomy/list.pm \
	lib/Bio/DB/Taxonomy/silva.pm \
	lib/Bio/DB/UpdateableSeqI.pm \
	lib/Bio/DB/WebDBSeqI.pm \
	lib/Bio/DBLinkContainerI.pm \
	lib/Bio/Das/FeatureTypeI.pm \
	lib/Bio/Das/SegmentI.pm \
	lib/Bio/DasI.pm \
	lib/Bio/DescribableI.pm \
	lib/Bio/Event/EventGeneratorI.pm \
	lib/Bio/Event/EventHandlerI.pm \
	lib/Bio/Factory/AnalysisI.pm \
	lib/Bio/Factory/ApplicationFactoryI.pm \
	lib/Bio/Factory/DriverFactory.pm \
	lib/Bio/Factory/FTLocationFactory.pm \
	lib/Bio/Factory/LocationFactoryI.pm \
	lib/Bio/Factory/ObjectBuilderI.pm \
	lib/Bio/Factory/ObjectFactory.pm \
	lib/Bio/Factory/ObjectFactoryI.pm \
	lib/Bio/Factory/SeqAnalysisParserFactory.pm \
	lib/Bio/Factory/SeqAnalysisParserFactoryI.pm \
	lib/Bio/Factory/SequenceFactoryI.pm \
	lib/Bio/Factory/SequenceProcessorI.pm \
	lib/Bio/Factory/SequenceStreamI.pm \
	lib/Bio/Factory/TreeFactoryI.pm \
	lib/Bio/FeatureHolderI.pm \
	lib/Bio/HandlerBaseI.pm \
	lib/Bio/IdCollectionI.pm \
	lib/Bio/IdentifiableI.pm \
	lib/Bio/Index/Abstract.pm \
	lib/Bio/Index/AbstractSeq.pm \
	lib/Bio/Index/Blast.pm \
	lib/Bio/Index/BlastTable.pm \
	lib/Bio/Index/EMBL.pm \
	lib/Bio/Index/Fasta.pm \
	lib/Bio/Index/Fastq.pm \
	lib/Bio/Index/GenBank.pm \
	lib/Bio/Index/Qual.pm \
	lib/Bio/Index/SwissPfam.pm \
	lib/Bio/Index/Swissprot.pm \
	lib/Bio/LocatableSeq.pm \
	lib/Bio/Location/Atomic.pm \
	lib/Bio/Location/AvWithinCoordPolicy.pm \
	lib/Bio/Location/CoordinatePolicyI.pm \
	lib/Bio/Location/Fuzzy.pm \
	lib/Bio/Location/FuzzyLocationI.pm \
	lib/Bio/Location/NarrowestCoordPolicy.pm \
	lib/Bio/Location/Simple.pm \
	lib/Bio/Location/Split.pm \
	lib/Bio/Location/SplitLocationI.pm \
	lib/Bio/Location/WidestCoordPolicy.pm \
	lib/Bio/LocationI.pm \
	lib/Bio/Matrix/Generic.pm \
	lib/Bio/Matrix/IO.pm \
	lib/Bio/Matrix/IO/mlagan.pm \
	lib/Bio/Matrix/IO/phylip.pm \
	lib/Bio/Matrix/IO/scoring.pm \
	lib/Bio/Matrix/MatrixI.pm \
	lib/Bio/Matrix/Mlagan.pm \
	lib/Bio/Matrix/PSM/IO.pm \
	lib/Bio/Matrix/PSM/IO/mast.pm \
	lib/Bio/Matrix/PSM/IO/masta.pm \
	lib/Bio/Matrix/PSM/IO/meme.pm \
	lib/Bio/Matrix/PSM/IO/psiblast.pm \
	lib/Bio/Matrix/PSM/IO/transfac.pm \
	lib/Bio/Matrix/PSM/InstanceSite.pm \
	lib/Bio/Matrix/PSM/InstanceSiteI.pm \
	lib/Bio/Matrix/PSM/ProtMatrix.pm \
	lib/Bio/Matrix/PSM/ProtPsm.pm \
	lib/Bio/Matrix/PSM/Psm.pm \
	lib/Bio/Matrix/PSM/PsmHeader.pm \
	lib/Bio/Matrix/PSM/PsmHeaderI.pm \
	lib/Bio/Matrix/PSM/PsmI.pm \
	lib/Bio/Matrix/PSM/SiteMatrix.pm \
	lib/Bio/Matrix/PSM/SiteMatrixI.pm \
	lib/Bio/Matrix/PhylipDist.pm \
	lib/Bio/Matrix/Scoring.pm \
	lib/Bio/Ontology/DocumentRegistry.pm \
	lib/Bio/Ontology/GOterm.pm \
	lib/Bio/Ontology/InterProTerm.pm \
	lib/Bio/Ontology/OBOEngine.pm \
	lib/Bio/Ontology/OBOterm.pm \
	lib/Bio/Ontology/Ontology.pm \
	lib/Bio/Ontology/OntologyEngineI.pm \
	lib/Bio/Ontology/OntologyI.pm \
	lib/Bio/Ontology/OntologyStore.pm \
	lib/Bio/Ontology/Path.pm \
	lib/Bio/Ontology/PathI.pm \
	lib/Bio/Ontology/Relationship.pm \
	lib/Bio/Ontology/RelationshipFactory.pm \
	lib/Bio/Ontology/RelationshipI.pm \
	lib/Bio/Ontology/RelationshipType.pm \
	lib/Bio/Ontology/SimpleGOEngine/GraphAdaptor.pm \
	lib/Bio/Ontology/SimpleOntologyEngine.pm \
	lib/Bio/Ontology/Term.pm \
	lib/Bio/Ontology/TermFactory.pm \
	lib/Bio/Ontology/TermI.pm \
	lib/Bio/OntologyIO.pm \
	lib/Bio/OntologyIO/Handlers/BaseSAXHandler.pm \
	lib/Bio/OntologyIO/Handlers/InterProHandler.pm \
	lib/Bio/OntologyIO/Handlers/InterPro_BioSQL_Handler.pm \
	lib/Bio/OntologyIO/InterProParser.pm \
	lib/Bio/OntologyIO/dagflat.pm \
	lib/Bio/OntologyIO/goflat.pm \
	lib/Bio/OntologyIO/obo.pm \
	lib/Bio/OntologyIO/simplehierarchy.pm \
	lib/Bio/OntologyIO/soflat.pm \
	lib/Bio/ParameterBaseI.pm \
	lib/Bio/PrimarySeq.pm \
	lib/Bio/PrimarySeqI.pm \
	lib/Bio/PullParserI.pm \
	lib/Bio/Range.pm \
	lib/Bio/RangeI.pm \
	lib/Bio/Root/Exception.pm \
	lib/Bio/Root/HTTPget.pm \
	lib/Bio/Root/IO.pm \
	lib/Bio/Root/Root.pm \
	lib/Bio/Root/RootI.pm \
	lib/Bio/Root/Storable.pm \
	lib/Bio/Root/Test.pm \
	lib/Bio/Root/TestObject.pm \
	lib/Bio/Root/Utilities.pm \
	lib/Bio/Root/Version.pm \
	lib/Bio/Search/BlastStatistics.pm \
	lib/Bio/Search/BlastUtils.pm \
	lib/Bio/Search/DatabaseI.pm \
	lib/Bio/Search/GenericDatabase.pm \
	lib/Bio/Search/GenericStatistics.pm \
	lib/Bio/Search/HSP/BlastHSP.pm \
	lib/Bio/Search/HSP/BlastPullHSP.pm \
	lib/Bio/Search/HSP/FastaHSP.pm \
	lib/Bio/Search/HSP/GenericHSP.pm \
	lib/Bio/Search/HSP/HSPFactory.pm \
	lib/Bio/Search/HSP/HSPI.pm \
	lib/Bio/Search/HSP/ModelHSP.pm \
	lib/Bio/Search/HSP/PSLHSP.pm \
	lib/Bio/Search/HSP/PsiBlastHSP.pm \
	lib/Bio/Search/HSP/PullHSPI.pm \
	lib/Bio/Search/HSP/WABAHSP.pm \
	lib/Bio/Search/Hit/BlastHit.pm \
	lib/Bio/Search/Hit/BlastPullHit.pm \
	lib/Bio/Search/Hit/Fasta.pm \
	lib/Bio/Search/Hit/GenericHit.pm \
	lib/Bio/Search/Hit/HitFactory.pm \
	lib/Bio/Search/Hit/HitI.pm \
	lib/Bio/Search/Hit/ModelHit.pm \
	lib/Bio/Search/Hit/PsiBlastHit.pm \
	lib/Bio/Search/Hit/PullHitI.pm \
	lib/Bio/Search/Iteration/GenericIteration.pm \
	lib/Bio/Search/Iteration/IterationI.pm \
	lib/Bio/Search/Processor.pm \
	lib/Bio/Search/Result/BlastPullResult.pm \
	lib/Bio/Search/Result/BlastResult.pm \
	lib/Bio/Search/Result/CrossMatchResult.pm \
	lib/Bio/Search/Result/GenericResult.pm \
	lib/Bio/Search/Result/INFERNALResult.pm \
	lib/Bio/Search/Result/PullResultI.pm \
	lib/Bio/Search/Result/ResultFactory.pm \
	lib/Bio/Search/Result/ResultI.pm \
	lib/Bio/Search/Result/WABAResult.pm \
	lib/Bio/Search/SearchUtils.pm \
	lib/Bio/Search/StatisticsI.pm \
	lib/Bio/Search/Tiling/MapTileUtils.pm \
	lib/Bio/Search/Tiling/MapTiling.pm \
	lib/Bio/Search/Tiling/TilingI.pm \
	lib/Bio/SearchIO.pm \
	lib/Bio/SearchIO/EventHandlerI.pm \
	lib/Bio/SearchIO/FastHitEventBuilder.pm \
	lib/Bio/SearchIO/IteratedSearchResultEventBuilder.pm \
	lib/Bio/SearchIO/SearchResultEventBuilder.pm \
	lib/Bio/SearchIO/SearchWriterI.pm \
	lib/Bio/SearchIO/Writer/GbrowseGFF.pm \
	lib/Bio/SearchIO/Writer/HSPTableWriter.pm \
	lib/Bio/SearchIO/Writer/HTMLResultWriter.pm \
	lib/Bio/SearchIO/Writer/HitTableWriter.pm \
	lib/Bio/SearchIO/Writer/ResultTableWriter.pm \
	lib/Bio/SearchIO/Writer/TextResultWriter.pm \
	lib/Bio/SearchIO/axt.pm \
	lib/Bio/SearchIO/blast.pm \
	lib/Bio/SearchIO/blast_pull.pm \
	lib/Bio/SearchIO/blasttable.pm \
	lib/Bio/SearchIO/cross_match.pm \
	lib/Bio/SearchIO/erpin.pm \
	lib/Bio/SearchIO/exonerate.pm \
	lib/Bio/SearchIO/fasta.pm \
	lib/Bio/SearchIO/gmap_f9.pm \
	lib/Bio/SearchIO/infernal.pm \
	lib/Bio/SearchIO/megablast.pm \
	lib/Bio/SearchIO/psl.pm \
	lib/Bio/SearchIO/rnamotif.pm \
	lib/Bio/SearchIO/sim4.pm \
	lib/Bio/SearchIO/waba.pm \
	lib/Bio/SearchIO/wise.pm \
	lib/Bio/Seq.pm \
	lib/Bio/Seq/BaseSeqProcessor.pm \
	lib/Bio/Seq/EncodedSeq.pm \
	lib/Bio/Seq/LargeLocatableSeq.pm \
	lib/Bio/Seq/LargePrimarySeq.pm \
	lib/Bio/Seq/LargeSeq.pm \
	lib/Bio/Seq/LargeSeqI.pm \
	lib/Bio/Seq/Meta.pm \
	lib/Bio/Seq/Meta/Array.pm \
	lib/Bio/Seq/MetaI.pm \
	lib/Bio/Seq/PrimaryQual.pm \
	lib/Bio/Seq/PrimedSeq.pm \
	lib/Bio/Seq/QualI.pm \
	lib/Bio/Seq/Quality.pm \
	lib/Bio/Seq/RichSeq.pm \
	lib/Bio/Seq/RichSeqI.pm \
	lib/Bio/Seq/SeqBuilder.pm \
	lib/Bio/Seq/SeqFactory.pm \
	lib/Bio/Seq/SeqFastaSpeedFactory.pm \
	lib/Bio/Seq/SequenceTrace.pm \
	lib/Bio/Seq/SimulatedRead.pm \
	lib/Bio/Seq/TraceI.pm \
	lib/Bio/SeqAnalysisParserI.pm \
	lib/Bio/SeqFeature/Amplicon.pm \
	lib/Bio/SeqFeature/AnnotationAdaptor.pm \
	lib/Bio/SeqFeature/Collection.pm \
	lib/Bio/SeqFeature/CollectionI.pm \
	lib/Bio/SeqFeature/Computation.pm \
	lib/Bio/SeqFeature/FeaturePair.pm \
	lib/Bio/SeqFeature/Gene/Exon.pm \
	lib/Bio/SeqFeature/Gene/ExonI.pm \
	lib/Bio/SeqFeature/Gene/GeneStructure.pm \
	lib/Bio/SeqFeature/Gene/GeneStructureI.pm \
	lib/Bio/SeqFeature/Gene/Intron.pm \
	lib/Bio/SeqFeature/Gene/NC_Feature.pm \
	lib/Bio/SeqFeature/Gene/Poly_A_site.pm \
	lib/Bio/SeqFeature/Gene/Promoter.pm \
	lib/Bio/SeqFeature/Gene/Transcript.pm \
	lib/Bio/SeqFeature/Gene/TranscriptI.pm \
	lib/Bio/SeqFeature/Gene/UTR.pm \
	lib/Bio/SeqFeature/Generic.pm \
	lib/Bio/SeqFeature/Lite.pm \
	lib/Bio/SeqFeature/PositionProxy.pm \
	lib/Bio/SeqFeature/Primer.pm \
	lib/Bio/SeqFeature/Similarity.pm \
	lib/Bio/SeqFeature/SimilarityPair.pm \
	lib/Bio/SeqFeature/SubSeq.pm \
	lib/Bio/SeqFeature/Tools/FeatureNamer.pm \
	lib/Bio/SeqFeature/Tools/IDHandler.pm \
	lib/Bio/SeqFeature/Tools/TypeMapper.pm \
	lib/Bio/SeqFeature/Tools/Unflattener.pm \
	lib/Bio/SeqFeature/TypedSeqFeatureI.pm \
	lib/Bio/SeqFeatureI.pm \
	lib/Bio/SeqI.pm \
	lib/Bio/SeqIO.pm \
	lib/Bio/SeqIO/FTHelper.pm \
	lib/Bio/SeqIO/Handler/GenericRichSeqHandler.pm \
	lib/Bio/SeqIO/MultiFile.pm \
	lib/Bio/SeqIO/ace.pm \
	lib/Bio/SeqIO/asciitree.pm \
	lib/Bio/SeqIO/bsml.pm \
	lib/Bio/SeqIO/bsml_sax.pm \
	lib/Bio/SeqIO/embl.pm \
	lib/Bio/SeqIO/embldriver.pm \
	lib/Bio/SeqIO/fasta.pm \
	lib/Bio/SeqIO/fastq.pm \
	lib/Bio/SeqIO/game.pm \
	lib/Bio/SeqIO/game/featHandler.pm \
	lib/Bio/SeqIO/game/gameHandler.pm \
	lib/Bio/SeqIO/game/gameSubs.pm \
	lib/Bio/SeqIO/game/gameWriter.pm \
	lib/Bio/SeqIO/game/seqHandler.pm \
	lib/Bio/SeqIO/gbdriver.pm \
	lib/Bio/SeqIO/gbxml.pm \
	lib/Bio/SeqIO/gcg.pm \
	lib/Bio/SeqIO/genbank.pm \
	lib/Bio/SeqIO/kegg.pm \
	lib/Bio/SeqIO/largefasta.pm \
	lib/Bio/SeqIO/locuslink.pm \
	lib/Bio/SeqIO/mbsout.pm \
	lib/Bio/SeqIO/metafasta.pm \
	lib/Bio/SeqIO/msout.pm \
	lib/Bio/SeqIO/phd.pm \
	lib/Bio/SeqIO/pir.pm \
	lib/Bio/SeqIO/qual.pm \
	lib/Bio/SeqIO/raw.pm \
	lib/Bio/SeqIO/scf.pm \
	lib/Bio/SeqIO/seqxml.pm \
	lib/Bio/SeqIO/swiss.pm \
	lib/Bio/SeqIO/swissdriver.pm \
	lib/Bio/SeqIO/tab.pm \
	lib/Bio/SeqIO/table.pm \
	lib/Bio/SeqIO/tigr.pm \
	lib/Bio/SeqIO/tigrxml.pm \
	lib/Bio/SeqIO/tinyseq.pm \
	lib/Bio/SeqIO/tinyseq/tinyseqHandler.pm \
	lib/Bio/SeqUtils.pm \
	lib/Bio/SimpleAlign.pm \
	lib/Bio/SimpleAnalysisI.pm \
	lib/Bio/Species.pm \
	lib/Bio/Taxon.pm \
	lib/Bio/Tools/Alignment/Consed.pm \
	lib/Bio/Tools/Alignment/Trim.pm \
	lib/Bio/Tools/AmpliconSearch.pm \
	lib/Bio/Tools/Analysis/SimpleAnalysisBase.pm \
	lib/Bio/Tools/AnalysisResult.pm \
	lib/Bio/Tools/Blat.pm \
	lib/Bio/Tools/CodonTable.pm \
	lib/Bio/Tools/Coil.pm \
	lib/Bio/Tools/ECnumber.pm \
	lib/Bio/Tools/EMBOSS/Palindrome.pm \
	lib/Bio/Tools/EPCR.pm \
	lib/Bio/Tools/ESTScan.pm \
	lib/Bio/Tools/Eponine.pm \
	lib/Bio/Tools/Est2Genome.pm \
	lib/Bio/Tools/Fgenesh.pm \
	lib/Bio/Tools/FootPrinter.pm \
	lib/Bio/Tools/GFF.pm \
	lib/Bio/Tools/Geneid.pm \
	lib/Bio/Tools/Genemark.pm \
	lib/Bio/Tools/Genewise.pm \
	lib/Bio/Tools/Genomewise.pm \
	lib/Bio/Tools/Genscan.pm \
	lib/Bio/Tools/Glimmer.pm \
	lib/Bio/Tools/Grail.pm \
	lib/Bio/Tools/GuessSeqFormat.pm \
	lib/Bio/Tools/IUPAC.pm \
	lib/Bio/Tools/Lucy.pm \
	lib/Bio/Tools/MZEF.pm \
	lib/Bio/Tools/Match.pm \
	lib/Bio/Tools/OddCodes.pm \
	lib/Bio/Tools/Phylo/Gerp.pm \
	lib/Bio/Tools/Phylo/Molphy.pm \
	lib/Bio/Tools/Phylo/Molphy/Result.pm \
	lib/Bio/Tools/Phylo/Phylip/ProtDist.pm \
	lib/Bio/Tools/Prediction/Exon.pm \
	lib/Bio/Tools/Prediction/Gene.pm \
	lib/Bio/Tools/Primer/Assessor/Base.pm \
	lib/Bio/Tools/Primer/AssessorI.pm \
	lib/Bio/Tools/Primer/Feature.pm \
	lib/Bio/Tools/Primer/Pair.pm \
	lib/Bio/Tools/Primer3.pm \
	lib/Bio/Tools/Prints.pm \
	lib/Bio/Tools/Profile.pm \
	lib/Bio/Tools/Promoterwise.pm \
	lib/Bio/Tools/PrositeScan.pm \
	lib/Bio/Tools/Pseudowise.pm \
	lib/Bio/Tools/QRNA.pm \
	lib/Bio/Tools/RandomDistFunctions.pm \
	lib/Bio/Tools/RepeatMasker.pm \
	lib/Bio/Tools/Run/Analysis.pm \
	lib/Bio/Tools/Run/AnalysisFactory.pm \
	lib/Bio/Tools/Run/GenericParameters.pm \
	lib/Bio/Tools/Run/ParametersI.pm \
	lib/Bio/Tools/Run/Phylo/PhyloBase.pm \
	lib/Bio/Tools/Run/WrapperBase.pm \
	lib/Bio/Tools/Run/WrapperBase/CommandExts.pm \
	lib/Bio/Tools/Seg.pm \
	lib/Bio/Tools/SeqPattern.pm \
	lib/Bio/Tools/SeqPattern/Backtranslate.pm \
	lib/Bio/Tools/SeqStats.pm \
	lib/Bio/Tools/SeqWords.pm \
	lib/Bio/Tools/Sigcleave.pm \
	lib/Bio/Tools/Signalp.pm \
	lib/Bio/Tools/Signalp/ExtendedSignalp.pm \
	lib/Bio/Tools/Sim4/Exon.pm \
	lib/Bio/Tools/Sim4/Results.pm \
	lib/Bio/Tools/Spidey/Exon.pm \
	lib/Bio/Tools/Spidey/Results.pm \
	lib/Bio/Tools/TandemRepeatsFinder.pm \
	lib/Bio/Tools/TargetP.pm \
	lib/Bio/Tools/Tmhmm.pm \
	lib/Bio/Tools/ipcress.pm \
	lib/Bio/Tools/isPcr.pm \
	lib/Bio/Tools/pICalculator.pm \
	lib/Bio/Tools/tRNAscanSE.pm \
	lib/Bio/Tree/AnnotatableNode.pm \
	lib/Bio/Tree/Compatible.pm \
	lib/Bio/Tree/DistanceFactory.pm \
	lib/Bio/Tree/Node.pm \
	lib/Bio/Tree/NodeI.pm \
	lib/Bio/Tree/NodeNHX.pm \
	lib/Bio/Tree/RandomFactory.pm \
	lib/Bio/Tree/Statistics.pm \
	lib/Bio/Tree/Tree.pm \
	lib/Bio/Tree/TreeFunctionsI.pm \
	lib/Bio/Tree/TreeI.pm \
	lib/Bio/TreeIO.pm \
	lib/Bio/TreeIO/NewickParser.pm \
	lib/Bio/TreeIO/TreeEventBuilder.pm \
	lib/Bio/TreeIO/cluster.pm \
	lib/Bio/TreeIO/lintree.pm \
	lib/Bio/TreeIO/newick.pm \
	lib/Bio/TreeIO/nexus.pm \
	lib/Bio/TreeIO/nhx.pm \
	lib/Bio/TreeIO/pag.pm \
	lib/Bio/TreeIO/phyloxml.pm \
	lib/Bio/TreeIO/tabtree.pm \
	lib/Bio/UpdateableSeqI.pm \
	lib/Bio/WebAgent.pm \
	lib/BioPerl.pm

# Where is the Config information that we are using/depend on
CONFIGDEP = $(PERL_ARCHLIBDEP)$(DFSEP)Config.pm $(PERL_INCDEP)$(DFSEP)config.h

# Where to build things
INST_LIBDIR      = $(INST_LIB)
INST_ARCHLIBDIR  = $(INST_ARCHLIB)

INST_AUTODIR     = $(INST_LIB)/auto/$(FULLEXT)
INST_ARCHAUTODIR = $(INST_ARCHLIB)/auto/$(FULLEXT)

INST_STATIC      = 
INST_DYNAMIC     = 
INST_BOOT        = 

# Extra linker info
EXPORT_LIST        = 
PERL_ARCHIVE       = 
PERL_ARCHIVEDEP    = 
PERL_ARCHIVE_AFTER = 


TO_INST_PM = lib/Bio/Align/AlignI.pm \
	lib/Bio/Align/DNAStatistics.pm \
	lib/Bio/Align/PairwiseStatistics.pm \
	lib/Bio/Align/ProteinStatistics.pm \
	lib/Bio/Align/StatisticsI.pm \
	lib/Bio/Align/Utilities.pm \
	lib/Bio/AlignIO.pm \
	lib/Bio/AlignIO/Handler/GenericAlignHandler.pm \
	lib/Bio/AlignIO/arp.pm \
	lib/Bio/AlignIO/bl2seq.pm \
	lib/Bio/AlignIO/clustalw.pm \
	lib/Bio/AlignIO/emboss.pm \
	lib/Bio/AlignIO/fasta.pm \
	lib/Bio/AlignIO/largemultifasta.pm \
	lib/Bio/AlignIO/maf.pm \
	lib/Bio/AlignIO/mase.pm \
	lib/Bio/AlignIO/mega.pm \
	lib/Bio/AlignIO/meme.pm \
	lib/Bio/AlignIO/metafasta.pm \
	lib/Bio/AlignIO/msf.pm \
	lib/Bio/AlignIO/nexus.pm \
	lib/Bio/AlignIO/pfam.pm \
	lib/Bio/AlignIO/phylip.pm \
	lib/Bio/AlignIO/po.pm \
	lib/Bio/AlignIO/proda.pm \
	lib/Bio/AlignIO/prodom.pm \
	lib/Bio/AlignIO/psi.pm \
	lib/Bio/AlignIO/selex.pm \
	lib/Bio/AlignIO/xmfa.pm \
	lib/Bio/AnalysisI.pm \
	lib/Bio/AnalysisParserI.pm \
	lib/Bio/AnalysisResultI.pm \
	lib/Bio/AnnotatableI.pm \
	lib/Bio/Annotation/AnnotationFactory.pm \
	lib/Bio/Annotation/Collection.pm \
	lib/Bio/Annotation/Comment.pm \
	lib/Bio/Annotation/DBLink.pm \
	lib/Bio/Annotation/OntologyTerm.pm \
	lib/Bio/Annotation/Reference.pm \
	lib/Bio/Annotation/Relation.pm \
	lib/Bio/Annotation/SimpleValue.pm \
	lib/Bio/Annotation/StructuredValue.pm \
	lib/Bio/Annotation/TagTree.pm \
	lib/Bio/Annotation/Target.pm \
	lib/Bio/Annotation/Tree.pm \
	lib/Bio/Annotation/TypeManager.pm \
	lib/Bio/AnnotationCollectionI.pm \
	lib/Bio/AnnotationI.pm \
	lib/Bio/CodonUsage/IO.pm \
	lib/Bio/CodonUsage/Table.pm \
	lib/Bio/DB/DBFetch.pm \
	lib/Bio/DB/Failover.pm \
	lib/Bio/DB/Fasta.pm \
	lib/Bio/DB/FileCache.pm \
	lib/Bio/DB/Flat.pm \
	lib/Bio/DB/Flat/BDB.pm \
	lib/Bio/DB/Flat/BDB/embl.pm \
	lib/Bio/DB/Flat/BDB/fasta.pm \
	lib/Bio/DB/Flat/BDB/genbank.pm \
	lib/Bio/DB/Flat/BDB/swiss.pm \
	lib/Bio/DB/Flat/BinarySearch.pm \
	lib/Bio/DB/GFF/Util/Binning.pm \
	lib/Bio/DB/GFF/Util/Rearrange.pm \
	lib/Bio/DB/GenericWebAgent.pm \
	lib/Bio/DB/InMemoryCache.pm \
	lib/Bio/DB/IndexedBase.pm \
	lib/Bio/DB/LocationI.pm \
	lib/Bio/DB/Qual.pm \
	lib/Bio/DB/Query/WebQuery.pm \
	lib/Bio/DB/QueryI.pm \
	lib/Bio/DB/RandomAccessI.pm \
	lib/Bio/DB/ReferenceI.pm \
	lib/Bio/DB/Registry.pm \
	lib/Bio/DB/SeqI.pm \
	lib/Bio/DB/Taxonomy.pm \
	lib/Bio/DB/Taxonomy/flatfile.pm \
	lib/Bio/DB/Taxonomy/greengenes.pm \
	lib/Bio/DB/Taxonomy/list.pm \
	lib/Bio/DB/Taxonomy/silva.pm \
	lib/Bio/DB/UpdateableSeqI.pm \
	lib/Bio/DB/WebDBSeqI.pm \
	lib/Bio/DBLinkContainerI.pm \
	lib/Bio/Das/FeatureTypeI.pm \
	lib/Bio/Das/SegmentI.pm \
	lib/Bio/DasI.pm \
	lib/Bio/DescribableI.pm \
	lib/Bio/Event/EventGeneratorI.pm \
	lib/Bio/Event/EventHandlerI.pm \
	lib/Bio/Factory/AnalysisI.pm \
	lib/Bio/Factory/ApplicationFactoryI.pm \
	lib/Bio/Factory/DriverFactory.pm \
	lib/Bio/Factory/FTLocationFactory.pm \
	lib/Bio/Factory/LocationFactoryI.pm \
	lib/Bio/Factory/ObjectBuilderI.pm \
	lib/Bio/Factory/ObjectFactory.pm \
	lib/Bio/Factory/ObjectFactoryI.pm \
	lib/Bio/Factory/SeqAnalysisParserFactory.pm \
	lib/Bio/Factory/SeqAnalysisParserFactoryI.pm \
	lib/Bio/Factory/SequenceFactoryI.pm \
	lib/Bio/Factory/SequenceProcessorI.pm \
	lib/Bio/Factory/SequenceStreamI.pm \
	lib/Bio/Factory/TreeFactoryI.pm \
	lib/Bio/FeatureHolderI.pm \
	lib/Bio/HandlerBaseI.pm \
	lib/Bio/IdCollectionI.pm \
	lib/Bio/IdentifiableI.pm \
	lib/Bio/Index/Abstract.pm \
	lib/Bio/Index/AbstractSeq.pm \
	lib/Bio/Index/Blast.pm \
	lib/Bio/Index/BlastTable.pm \
	lib/Bio/Index/EMBL.pm \
	lib/Bio/Index/Fasta.pm \
	lib/Bio/Index/Fastq.pm \
	lib/Bio/Index/GenBank.pm \
	lib/Bio/Index/Qual.pm \
	lib/Bio/Index/SwissPfam.pm \
	lib/Bio/Index/Swissprot.pm \
	lib/Bio/LocatableSeq.pm \
	lib/Bio/Location/Atomic.pm \
	lib/Bio/Location/AvWithinCoordPolicy.pm \
	lib/Bio/Location/CoordinatePolicyI.pm \
	lib/Bio/Location/Fuzzy.pm \
	lib/Bio/Location/FuzzyLocationI.pm \
	lib/Bio/Location/NarrowestCoordPolicy.pm \
	lib/Bio/Location/Simple.pm \
	lib/Bio/Location/Split.pm \
	lib/Bio/Location/SplitLocationI.pm \
	lib/Bio/Location/WidestCoordPolicy.pm \
	lib/Bio/LocationI.pm \
	lib/Bio/Matrix/Generic.pm \
	lib/Bio/Matrix/IO.pm \
	lib/Bio/Matrix/IO/mlagan.pm \
	lib/Bio/Matrix/IO/phylip.pm \
	lib/Bio/Matrix/IO/scoring.pm \
	lib/Bio/Matrix/MatrixI.pm \
	lib/Bio/Matrix/Mlagan.pm \
	lib/Bio/Matrix/PSM/IO.pm \
	lib/Bio/Matrix/PSM/IO/mast.pm \
	lib/Bio/Matrix/PSM/IO/masta.pm \
	lib/Bio/Matrix/PSM/IO/meme.pm \
	lib/Bio/Matrix/PSM/IO/psiblast.pm \
	lib/Bio/Matrix/PSM/IO/transfac.pm \
	lib/Bio/Matrix/PSM/InstanceSite.pm \
	lib/Bio/Matrix/PSM/InstanceSiteI.pm \
	lib/Bio/Matrix/PSM/ProtMatrix.pm \
	lib/Bio/Matrix/PSM/ProtPsm.pm \
	lib/Bio/Matrix/PSM/Psm.pm \
	lib/Bio/Matrix/PSM/PsmHeader.pm \
	lib/Bio/Matrix/PSM/PsmHeaderI.pm \
	lib/Bio/Matrix/PSM/PsmI.pm \
	lib/Bio/Matrix/PSM/SiteMatrix.pm \
	lib/Bio/Matrix/PSM/SiteMatrixI.pm \
	lib/Bio/Matrix/PhylipDist.pm \
	lib/Bio/Matrix/Scoring.pm \
	lib/Bio/Ontology/DocumentRegistry.pm \
	lib/Bio/Ontology/GOterm.pm \
	lib/Bio/Ontology/InterProTerm.pm \
	lib/Bio/Ontology/OBOEngine.pm \
	lib/Bio/Ontology/OBOterm.pm \
	lib/Bio/Ontology/Ontology.pm \
	lib/Bio/Ontology/OntologyEngineI.pm \
	lib/Bio/Ontology/OntologyI.pm \
	lib/Bio/Ontology/OntologyStore.pm \
	lib/Bio/Ontology/Path.pm \
	lib/Bio/Ontology/PathI.pm \
	lib/Bio/Ontology/Relationship.pm \
	lib/Bio/Ontology/RelationshipFactory.pm \
	lib/Bio/Ontology/RelationshipI.pm \
	lib/Bio/Ontology/RelationshipType.pm \
	lib/Bio/Ontology/SimpleGOEngine/GraphAdaptor.pm \
	lib/Bio/Ontology/SimpleOntologyEngine.pm \
	lib/Bio/Ontology/Term.pm \
	lib/Bio/Ontology/TermFactory.pm \
	lib/Bio/Ontology/TermI.pm \
	lib/Bio/OntologyIO.pm \
	lib/Bio/OntologyIO/Handlers/BaseSAXHandler.pm \
	lib/Bio/OntologyIO/Handlers/InterProHandler.pm \
	lib/Bio/OntologyIO/Handlers/InterPro_BioSQL_Handler.pm \
	lib/Bio/OntologyIO/InterProParser.pm \
	lib/Bio/OntologyIO/dagflat.pm \
	lib/Bio/OntologyIO/goflat.pm \
	lib/Bio/OntologyIO/obo.pm \
	lib/Bio/OntologyIO/simplehierarchy.pm \
	lib/Bio/OntologyIO/soflat.pm \
	lib/Bio/ParameterBaseI.pm \
	lib/Bio/PrimarySeq.pm \
	lib/Bio/PrimarySeqI.pm \
	lib/Bio/PullParserI.pm \
	lib/Bio/Range.pm \
	lib/Bio/RangeI.pm \
	lib/Bio/Root/Exception.pm \
	lib/Bio/Root/HTTPget.pm \
	lib/Bio/Root/IO.pm \
	lib/Bio/Root/Root.pm \
	lib/Bio/Root/RootI.pm \
	lib/Bio/Root/Storable.pm \
	lib/Bio/Root/Test.pm \
	lib/Bio/Root/TestObject.pm \
	lib/Bio/Root/Utilities.pm \
	lib/Bio/Root/Version.pm \
	lib/Bio/Search/BlastStatistics.pm \
	lib/Bio/Search/BlastUtils.pm \
	lib/Bio/Search/DatabaseI.pm \
	lib/Bio/Search/GenericDatabase.pm \
	lib/Bio/Search/GenericStatistics.pm \
	lib/Bio/Search/HSP/BlastHSP.pm \
	lib/Bio/Search/HSP/BlastPullHSP.pm \
	lib/Bio/Search/HSP/FastaHSP.pm \
	lib/Bio/Search/HSP/GenericHSP.pm \
	lib/Bio/Search/HSP/HSPFactory.pm \
	lib/Bio/Search/HSP/HSPI.pm \
	lib/Bio/Search/HSP/ModelHSP.pm \
	lib/Bio/Search/HSP/PSLHSP.pm \
	lib/Bio/Search/HSP/PsiBlastHSP.pm \
	lib/Bio/Search/HSP/PullHSPI.pm \
	lib/Bio/Search/HSP/WABAHSP.pm \
	lib/Bio/Search/Hit/BlastHit.pm \
	lib/Bio/Search/Hit/BlastPullHit.pm \
	lib/Bio/Search/Hit/Fasta.pm \
	lib/Bio/Search/Hit/GenericHit.pm \
	lib/Bio/Search/Hit/HitFactory.pm \
	lib/Bio/Search/Hit/HitI.pm \
	lib/Bio/Search/Hit/ModelHit.pm \
	lib/Bio/Search/Hit/PsiBlastHit.pm \
	lib/Bio/Search/Hit/PullHitI.pm \
	lib/Bio/Search/Iteration/GenericIteration.pm \
	lib/Bio/Search/Iteration/IterationI.pm \
	lib/Bio/Search/Processor.pm \
	lib/Bio/Search/Result/BlastPullResult.pm \
	lib/Bio/Search/Result/BlastResult.pm \
	lib/Bio/Search/Result/CrossMatchResult.pm \
	lib/Bio/Search/Result/GenericResult.pm \
	lib/Bio/Search/Result/INFERNALResult.pm \
	lib/Bio/Search/Result/PullResultI.pm \
	lib/Bio/Search/Result/ResultFactory.pm \
	lib/Bio/Search/Result/ResultI.pm \
	lib/Bio/Search/Result/WABAResult.pm \
	lib/Bio/Search/SearchUtils.pm \
	lib/Bio/Search/StatisticsI.pm \
	lib/Bio/Search/Tiling/MapTileUtils.pm \
	lib/Bio/Search/Tiling/MapTiling.pm \
	lib/Bio/Search/Tiling/TilingI.pm \
	lib/Bio/SearchIO.pm \
	lib/Bio/SearchIO/EventHandlerI.pm \
	lib/Bio/SearchIO/FastHitEventBuilder.pm \
	lib/Bio/SearchIO/IteratedSearchResultEventBuilder.pm \
	lib/Bio/SearchIO/SearchResultEventBuilder.pm \
	lib/Bio/SearchIO/SearchWriterI.pm \
	lib/Bio/SearchIO/Writer/GbrowseGFF.pm \
	lib/Bio/SearchIO/Writer/HSPTableWriter.pm \
	lib/Bio/SearchIO/Writer/HTMLResultWriter.pm \
	lib/Bio/SearchIO/Writer/HitTableWriter.pm \
	lib/Bio/SearchIO/Writer/ResultTableWriter.pm \
	lib/Bio/SearchIO/Writer/TextResultWriter.pm \
	lib/Bio/SearchIO/axt.pm \
	lib/Bio/SearchIO/blast.pm \
	lib/Bio/SearchIO/blast_pull.pm \
	lib/Bio/SearchIO/blasttable.pm \
	lib/Bio/SearchIO/cross_match.pm \
	lib/Bio/SearchIO/erpin.pm \
	lib/Bio/SearchIO/exonerate.pm \
	lib/Bio/SearchIO/fasta.pm \
	lib/Bio/SearchIO/gmap_f9.pm \
	lib/Bio/SearchIO/infernal.pm \
	lib/Bio/SearchIO/megablast.pm \
	lib/Bio/SearchIO/psl.pm \
	lib/Bio/SearchIO/rnamotif.pm \
	lib/Bio/SearchIO/sim4.pm \
	lib/Bio/SearchIO/waba.pm \
	lib/Bio/SearchIO/wise.pm \
	lib/Bio/Seq.pm \
	lib/Bio/Seq/BaseSeqProcessor.pm \
	lib/Bio/Seq/EncodedSeq.pm \
	lib/Bio/Seq/LargeLocatableSeq.pm \
	lib/Bio/Seq/LargePrimarySeq.pm \
	lib/Bio/Seq/LargeSeq.pm \
	lib/Bio/Seq/LargeSeqI.pm \
	lib/Bio/Seq/Meta.pm \
	lib/Bio/Seq/Meta/Array.pm \
	lib/Bio/Seq/MetaI.pm \
	lib/Bio/Seq/PrimaryQual.pm \
	lib/Bio/Seq/PrimedSeq.pm \
	lib/Bio/Seq/QualI.pm \
	lib/Bio/Seq/Quality.pm \
	lib/Bio/Seq/RichSeq.pm \
	lib/Bio/Seq/RichSeqI.pm \
	lib/Bio/Seq/SeqBuilder.pm \
	lib/Bio/Seq/SeqFactory.pm \
	lib/Bio/Seq/SeqFastaSpeedFactory.pm \
	lib/Bio/Seq/SequenceTrace.pm \
	lib/Bio/Seq/SimulatedRead.pm \
	lib/Bio/Seq/TraceI.pm \
	lib/Bio/SeqAnalysisParserI.pm \
	lib/Bio/SeqFeature/Amplicon.pm \
	lib/Bio/SeqFeature/AnnotationAdaptor.pm \
	lib/Bio/SeqFeature/Collection.pm \
	lib/Bio/SeqFeature/CollectionI.pm \
	lib/Bio/SeqFeature/Computation.pm \
	lib/Bio/SeqFeature/FeaturePair.pm \
	lib/Bio/SeqFeature/Gene/Exon.pm \
	lib/Bio/SeqFeature/Gene/ExonI.pm \
	lib/Bio/SeqFeature/Gene/GeneStructure.pm \
	lib/Bio/SeqFeature/Gene/GeneStructureI.pm \
	lib/Bio/SeqFeature/Gene/Intron.pm \
	lib/Bio/SeqFeature/Gene/NC_Feature.pm \
	lib/Bio/SeqFeature/Gene/Poly_A_site.pm \
	lib/Bio/SeqFeature/Gene/Promoter.pm \
	lib/Bio/SeqFeature/Gene/Transcript.pm \
	lib/Bio/SeqFeature/Gene/TranscriptI.pm \
	lib/Bio/SeqFeature/Gene/UTR.pm \
	lib/Bio/SeqFeature/Generic.pm \
	lib/Bio/SeqFeature/Lite.pm \
	lib/Bio/SeqFeature/PositionProxy.pm \
	lib/Bio/SeqFeature/Primer.pm \
	lib/Bio/SeqFeature/Similarity.pm \
	lib/Bio/SeqFeature/SimilarityPair.pm \
	lib/Bio/SeqFeature/SubSeq.pm \
	lib/Bio/SeqFeature/Tools/FeatureNamer.pm \
	lib/Bio/SeqFeature/Tools/IDHandler.pm \
	lib/Bio/SeqFeature/Tools/TypeMapper.pm \
	lib/Bio/SeqFeature/Tools/Unflattener.pm \
	lib/Bio/SeqFeature/TypedSeqFeatureI.pm \
	lib/Bio/SeqFeatureI.pm \
	lib/Bio/SeqI.pm \
	lib/Bio/SeqIO.pm \
	lib/Bio/SeqIO/FTHelper.pm \
	lib/Bio/SeqIO/Handler/GenericRichSeqHandler.pm \
	lib/Bio/SeqIO/MultiFile.pm \
	lib/Bio/SeqIO/ace.pm \
	lib/Bio/SeqIO/asciitree.pm \
	lib/Bio/SeqIO/bsml.pm \
	lib/Bio/SeqIO/bsml_sax.pm \
	lib/Bio/SeqIO/embl.pm \
	lib/Bio/SeqIO/embldriver.pm \
	lib/Bio/SeqIO/fasta.pm \
	lib/Bio/SeqIO/fastq.pm \
	lib/Bio/SeqIO/game.pm \
	lib/Bio/SeqIO/game/featHandler.pm \
	lib/Bio/SeqIO/game/gameHandler.pm \
	lib/Bio/SeqIO/game/gameSubs.pm \
	lib/Bio/SeqIO/game/gameWriter.pm \
	lib/Bio/SeqIO/game/seqHandler.pm \
	lib/Bio/SeqIO/gbdriver.pm \
	lib/Bio/SeqIO/gbxml.pm \
	lib/Bio/SeqIO/gcg.pm \
	lib/Bio/SeqIO/genbank.pm \
	lib/Bio/SeqIO/kegg.pm \
	lib/Bio/SeqIO/largefasta.pm \
	lib/Bio/SeqIO/locuslink.pm \
	lib/Bio/SeqIO/mbsout.pm \
	lib/Bio/SeqIO/metafasta.pm \
	lib/Bio/SeqIO/msout.pm \
	lib/Bio/SeqIO/phd.pm \
	lib/Bio/SeqIO/pir.pm \
	lib/Bio/SeqIO/qual.pm \
	lib/Bio/SeqIO/raw.pm \
	lib/Bio/SeqIO/scf.pm \
	lib/Bio/SeqIO/seqxml.pm \
	lib/Bio/SeqIO/swiss.pm \
	lib/Bio/SeqIO/swissdriver.pm \
	lib/Bio/SeqIO/tab.pm \
	lib/Bio/SeqIO/table.pm \
	lib/Bio/SeqIO/tigr.pm \
	lib/Bio/SeqIO/tigrxml.pm \
	lib/Bio/SeqIO/tinyseq.pm \
	lib/Bio/SeqIO/tinyseq/tinyseqHandler.pm \
	lib/Bio/SeqUtils.pm \
	lib/Bio/SimpleAlign.pm \
	lib/Bio/SimpleAnalysisI.pm \
	lib/Bio/Species.pm \
	lib/Bio/Taxon.pm \
	lib/Bio/Tools/Alignment/Consed.pm \
	lib/Bio/Tools/Alignment/Trim.pm \
	lib/Bio/Tools/AmpliconSearch.pm \
	lib/Bio/Tools/Analysis/SimpleAnalysisBase.pm \
	lib/Bio/Tools/AnalysisResult.pm \
	lib/Bio/Tools/Blat.pm \
	lib/Bio/Tools/CodonTable.pm \
	lib/Bio/Tools/Coil.pm \
	lib/Bio/Tools/ECnumber.pm \
	lib/Bio/Tools/EMBOSS/Palindrome.pm \
	lib/Bio/Tools/EPCR.pm \
	lib/Bio/Tools/ESTScan.pm \
	lib/Bio/Tools/Eponine.pm \
	lib/Bio/Tools/Est2Genome.pm \
	lib/Bio/Tools/Fgenesh.pm \
	lib/Bio/Tools/FootPrinter.pm \
	lib/Bio/Tools/GFF.pm \
	lib/Bio/Tools/Geneid.pm \
	lib/Bio/Tools/Genemark.pm \
	lib/Bio/Tools/Genewise.pm \
	lib/Bio/Tools/Genomewise.pm \
	lib/Bio/Tools/Genscan.pm \
	lib/Bio/Tools/Glimmer.pm \
	lib/Bio/Tools/Grail.pm \
	lib/Bio/Tools/GuessSeqFormat.pm \
	lib/Bio/Tools/IUPAC.pm \
	lib/Bio/Tools/Lucy.pm \
	lib/Bio/Tools/MZEF.pm \
	lib/Bio/Tools/Match.pm \
	lib/Bio/Tools/OddCodes.pm \
	lib/Bio/Tools/Phylo/Gerp.pm \
	lib/Bio/Tools/Phylo/Molphy.pm \
	lib/Bio/Tools/Phylo/Molphy/Result.pm \
	lib/Bio/Tools/Phylo/Phylip/ProtDist.pm \
	lib/Bio/Tools/Prediction/Exon.pm \
	lib/Bio/Tools/Prediction/Gene.pm \
	lib/Bio/Tools/Primer/Assessor/Base.pm \
	lib/Bio/Tools/Primer/AssessorI.pm \
	lib/Bio/Tools/Primer/Feature.pm \
	lib/Bio/Tools/Primer/Pair.pm \
	lib/Bio/Tools/Primer3.pm \
	lib/Bio/Tools/Prints.pm \
	lib/Bio/Tools/Profile.pm \
	lib/Bio/Tools/Promoterwise.pm \
	lib/Bio/Tools/PrositeScan.pm \
	lib/Bio/Tools/Pseudowise.pm \
	lib/Bio/Tools/QRNA.pm \
	lib/Bio/Tools/RandomDistFunctions.pm \
	lib/Bio/Tools/RepeatMasker.pm \
	lib/Bio/Tools/Run/Analysis.pm \
	lib/Bio/Tools/Run/AnalysisFactory.pm \
	lib/Bio/Tools/Run/GenericParameters.pm \
	lib/Bio/Tools/Run/ParametersI.pm \
	lib/Bio/Tools/Run/Phylo/PhyloBase.pm \
	lib/Bio/Tools/Run/WrapperBase.pm \
	lib/Bio/Tools/Run/WrapperBase/CommandExts.pm \
	lib/Bio/Tools/Seg.pm \
	lib/Bio/Tools/SeqPattern.pm \
	lib/Bio/Tools/SeqPattern/Backtranslate.pm \
	lib/Bio/Tools/SeqStats.pm \
	lib/Bio/Tools/SeqWords.pm \
	lib/Bio/Tools/Sigcleave.pm \
	lib/Bio/Tools/Signalp.pm \
	lib/Bio/Tools/Signalp/ExtendedSignalp.pm \
	lib/Bio/Tools/Sim4/Exon.pm \
	lib/Bio/Tools/Sim4/Results.pm \
	lib/Bio/Tools/Spidey/Exon.pm \
	lib/Bio/Tools/Spidey/Results.pm \
	lib/Bio/Tools/TandemRepeatsFinder.pm \
	lib/Bio/Tools/TargetP.pm \
	lib/Bio/Tools/Tmhmm.pm \
	lib/Bio/Tools/ipcress.pm \
	lib/Bio/Tools/isPcr.pm \
	lib/Bio/Tools/pICalculator.pm \
	lib/Bio/Tools/tRNAscanSE.pm \
	lib/Bio/Tree/AnnotatableNode.pm \
	lib/Bio/Tree/Compatible.pm \
	lib/Bio/Tree/DistanceFactory.pm \
	lib/Bio/Tree/Node.pm \
	lib/Bio/Tree/NodeI.pm \
	lib/Bio/Tree/NodeNHX.pm \
	lib/Bio/Tree/RandomFactory.pm \
	lib/Bio/Tree/Statistics.pm \
	lib/Bio/Tree/Tree.pm \
	lib/Bio/Tree/TreeFunctionsI.pm \
	lib/Bio/Tree/TreeI.pm \
	lib/Bio/TreeIO.pm \
	lib/Bio/TreeIO/NewickParser.pm \
	lib/Bio/TreeIO/TreeEventBuilder.pm \
	lib/Bio/TreeIO/cluster.pm \
	lib/Bio/TreeIO/lintree.pm \
	lib/Bio/TreeIO/newick.pm \
	lib/Bio/TreeIO/nexus.pm \
	lib/Bio/TreeIO/nhx.pm \
	lib/Bio/TreeIO/pag.pm \
	lib/Bio/TreeIO/phyloxml.pm \
	lib/Bio/TreeIO/tabtree.pm \
	lib/Bio/UpdateableSeqI.pm \
	lib/Bio/WebAgent.pm \
	lib/BioPerl.pm

PM_TO_BLIB = lib/Bio/Align/AlignI.pm \
	blib/lib/Bio/Align/AlignI.pm \
	lib/Bio/Align/DNAStatistics.pm \
	blib/lib/Bio/Align/DNAStatistics.pm \
	lib/Bio/Align/PairwiseStatistics.pm \
	blib/lib/Bio/Align/PairwiseStatistics.pm \
	lib/Bio/Align/ProteinStatistics.pm \
	blib/lib/Bio/Align/ProteinStatistics.pm \
	lib/Bio/Align/StatisticsI.pm \
	blib/lib/Bio/Align/StatisticsI.pm \
	lib/Bio/Align/Utilities.pm \
	blib/lib/Bio/Align/Utilities.pm \
	lib/Bio/AlignIO.pm \
	blib/lib/Bio/AlignIO.pm \
	lib/Bio/AlignIO/Handler/GenericAlignHandler.pm \
	blib/lib/Bio/AlignIO/Handler/GenericAlignHandler.pm \
	lib/Bio/AlignIO/arp.pm \
	blib/lib/Bio/AlignIO/arp.pm \
	lib/Bio/AlignIO/bl2seq.pm \
	blib/lib/Bio/AlignIO/bl2seq.pm \
	lib/Bio/AlignIO/clustalw.pm \
	blib/lib/Bio/AlignIO/clustalw.pm \
	lib/Bio/AlignIO/emboss.pm \
	blib/lib/Bio/AlignIO/emboss.pm \
	lib/Bio/AlignIO/fasta.pm \
	blib/lib/Bio/AlignIO/fasta.pm \
	lib/Bio/AlignIO/largemultifasta.pm \
	blib/lib/Bio/AlignIO/largemultifasta.pm \
	lib/Bio/AlignIO/maf.pm \
	blib/lib/Bio/AlignIO/maf.pm \
	lib/Bio/AlignIO/mase.pm \
	blib/lib/Bio/AlignIO/mase.pm \
	lib/Bio/AlignIO/mega.pm \
	blib/lib/Bio/AlignIO/mega.pm \
	lib/Bio/AlignIO/meme.pm \
	blib/lib/Bio/AlignIO/meme.pm \
	lib/Bio/AlignIO/metafasta.pm \
	blib/lib/Bio/AlignIO/metafasta.pm \
	lib/Bio/AlignIO/msf.pm \
	blib/lib/Bio/AlignIO/msf.pm \
	lib/Bio/AlignIO/nexus.pm \
	blib/lib/Bio/AlignIO/nexus.pm \
	lib/Bio/AlignIO/pfam.pm \
	blib/lib/Bio/AlignIO/pfam.pm \
	lib/Bio/AlignIO/phylip.pm \
	blib/lib/Bio/AlignIO/phylip.pm \
	lib/Bio/AlignIO/po.pm \
	blib/lib/Bio/AlignIO/po.pm \
	lib/Bio/AlignIO/proda.pm \
	blib/lib/Bio/AlignIO/proda.pm \
	lib/Bio/AlignIO/prodom.pm \
	blib/lib/Bio/AlignIO/prodom.pm \
	lib/Bio/AlignIO/psi.pm \
	blib/lib/Bio/AlignIO/psi.pm \
	lib/Bio/AlignIO/selex.pm \
	blib/lib/Bio/AlignIO/selex.pm \
	lib/Bio/AlignIO/xmfa.pm \
	blib/lib/Bio/AlignIO/xmfa.pm \
	lib/Bio/AnalysisI.pm \
	blib/lib/Bio/AnalysisI.pm \
	lib/Bio/AnalysisParserI.pm \
	blib/lib/Bio/AnalysisParserI.pm \
	lib/Bio/AnalysisResultI.pm \
	blib/lib/Bio/AnalysisResultI.pm \
	lib/Bio/AnnotatableI.pm \
	blib/lib/Bio/AnnotatableI.pm \
	lib/Bio/Annotation/AnnotationFactory.pm \
	blib/lib/Bio/Annotation/AnnotationFactory.pm \
	lib/Bio/Annotation/Collection.pm \
	blib/lib/Bio/Annotation/Collection.pm \
	lib/Bio/Annotation/Comment.pm \
	blib/lib/Bio/Annotation/Comment.pm \
	lib/Bio/Annotation/DBLink.pm \
	blib/lib/Bio/Annotation/DBLink.pm \
	lib/Bio/Annotation/OntologyTerm.pm \
	blib/lib/Bio/Annotation/OntologyTerm.pm \
	lib/Bio/Annotation/Reference.pm \
	blib/lib/Bio/Annotation/Reference.pm \
	lib/Bio/Annotation/Relation.pm \
	blib/lib/Bio/Annotation/Relation.pm \
	lib/Bio/Annotation/SimpleValue.pm \
	blib/lib/Bio/Annotation/SimpleValue.pm \
	lib/Bio/Annotation/StructuredValue.pm \
	blib/lib/Bio/Annotation/StructuredValue.pm \
	lib/Bio/Annotation/TagTree.pm \
	blib/lib/Bio/Annotation/TagTree.pm \
	lib/Bio/Annotation/Target.pm \
	blib/lib/Bio/Annotation/Target.pm \
	lib/Bio/Annotation/Tree.pm \
	blib/lib/Bio/Annotation/Tree.pm \
	lib/Bio/Annotation/TypeManager.pm \
	blib/lib/Bio/Annotation/TypeManager.pm \
	lib/Bio/AnnotationCollectionI.pm \
	blib/lib/Bio/AnnotationCollectionI.pm \
	lib/Bio/AnnotationI.pm \
	blib/lib/Bio/AnnotationI.pm \
	lib/Bio/CodonUsage/IO.pm \
	blib/lib/Bio/CodonUsage/IO.pm \
	lib/Bio/CodonUsage/Table.pm \
	blib/lib/Bio/CodonUsage/Table.pm \
	lib/Bio/DB/DBFetch.pm \
	blib/lib/Bio/DB/DBFetch.pm \
	lib/Bio/DB/Failover.pm \
	blib/lib/Bio/DB/Failover.pm \
	lib/Bio/DB/Fasta.pm \
	blib/lib/Bio/DB/Fasta.pm \
	lib/Bio/DB/FileCache.pm \
	blib/lib/Bio/DB/FileCache.pm \
	lib/Bio/DB/Flat.pm \
	blib/lib/Bio/DB/Flat.pm \
	lib/Bio/DB/Flat/BDB.pm \
	blib/lib/Bio/DB/Flat/BDB.pm \
	lib/Bio/DB/Flat/BDB/embl.pm \
	blib/lib/Bio/DB/Flat/BDB/embl.pm \
	lib/Bio/DB/Flat/BDB/fasta.pm \
	blib/lib/Bio/DB/Flat/BDB/fasta.pm \
	lib/Bio/DB/Flat/BDB/genbank.pm \
	blib/lib/Bio/DB/Flat/BDB/genbank.pm \
	lib/Bio/DB/Flat/BDB/swiss.pm \
	blib/lib/Bio/DB/Flat/BDB/swiss.pm \
	lib/Bio/DB/Flat/BinarySearch.pm \
	blib/lib/Bio/DB/Flat/BinarySearch.pm \
	lib/Bio/DB/GFF/Util/Binning.pm \
	blib/lib/Bio/DB/GFF/Util/Binning.pm \
	lib/Bio/DB/GFF/Util/Rearrange.pm \
	blib/lib/Bio/DB/GFF/Util/Rearrange.pm \
	lib/Bio/DB/GenericWebAgent.pm \
	blib/lib/Bio/DB/GenericWebAgent.pm \
	lib/Bio/DB/InMemoryCache.pm \
	blib/lib/Bio/DB/InMemoryCache.pm \
	lib/Bio/DB/IndexedBase.pm \
	blib/lib/Bio/DB/IndexedBase.pm \
	lib/Bio/DB/LocationI.pm \
	blib/lib/Bio/DB/LocationI.pm \
	lib/Bio/DB/Qual.pm \
	blib/lib/Bio/DB/Qual.pm \
	lib/Bio/DB/Query/WebQuery.pm \
	blib/lib/Bio/DB/Query/WebQuery.pm \
	lib/Bio/DB/QueryI.pm \
	blib/lib/Bio/DB/QueryI.pm \
	lib/Bio/DB/RandomAccessI.pm \
	blib/lib/Bio/DB/RandomAccessI.pm \
	lib/Bio/DB/ReferenceI.pm \
	blib/lib/Bio/DB/ReferenceI.pm \
	lib/Bio/DB/Registry.pm \
	blib/lib/Bio/DB/Registry.pm \
	lib/Bio/DB/SeqI.pm \
	blib/lib/Bio/DB/SeqI.pm \
	lib/Bio/DB/Taxonomy.pm \
	blib/lib/Bio/DB/Taxonomy.pm \
	lib/Bio/DB/Taxonomy/flatfile.pm \
	blib/lib/Bio/DB/Taxonomy/flatfile.pm \
	lib/Bio/DB/Taxonomy/greengenes.pm \
	blib/lib/Bio/DB/Taxonomy/greengenes.pm \
	lib/Bio/DB/Taxonomy/list.pm \
	blib/lib/Bio/DB/Taxonomy/list.pm \
	lib/Bio/DB/Taxonomy/silva.pm \
	blib/lib/Bio/DB/Taxonomy/silva.pm \
	lib/Bio/DB/UpdateableSeqI.pm \
	blib/lib/Bio/DB/UpdateableSeqI.pm \
	lib/Bio/DB/WebDBSeqI.pm \
	blib/lib/Bio/DB/WebDBSeqI.pm \
	lib/Bio/DBLinkContainerI.pm \
	blib/lib/Bio/DBLinkContainerI.pm \
	lib/Bio/Das/FeatureTypeI.pm \
	blib/lib/Bio/Das/FeatureTypeI.pm \
	lib/Bio/Das/SegmentI.pm \
	blib/lib/Bio/Das/SegmentI.pm \
	lib/Bio/DasI.pm \
	blib/lib/Bio/DasI.pm \
	lib/Bio/DescribableI.pm \
	blib/lib/Bio/DescribableI.pm \
	lib/Bio/Event/EventGeneratorI.pm \
	blib/lib/Bio/Event/EventGeneratorI.pm \
	lib/Bio/Event/EventHandlerI.pm \
	blib/lib/Bio/Event/EventHandlerI.pm \
	lib/Bio/Factory/AnalysisI.pm \
	blib/lib/Bio/Factory/AnalysisI.pm \
	lib/Bio/Factory/ApplicationFactoryI.pm \
	blib/lib/Bio/Factory/ApplicationFactoryI.pm \
	lib/Bio/Factory/DriverFactory.pm \
	blib/lib/Bio/Factory/DriverFactory.pm \
	lib/Bio/Factory/FTLocationFactory.pm \
	blib/lib/Bio/Factory/FTLocationFactory.pm \
	lib/Bio/Factory/LocationFactoryI.pm \
	blib/lib/Bio/Factory/LocationFactoryI.pm \
	lib/Bio/Factory/ObjectBuilderI.pm \
	blib/lib/Bio/Factory/ObjectBuilderI.pm \
	lib/Bio/Factory/ObjectFactory.pm \
	blib/lib/Bio/Factory/ObjectFactory.pm \
	lib/Bio/Factory/ObjectFactoryI.pm \
	blib/lib/Bio/Factory/ObjectFactoryI.pm \
	lib/Bio/Factory/SeqAnalysisParserFactory.pm \
	blib/lib/Bio/Factory/SeqAnalysisParserFactory.pm \
	lib/Bio/Factory/SeqAnalysisParserFactoryI.pm \
	blib/lib/Bio/Factory/SeqAnalysisParserFactoryI.pm \
	lib/Bio/Factory/SequenceFactoryI.pm \
	blib/lib/Bio/Factory/SequenceFactoryI.pm \
	lib/Bio/Factory/SequenceProcessorI.pm \
	blib/lib/Bio/Factory/SequenceProcessorI.pm \
	lib/Bio/Factory/SequenceStreamI.pm \
	blib/lib/Bio/Factory/SequenceStreamI.pm \
	lib/Bio/Factory/TreeFactoryI.pm \
	blib/lib/Bio/Factory/TreeFactoryI.pm \
	lib/Bio/FeatureHolderI.pm \
	blib/lib/Bio/FeatureHolderI.pm \
	lib/Bio/HandlerBaseI.pm \
	blib/lib/Bio/HandlerBaseI.pm \
	lib/Bio/IdCollectionI.pm \
	blib/lib/Bio/IdCollectionI.pm \
	lib/Bio/IdentifiableI.pm \
	blib/lib/Bio/IdentifiableI.pm \
	lib/Bio/Index/Abstract.pm \
	blib/lib/Bio/Index/Abstract.pm \
	lib/Bio/Index/AbstractSeq.pm \
	blib/lib/Bio/Index/AbstractSeq.pm \
	lib/Bio/Index/Blast.pm \
	blib/lib/Bio/Index/Blast.pm \
	lib/Bio/Index/BlastTable.pm \
	blib/lib/Bio/Index/BlastTable.pm \
	lib/Bio/Index/EMBL.pm \
	blib/lib/Bio/Index/EMBL.pm \
	lib/Bio/Index/Fasta.pm \
	blib/lib/Bio/Index/Fasta.pm \
	lib/Bio/Index/Fastq.pm \
	blib/lib/Bio/Index/Fastq.pm \
	lib/Bio/Index/GenBank.pm \
	blib/lib/Bio/Index/GenBank.pm \
	lib/Bio/Index/Qual.pm \
	blib/lib/Bio/Index/Qual.pm \
	lib/Bio/Index/SwissPfam.pm \
	blib/lib/Bio/Index/SwissPfam.pm \
	lib/Bio/Index/Swissprot.pm \
	blib/lib/Bio/Index/Swissprot.pm \
	lib/Bio/LocatableSeq.pm \
	blib/lib/Bio/LocatableSeq.pm \
	lib/Bio/Location/Atomic.pm \
	blib/lib/Bio/Location/Atomic.pm \
	lib/Bio/Location/AvWithinCoordPolicy.pm \
	blib/lib/Bio/Location/AvWithinCoordPolicy.pm \
	lib/Bio/Location/CoordinatePolicyI.pm \
	blib/lib/Bio/Location/CoordinatePolicyI.pm \
	lib/Bio/Location/Fuzzy.pm \
	blib/lib/Bio/Location/Fuzzy.pm \
	lib/Bio/Location/FuzzyLocationI.pm \
	blib/lib/Bio/Location/FuzzyLocationI.pm \
	lib/Bio/Location/NarrowestCoordPolicy.pm \
	blib/lib/Bio/Location/NarrowestCoordPolicy.pm \
	lib/Bio/Location/Simple.pm \
	blib/lib/Bio/Location/Simple.pm \
	lib/Bio/Location/Split.pm \
	blib/lib/Bio/Location/Split.pm \
	lib/Bio/Location/SplitLocationI.pm \
	blib/lib/Bio/Location/SplitLocationI.pm \
	lib/Bio/Location/WidestCoordPolicy.pm \
	blib/lib/Bio/Location/WidestCoordPolicy.pm \
	lib/Bio/LocationI.pm \
	blib/lib/Bio/LocationI.pm \
	lib/Bio/Matrix/Generic.pm \
	blib/lib/Bio/Matrix/Generic.pm \
	lib/Bio/Matrix/IO.pm \
	blib/lib/Bio/Matrix/IO.pm \
	lib/Bio/Matrix/IO/mlagan.pm \
	blib/lib/Bio/Matrix/IO/mlagan.pm \
	lib/Bio/Matrix/IO/phylip.pm \
	blib/lib/Bio/Matrix/IO/phylip.pm \
	lib/Bio/Matrix/IO/scoring.pm \
	blib/lib/Bio/Matrix/IO/scoring.pm \
	lib/Bio/Matrix/MatrixI.pm \
	blib/lib/Bio/Matrix/MatrixI.pm \
	lib/Bio/Matrix/Mlagan.pm \
	blib/lib/Bio/Matrix/Mlagan.pm \
	lib/Bio/Matrix/PSM/IO.pm \
	blib/lib/Bio/Matrix/PSM/IO.pm \
	lib/Bio/Matrix/PSM/IO/mast.pm \
	blib/lib/Bio/Matrix/PSM/IO/mast.pm \
	lib/Bio/Matrix/PSM/IO/masta.pm \
	blib/lib/Bio/Matrix/PSM/IO/masta.pm \
	lib/Bio/Matrix/PSM/IO/meme.pm \
	blib/lib/Bio/Matrix/PSM/IO/meme.pm \
	lib/Bio/Matrix/PSM/IO/psiblast.pm \
	blib/lib/Bio/Matrix/PSM/IO/psiblast.pm \
	lib/Bio/Matrix/PSM/IO/transfac.pm \
	blib/lib/Bio/Matrix/PSM/IO/transfac.pm \
	lib/Bio/Matrix/PSM/InstanceSite.pm \
	blib/lib/Bio/Matrix/PSM/InstanceSite.pm \
	lib/Bio/Matrix/PSM/InstanceSiteI.pm \
	blib/lib/Bio/Matrix/PSM/InstanceSiteI.pm \
	lib/Bio/Matrix/PSM/ProtMatrix.pm \
	blib/lib/Bio/Matrix/PSM/ProtMatrix.pm \
	lib/Bio/Matrix/PSM/ProtPsm.pm \
	blib/lib/Bio/Matrix/PSM/ProtPsm.pm \
	lib/Bio/Matrix/PSM/Psm.pm \
	blib/lib/Bio/Matrix/PSM/Psm.pm \
	lib/Bio/Matrix/PSM/PsmHeader.pm \
	blib/lib/Bio/Matrix/PSM/PsmHeader.pm \
	lib/Bio/Matrix/PSM/PsmHeaderI.pm \
	blib/lib/Bio/Matrix/PSM/PsmHeaderI.pm \
	lib/Bio/Matrix/PSM/PsmI.pm \
	blib/lib/Bio/Matrix/PSM/PsmI.pm \
	lib/Bio/Matrix/PSM/SiteMatrix.pm \
	blib/lib/Bio/Matrix/PSM/SiteMatrix.pm \
	lib/Bio/Matrix/PSM/SiteMatrixI.pm \
	blib/lib/Bio/Matrix/PSM/SiteMatrixI.pm \
	lib/Bio/Matrix/PhylipDist.pm \
	blib/lib/Bio/Matrix/PhylipDist.pm \
	lib/Bio/Matrix/Scoring.pm \
	blib/lib/Bio/Matrix/Scoring.pm \
	lib/Bio/Ontology/DocumentRegistry.pm \
	blib/lib/Bio/Ontology/DocumentRegistry.pm \
	lib/Bio/Ontology/GOterm.pm \
	blib/lib/Bio/Ontology/GOterm.pm \
	lib/Bio/Ontology/InterProTerm.pm \
	blib/lib/Bio/Ontology/InterProTerm.pm \
	lib/Bio/Ontology/OBOEngine.pm \
	blib/lib/Bio/Ontology/OBOEngine.pm \
	lib/Bio/Ontology/OBOterm.pm \
	blib/lib/Bio/Ontology/OBOterm.pm \
	lib/Bio/Ontology/Ontology.pm \
	blib/lib/Bio/Ontology/Ontology.pm \
	lib/Bio/Ontology/OntologyEngineI.pm \
	blib/lib/Bio/Ontology/OntologyEngineI.pm \
	lib/Bio/Ontology/OntologyI.pm \
	blib/lib/Bio/Ontology/OntologyI.pm \
	lib/Bio/Ontology/OntologyStore.pm \
	blib/lib/Bio/Ontology/OntologyStore.pm \
	lib/Bio/Ontology/Path.pm \
	blib/lib/Bio/Ontology/Path.pm \
	lib/Bio/Ontology/PathI.pm \
	blib/lib/Bio/Ontology/PathI.pm \
	lib/Bio/Ontology/Relationship.pm \
	blib/lib/Bio/Ontology/Relationship.pm \
	lib/Bio/Ontology/RelationshipFactory.pm \
	blib/lib/Bio/Ontology/RelationshipFactory.pm \
	lib/Bio/Ontology/RelationshipI.pm \
	blib/lib/Bio/Ontology/RelationshipI.pm \
	lib/Bio/Ontology/RelationshipType.pm \
	blib/lib/Bio/Ontology/RelationshipType.pm \
	lib/Bio/Ontology/SimpleGOEngine/GraphAdaptor.pm \
	blib/lib/Bio/Ontology/SimpleGOEngine/GraphAdaptor.pm \
	lib/Bio/Ontology/SimpleOntologyEngine.pm \
	blib/lib/Bio/Ontology/SimpleOntologyEngine.pm \
	lib/Bio/Ontology/Term.pm \
	blib/lib/Bio/Ontology/Term.pm \
	lib/Bio/Ontology/TermFactory.pm \
	blib/lib/Bio/Ontology/TermFactory.pm \
	lib/Bio/Ontology/TermI.pm \
	blib/lib/Bio/Ontology/TermI.pm \
	lib/Bio/OntologyIO.pm \
	blib/lib/Bio/OntologyIO.pm \
	lib/Bio/OntologyIO/Handlers/BaseSAXHandler.pm \
	blib/lib/Bio/OntologyIO/Handlers/BaseSAXHandler.pm \
	lib/Bio/OntologyIO/Handlers/InterProHandler.pm \
	blib/lib/Bio/OntologyIO/Handlers/InterProHandler.pm \
	lib/Bio/OntologyIO/Handlers/InterPro_BioSQL_Handler.pm \
	blib/lib/Bio/OntologyIO/Handlers/InterPro_BioSQL_Handler.pm \
	lib/Bio/OntologyIO/InterProParser.pm \
	blib/lib/Bio/OntologyIO/InterProParser.pm \
	lib/Bio/OntologyIO/dagflat.pm \
	blib/lib/Bio/OntologyIO/dagflat.pm \
	lib/Bio/OntologyIO/goflat.pm \
	blib/lib/Bio/OntologyIO/goflat.pm \
	lib/Bio/OntologyIO/obo.pm \
	blib/lib/Bio/OntologyIO/obo.pm \
	lib/Bio/OntologyIO/simplehierarchy.pm \
	blib/lib/Bio/OntologyIO/simplehierarchy.pm \
	lib/Bio/OntologyIO/soflat.pm \
	blib/lib/Bio/OntologyIO/soflat.pm \
	lib/Bio/ParameterBaseI.pm \
	blib/lib/Bio/ParameterBaseI.pm \
	lib/Bio/PrimarySeq.pm \
	blib/lib/Bio/PrimarySeq.pm \
	lib/Bio/PrimarySeqI.pm \
	blib/lib/Bio/PrimarySeqI.pm \
	lib/Bio/PullParserI.pm \
	blib/lib/Bio/PullParserI.pm \
	lib/Bio/Range.pm \
	blib/lib/Bio/Range.pm \
	lib/Bio/RangeI.pm \
	blib/lib/Bio/RangeI.pm \
	lib/Bio/Root/Exception.pm \
	blib/lib/Bio/Root/Exception.pm \
	lib/Bio/Root/HTTPget.pm \
	blib/lib/Bio/Root/HTTPget.pm \
	lib/Bio/Root/IO.pm \
	blib/lib/Bio/Root/IO.pm \
	lib/Bio/Root/Root.pm \
	blib/lib/Bio/Root/Root.pm \
	lib/Bio/Root/RootI.pm \
	blib/lib/Bio/Root/RootI.pm \
	lib/Bio/Root/Storable.pm \
	blib/lib/Bio/Root/Storable.pm \
	lib/Bio/Root/Test.pm \
	blib/lib/Bio/Root/Test.pm \
	lib/Bio/Root/TestObject.pm \
	blib/lib/Bio/Root/TestObject.pm \
	lib/Bio/Root/Utilities.pm \
	blib/lib/Bio/Root/Utilities.pm \
	lib/Bio/Root/Version.pm \
	blib/lib/Bio/Root/Version.pm \
	lib/Bio/Search/BlastStatistics.pm \
	blib/lib/Bio/Search/BlastStatistics.pm \
	lib/Bio/Search/BlastUtils.pm \
	blib/lib/Bio/Search/BlastUtils.pm \
	lib/Bio/Search/DatabaseI.pm \
	blib/lib/Bio/Search/DatabaseI.pm \
	lib/Bio/Search/GenericDatabase.pm \
	blib/lib/Bio/Search/GenericDatabase.pm \
	lib/Bio/Search/GenericStatistics.pm \
	blib/lib/Bio/Search/GenericStatistics.pm \
	lib/Bio/Search/HSP/BlastHSP.pm \
	blib/lib/Bio/Search/HSP/BlastHSP.pm \
	lib/Bio/Search/HSP/BlastPullHSP.pm \
	blib/lib/Bio/Search/HSP/BlastPullHSP.pm \
	lib/Bio/Search/HSP/FastaHSP.pm \
	blib/lib/Bio/Search/HSP/FastaHSP.pm \
	lib/Bio/Search/HSP/GenericHSP.pm \
	blib/lib/Bio/Search/HSP/GenericHSP.pm \
	lib/Bio/Search/HSP/HSPFactory.pm \
	blib/lib/Bio/Search/HSP/HSPFactory.pm \
	lib/Bio/Search/HSP/HSPI.pm \
	blib/lib/Bio/Search/HSP/HSPI.pm \
	lib/Bio/Search/HSP/ModelHSP.pm \
	blib/lib/Bio/Search/HSP/ModelHSP.pm \
	lib/Bio/Search/HSP/PSLHSP.pm \
	blib/lib/Bio/Search/HSP/PSLHSP.pm \
	lib/Bio/Search/HSP/PsiBlastHSP.pm \
	blib/lib/Bio/Search/HSP/PsiBlastHSP.pm \
	lib/Bio/Search/HSP/PullHSPI.pm \
	blib/lib/Bio/Search/HSP/PullHSPI.pm \
	lib/Bio/Search/HSP/WABAHSP.pm \
	blib/lib/Bio/Search/HSP/WABAHSP.pm \
	lib/Bio/Search/Hit/BlastHit.pm \
	blib/lib/Bio/Search/Hit/BlastHit.pm \
	lib/Bio/Search/Hit/BlastPullHit.pm \
	blib/lib/Bio/Search/Hit/BlastPullHit.pm \
	lib/Bio/Search/Hit/Fasta.pm \
	blib/lib/Bio/Search/Hit/Fasta.pm \
	lib/Bio/Search/Hit/GenericHit.pm \
	blib/lib/Bio/Search/Hit/GenericHit.pm \
	lib/Bio/Search/Hit/HitFactory.pm \
	blib/lib/Bio/Search/Hit/HitFactory.pm \
	lib/Bio/Search/Hit/HitI.pm \
	blib/lib/Bio/Search/Hit/HitI.pm \
	lib/Bio/Search/Hit/ModelHit.pm \
	blib/lib/Bio/Search/Hit/ModelHit.pm \
	lib/Bio/Search/Hit/PsiBlastHit.pm \
	blib/lib/Bio/Search/Hit/PsiBlastHit.pm \
	lib/Bio/Search/Hit/PullHitI.pm \
	blib/lib/Bio/Search/Hit/PullHitI.pm \
	lib/Bio/Search/Iteration/GenericIteration.pm \
	blib/lib/Bio/Search/Iteration/GenericIteration.pm \
	lib/Bio/Search/Iteration/IterationI.pm \
	blib/lib/Bio/Search/Iteration/IterationI.pm \
	lib/Bio/Search/Processor.pm \
	blib/lib/Bio/Search/Processor.pm \
	lib/Bio/Search/Result/BlastPullResult.pm \
	blib/lib/Bio/Search/Result/BlastPullResult.pm \
	lib/Bio/Search/Result/BlastResult.pm \
	blib/lib/Bio/Search/Result/BlastResult.pm \
	lib/Bio/Search/Result/CrossMatchResult.pm \
	blib/lib/Bio/Search/Result/CrossMatchResult.pm \
	lib/Bio/Search/Result/GenericResult.pm \
	blib/lib/Bio/Search/Result/GenericResult.pm \
	lib/Bio/Search/Result/INFERNALResult.pm \
	blib/lib/Bio/Search/Result/INFERNALResult.pm \
	lib/Bio/Search/Result/PullResultI.pm \
	blib/lib/Bio/Search/Result/PullResultI.pm \
	lib/Bio/Search/Result/ResultFactory.pm \
	blib/lib/Bio/Search/Result/ResultFactory.pm \
	lib/Bio/Search/Result/ResultI.pm \
	blib/lib/Bio/Search/Result/ResultI.pm \
	lib/Bio/Search/Result/WABAResult.pm \
	blib/lib/Bio/Search/Result/WABAResult.pm \
	lib/Bio/Search/SearchUtils.pm \
	blib/lib/Bio/Search/SearchUtils.pm \
	lib/Bio/Search/StatisticsI.pm \
	blib/lib/Bio/Search/StatisticsI.pm \
	lib/Bio/Search/Tiling/MapTileUtils.pm \
	blib/lib/Bio/Search/Tiling/MapTileUtils.pm \
	lib/Bio/Search/Tiling/MapTiling.pm \
	blib/lib/Bio/Search/Tiling/MapTiling.pm \
	lib/Bio/Search/Tiling/TilingI.pm \
	blib/lib/Bio/Search/Tiling/TilingI.pm \
	lib/Bio/SearchIO.pm \
	blib/lib/Bio/SearchIO.pm \
	lib/Bio/SearchIO/EventHandlerI.pm \
	blib/lib/Bio/SearchIO/EventHandlerI.pm \
	lib/Bio/SearchIO/FastHitEventBuilder.pm \
	blib/lib/Bio/SearchIO/FastHitEventBuilder.pm \
	lib/Bio/SearchIO/IteratedSearchResultEventBuilder.pm \
	blib/lib/Bio/SearchIO/IteratedSearchResultEventBuilder.pm \
	lib/Bio/SearchIO/SearchResultEventBuilder.pm \
	blib/lib/Bio/SearchIO/SearchResultEventBuilder.pm \
	lib/Bio/SearchIO/SearchWriterI.pm \
	blib/lib/Bio/SearchIO/SearchWriterI.pm \
	lib/Bio/SearchIO/Writer/GbrowseGFF.pm \
	blib/lib/Bio/SearchIO/Writer/GbrowseGFF.pm \
	lib/Bio/SearchIO/Writer/HSPTableWriter.pm \
	blib/lib/Bio/SearchIO/Writer/HSPTableWriter.pm \
	lib/Bio/SearchIO/Writer/HTMLResultWriter.pm \
	blib/lib/Bio/SearchIO/Writer/HTMLResultWriter.pm \
	lib/Bio/SearchIO/Writer/HitTableWriter.pm \
	blib/lib/Bio/SearchIO/Writer/HitTableWriter.pm \
	lib/Bio/SearchIO/Writer/ResultTableWriter.pm \
	blib/lib/Bio/SearchIO/Writer/ResultTableWriter.pm \
	lib/Bio/SearchIO/Writer/TextResultWriter.pm \
	blib/lib/Bio/SearchIO/Writer/TextResultWriter.pm \
	lib/Bio/SearchIO/axt.pm \
	blib/lib/Bio/SearchIO/axt.pm \
	lib/Bio/SearchIO/blast.pm \
	blib/lib/Bio/SearchIO/blast.pm \
	lib/Bio/SearchIO/blast_pull.pm \
	blib/lib/Bio/SearchIO/blast_pull.pm \
	lib/Bio/SearchIO/blasttable.pm \
	blib/lib/Bio/SearchIO/blasttable.pm \
	lib/Bio/SearchIO/cross_match.pm \
	blib/lib/Bio/SearchIO/cross_match.pm \
	lib/Bio/SearchIO/erpin.pm \
	blib/lib/Bio/SearchIO/erpin.pm \
	lib/Bio/SearchIO/exonerate.pm \
	blib/lib/Bio/SearchIO/exonerate.pm \
	lib/Bio/SearchIO/fasta.pm \
	blib/lib/Bio/SearchIO/fasta.pm \
	lib/Bio/SearchIO/gmap_f9.pm \
	blib/lib/Bio/SearchIO/gmap_f9.pm \
	lib/Bio/SearchIO/infernal.pm \
	blib/lib/Bio/SearchIO/infernal.pm \
	lib/Bio/SearchIO/megablast.pm \
	blib/lib/Bio/SearchIO/megablast.pm \
	lib/Bio/SearchIO/psl.pm \
	blib/lib/Bio/SearchIO/psl.pm \
	lib/Bio/SearchIO/rnamotif.pm \
	blib/lib/Bio/SearchIO/rnamotif.pm \
	lib/Bio/SearchIO/sim4.pm \
	blib/lib/Bio/SearchIO/sim4.pm \
	lib/Bio/SearchIO/waba.pm \
	blib/lib/Bio/SearchIO/waba.pm \
	lib/Bio/SearchIO/wise.pm \
	blib/lib/Bio/SearchIO/wise.pm \
	lib/Bio/Seq.pm \
	blib/lib/Bio/Seq.pm \
	lib/Bio/Seq/BaseSeqProcessor.pm \
	blib/lib/Bio/Seq/BaseSeqProcessor.pm \
	lib/Bio/Seq/EncodedSeq.pm \
	blib/lib/Bio/Seq/EncodedSeq.pm \
	lib/Bio/Seq/LargeLocatableSeq.pm \
	blib/lib/Bio/Seq/LargeLocatableSeq.pm \
	lib/Bio/Seq/LargePrimarySeq.pm \
	blib/lib/Bio/Seq/LargePrimarySeq.pm \
	lib/Bio/Seq/LargeSeq.pm \
	blib/lib/Bio/Seq/LargeSeq.pm \
	lib/Bio/Seq/LargeSeqI.pm \
	blib/lib/Bio/Seq/LargeSeqI.pm \
	lib/Bio/Seq/Meta.pm \
	blib/lib/Bio/Seq/Meta.pm \
	lib/Bio/Seq/Meta/Array.pm \
	blib/lib/Bio/Seq/Meta/Array.pm \
	lib/Bio/Seq/MetaI.pm \
	blib/lib/Bio/Seq/MetaI.pm \
	lib/Bio/Seq/PrimaryQual.pm \
	blib/lib/Bio/Seq/PrimaryQual.pm \
	lib/Bio/Seq/PrimedSeq.pm \
	blib/lib/Bio/Seq/PrimedSeq.pm \
	lib/Bio/Seq/QualI.pm \
	blib/lib/Bio/Seq/QualI.pm \
	lib/Bio/Seq/Quality.pm \
	blib/lib/Bio/Seq/Quality.pm \
	lib/Bio/Seq/RichSeq.pm \
	blib/lib/Bio/Seq/RichSeq.pm \
	lib/Bio/Seq/RichSeqI.pm \
	blib/lib/Bio/Seq/RichSeqI.pm \
	lib/Bio/Seq/SeqBuilder.pm \
	blib/lib/Bio/Seq/SeqBuilder.pm \
	lib/Bio/Seq/SeqFactory.pm \
	blib/lib/Bio/Seq/SeqFactory.pm \
	lib/Bio/Seq/SeqFastaSpeedFactory.pm \
	blib/lib/Bio/Seq/SeqFastaSpeedFactory.pm \
	lib/Bio/Seq/SequenceTrace.pm \
	blib/lib/Bio/Seq/SequenceTrace.pm \
	lib/Bio/Seq/SimulatedRead.pm \
	blib/lib/Bio/Seq/SimulatedRead.pm \
	lib/Bio/Seq/TraceI.pm \
	blib/lib/Bio/Seq/TraceI.pm \
	lib/Bio/SeqAnalysisParserI.pm \
	blib/lib/Bio/SeqAnalysisParserI.pm \
	lib/Bio/SeqFeature/Amplicon.pm \
	blib/lib/Bio/SeqFeature/Amplicon.pm \
	lib/Bio/SeqFeature/AnnotationAdaptor.pm \
	blib/lib/Bio/SeqFeature/AnnotationAdaptor.pm \
	lib/Bio/SeqFeature/Collection.pm \
	blib/lib/Bio/SeqFeature/Collection.pm \
	lib/Bio/SeqFeature/CollectionI.pm \
	blib/lib/Bio/SeqFeature/CollectionI.pm \
	lib/Bio/SeqFeature/Computation.pm \
	blib/lib/Bio/SeqFeature/Computation.pm \
	lib/Bio/SeqFeature/FeaturePair.pm \
	blib/lib/Bio/SeqFeature/FeaturePair.pm \
	lib/Bio/SeqFeature/Gene/Exon.pm \
	blib/lib/Bio/SeqFeature/Gene/Exon.pm \
	lib/Bio/SeqFeature/Gene/ExonI.pm \
	blib/lib/Bio/SeqFeature/Gene/ExonI.pm \
	lib/Bio/SeqFeature/Gene/GeneStructure.pm \
	blib/lib/Bio/SeqFeature/Gene/GeneStructure.pm \
	lib/Bio/SeqFeature/Gene/GeneStructureI.pm \
	blib/lib/Bio/SeqFeature/Gene/GeneStructureI.pm \
	lib/Bio/SeqFeature/Gene/Intron.pm \
	blib/lib/Bio/SeqFeature/Gene/Intron.pm \
	lib/Bio/SeqFeature/Gene/NC_Feature.pm \
	blib/lib/Bio/SeqFeature/Gene/NC_Feature.pm \
	lib/Bio/SeqFeature/Gene/Poly_A_site.pm \
	blib/lib/Bio/SeqFeature/Gene/Poly_A_site.pm \
	lib/Bio/SeqFeature/Gene/Promoter.pm \
	blib/lib/Bio/SeqFeature/Gene/Promoter.pm \
	lib/Bio/SeqFeature/Gene/Transcript.pm \
	blib/lib/Bio/SeqFeature/Gene/Transcript.pm \
	lib/Bio/SeqFeature/Gene/TranscriptI.pm \
	blib/lib/Bio/SeqFeature/Gene/TranscriptI.pm \
	lib/Bio/SeqFeature/Gene/UTR.pm \
	blib/lib/Bio/SeqFeature/Gene/UTR.pm \
	lib/Bio/SeqFeature/Generic.pm \
	blib/lib/Bio/SeqFeature/Generic.pm \
	lib/Bio/SeqFeature/Lite.pm \
	blib/lib/Bio/SeqFeature/Lite.pm \
	lib/Bio/SeqFeature/PositionProxy.pm \
	blib/lib/Bio/SeqFeature/PositionProxy.pm \
	lib/Bio/SeqFeature/Primer.pm \
	blib/lib/Bio/SeqFeature/Primer.pm \
	lib/Bio/SeqFeature/Similarity.pm \
	blib/lib/Bio/SeqFeature/Similarity.pm \
	lib/Bio/SeqFeature/SimilarityPair.pm \
	blib/lib/Bio/SeqFeature/SimilarityPair.pm \
	lib/Bio/SeqFeature/SubSeq.pm \
	blib/lib/Bio/SeqFeature/SubSeq.pm \
	lib/Bio/SeqFeature/Tools/FeatureNamer.pm \
	blib/lib/Bio/SeqFeature/Tools/FeatureNamer.pm \
	lib/Bio/SeqFeature/Tools/IDHandler.pm \
	blib/lib/Bio/SeqFeature/Tools/IDHandler.pm \
	lib/Bio/SeqFeature/Tools/TypeMapper.pm \
	blib/lib/Bio/SeqFeature/Tools/TypeMapper.pm \
	lib/Bio/SeqFeature/Tools/Unflattener.pm \
	blib/lib/Bio/SeqFeature/Tools/Unflattener.pm \
	lib/Bio/SeqFeature/TypedSeqFeatureI.pm \
	blib/lib/Bio/SeqFeature/TypedSeqFeatureI.pm \
	lib/Bio/SeqFeatureI.pm \
	blib/lib/Bio/SeqFeatureI.pm \
	lib/Bio/SeqI.pm \
	blib/lib/Bio/SeqI.pm \
	lib/Bio/SeqIO.pm \
	blib/lib/Bio/SeqIO.pm \
	lib/Bio/SeqIO/FTHelper.pm \
	blib/lib/Bio/SeqIO/FTHelper.pm \
	lib/Bio/SeqIO/Handler/GenericRichSeqHandler.pm \
	blib/lib/Bio/SeqIO/Handler/GenericRichSeqHandler.pm \
	lib/Bio/SeqIO/MultiFile.pm \
	blib/lib/Bio/SeqIO/MultiFile.pm \
	lib/Bio/SeqIO/ace.pm \
	blib/lib/Bio/SeqIO/ace.pm \
	lib/Bio/SeqIO/asciitree.pm \
	blib/lib/Bio/SeqIO/asciitree.pm \
	lib/Bio/SeqIO/bsml.pm \
	blib/lib/Bio/SeqIO/bsml.pm \
	lib/Bio/SeqIO/bsml_sax.pm \
	blib/lib/Bio/SeqIO/bsml_sax.pm \
	lib/Bio/SeqIO/embl.pm \
	blib/lib/Bio/SeqIO/embl.pm \
	lib/Bio/SeqIO/embldriver.pm \
	blib/lib/Bio/SeqIO/embldriver.pm \
	lib/Bio/SeqIO/fasta.pm \
	blib/lib/Bio/SeqIO/fasta.pm \
	lib/Bio/SeqIO/fastq.pm \
	blib/lib/Bio/SeqIO/fastq.pm \
	lib/Bio/SeqIO/game.pm \
	blib/lib/Bio/SeqIO/game.pm \
	lib/Bio/SeqIO/game/featHandler.pm \
	blib/lib/Bio/SeqIO/game/featHandler.pm \
	lib/Bio/SeqIO/game/gameHandler.pm \
	blib/lib/Bio/SeqIO/game/gameHandler.pm \
	lib/Bio/SeqIO/game/gameSubs.pm \
	blib/lib/Bio/SeqIO/game/gameSubs.pm \
	lib/Bio/SeqIO/game/gameWriter.pm \
	blib/lib/Bio/SeqIO/game/gameWriter.pm \
	lib/Bio/SeqIO/game/seqHandler.pm \
	blib/lib/Bio/SeqIO/game/seqHandler.pm \
	lib/Bio/SeqIO/gbdriver.pm \
	blib/lib/Bio/SeqIO/gbdriver.pm \
	lib/Bio/SeqIO/gbxml.pm \
	blib/lib/Bio/SeqIO/gbxml.pm \
	lib/Bio/SeqIO/gcg.pm \
	blib/lib/Bio/SeqIO/gcg.pm \
	lib/Bio/SeqIO/genbank.pm \
	blib/lib/Bio/SeqIO/genbank.pm \
	lib/Bio/SeqIO/kegg.pm \
	blib/lib/Bio/SeqIO/kegg.pm \
	lib/Bio/SeqIO/largefasta.pm \
	blib/lib/Bio/SeqIO/largefasta.pm \
	lib/Bio/SeqIO/locuslink.pm \
	blib/lib/Bio/SeqIO/locuslink.pm \
	lib/Bio/SeqIO/mbsout.pm \
	blib/lib/Bio/SeqIO/mbsout.pm \
	lib/Bio/SeqIO/metafasta.pm \
	blib/lib/Bio/SeqIO/metafasta.pm \
	lib/Bio/SeqIO/msout.pm \
	blib/lib/Bio/SeqIO/msout.pm \
	lib/Bio/SeqIO/phd.pm \
	blib/lib/Bio/SeqIO/phd.pm \
	lib/Bio/SeqIO/pir.pm \
	blib/lib/Bio/SeqIO/pir.pm \
	lib/Bio/SeqIO/qual.pm \
	blib/lib/Bio/SeqIO/qual.pm \
	lib/Bio/SeqIO/raw.pm \
	blib/lib/Bio/SeqIO/raw.pm \
	lib/Bio/SeqIO/scf.pm \
	blib/lib/Bio/SeqIO/scf.pm \
	lib/Bio/SeqIO/seqxml.pm \
	blib/lib/Bio/SeqIO/seqxml.pm \
	lib/Bio/SeqIO/swiss.pm \
	blib/lib/Bio/SeqIO/swiss.pm \
	lib/Bio/SeqIO/swissdriver.pm \
	blib/lib/Bio/SeqIO/swissdriver.pm \
	lib/Bio/SeqIO/tab.pm \
	blib/lib/Bio/SeqIO/tab.pm \
	lib/Bio/SeqIO/table.pm \
	blib/lib/Bio/SeqIO/table.pm \
	lib/Bio/SeqIO/tigr.pm \
	blib/lib/Bio/SeqIO/tigr.pm \
	lib/Bio/SeqIO/tigrxml.pm \
	blib/lib/Bio/SeqIO/tigrxml.pm \
	lib/Bio/SeqIO/tinyseq.pm \
	blib/lib/Bio/SeqIO/tinyseq.pm \
	lib/Bio/SeqIO/tinyseq/tinyseqHandler.pm \
	blib/lib/Bio/SeqIO/tinyseq/tinyseqHandler.pm \
	lib/Bio/SeqUtils.pm \
	blib/lib/Bio/SeqUtils.pm \
	lib/Bio/SimpleAlign.pm \
	blib/lib/Bio/SimpleAlign.pm \
	lib/Bio/SimpleAnalysisI.pm \
	blib/lib/Bio/SimpleAnalysisI.pm \
	lib/Bio/Species.pm \
	blib/lib/Bio/Species.pm \
	lib/Bio/Taxon.pm \
	blib/lib/Bio/Taxon.pm \
	lib/Bio/Tools/Alignment/Consed.pm \
	blib/lib/Bio/Tools/Alignment/Consed.pm \
	lib/Bio/Tools/Alignment/Trim.pm \
	blib/lib/Bio/Tools/Alignment/Trim.pm \
	lib/Bio/Tools/AmpliconSearch.pm \
	blib/lib/Bio/Tools/AmpliconSearch.pm \
	lib/Bio/Tools/Analysis/SimpleAnalysisBase.pm \
	blib/lib/Bio/Tools/Analysis/SimpleAnalysisBase.pm \
	lib/Bio/Tools/AnalysisResult.pm \
	blib/lib/Bio/Tools/AnalysisResult.pm \
	lib/Bio/Tools/Blat.pm \
	blib/lib/Bio/Tools/Blat.pm \
	lib/Bio/Tools/CodonTable.pm \
	blib/lib/Bio/Tools/CodonTable.pm \
	lib/Bio/Tools/Coil.pm \
	blib/lib/Bio/Tools/Coil.pm \
	lib/Bio/Tools/ECnumber.pm \
	blib/lib/Bio/Tools/ECnumber.pm \
	lib/Bio/Tools/EMBOSS/Palindrome.pm \
	blib/lib/Bio/Tools/EMBOSS/Palindrome.pm \
	lib/Bio/Tools/EPCR.pm \
	blib/lib/Bio/Tools/EPCR.pm \
	lib/Bio/Tools/ESTScan.pm \
	blib/lib/Bio/Tools/ESTScan.pm \
	lib/Bio/Tools/Eponine.pm \
	blib/lib/Bio/Tools/Eponine.pm \
	lib/Bio/Tools/Est2Genome.pm \
	blib/lib/Bio/Tools/Est2Genome.pm \
	lib/Bio/Tools/Fgenesh.pm \
	blib/lib/Bio/Tools/Fgenesh.pm \
	lib/Bio/Tools/FootPrinter.pm \
	blib/lib/Bio/Tools/FootPrinter.pm \
	lib/Bio/Tools/GFF.pm \
	blib/lib/Bio/Tools/GFF.pm \
	lib/Bio/Tools/Geneid.pm \
	blib/lib/Bio/Tools/Geneid.pm \
	lib/Bio/Tools/Genemark.pm \
	blib/lib/Bio/Tools/Genemark.pm \
	lib/Bio/Tools/Genewise.pm \
	blib/lib/Bio/Tools/Genewise.pm \
	lib/Bio/Tools/Genomewise.pm \
	blib/lib/Bio/Tools/Genomewise.pm \
	lib/Bio/Tools/Genscan.pm \
	blib/lib/Bio/Tools/Genscan.pm \
	lib/Bio/Tools/Glimmer.pm \
	blib/lib/Bio/Tools/Glimmer.pm \
	lib/Bio/Tools/Grail.pm \
	blib/lib/Bio/Tools/Grail.pm \
	lib/Bio/Tools/GuessSeqFormat.pm \
	blib/lib/Bio/Tools/GuessSeqFormat.pm \
	lib/Bio/Tools/IUPAC.pm \
	blib/lib/Bio/Tools/IUPAC.pm \
	lib/Bio/Tools/Lucy.pm \
	blib/lib/Bio/Tools/Lucy.pm \
	lib/Bio/Tools/MZEF.pm \
	blib/lib/Bio/Tools/MZEF.pm \
	lib/Bio/Tools/Match.pm \
	blib/lib/Bio/Tools/Match.pm \
	lib/Bio/Tools/OddCodes.pm \
	blib/lib/Bio/Tools/OddCodes.pm \
	lib/Bio/Tools/Phylo/Gerp.pm \
	blib/lib/Bio/Tools/Phylo/Gerp.pm \
	lib/Bio/Tools/Phylo/Molphy.pm \
	blib/lib/Bio/Tools/Phylo/Molphy.pm \
	lib/Bio/Tools/Phylo/Molphy/Result.pm \
	blib/lib/Bio/Tools/Phylo/Molphy/Result.pm \
	lib/Bio/Tools/Phylo/Phylip/ProtDist.pm \
	blib/lib/Bio/Tools/Phylo/Phylip/ProtDist.pm \
	lib/Bio/Tools/Prediction/Exon.pm \
	blib/lib/Bio/Tools/Prediction/Exon.pm \
	lib/Bio/Tools/Prediction/Gene.pm \
	blib/lib/Bio/Tools/Prediction/Gene.pm \
	lib/Bio/Tools/Primer/Assessor/Base.pm \
	blib/lib/Bio/Tools/Primer/Assessor/Base.pm \
	lib/Bio/Tools/Primer/AssessorI.pm \
	blib/lib/Bio/Tools/Primer/AssessorI.pm \
	lib/Bio/Tools/Primer/Feature.pm \
	blib/lib/Bio/Tools/Primer/Feature.pm \
	lib/Bio/Tools/Primer/Pair.pm \
	blib/lib/Bio/Tools/Primer/Pair.pm \
	lib/Bio/Tools/Primer3.pm \
	blib/lib/Bio/Tools/Primer3.pm \
	lib/Bio/Tools/Prints.pm \
	blib/lib/Bio/Tools/Prints.pm \
	lib/Bio/Tools/Profile.pm \
	blib/lib/Bio/Tools/Profile.pm \
	lib/Bio/Tools/Promoterwise.pm \
	blib/lib/Bio/Tools/Promoterwise.pm \
	lib/Bio/Tools/PrositeScan.pm \
	blib/lib/Bio/Tools/PrositeScan.pm \
	lib/Bio/Tools/Pseudowise.pm \
	blib/lib/Bio/Tools/Pseudowise.pm \
	lib/Bio/Tools/QRNA.pm \
	blib/lib/Bio/Tools/QRNA.pm \
	lib/Bio/Tools/RandomDistFunctions.pm \
	blib/lib/Bio/Tools/RandomDistFunctions.pm \
	lib/Bio/Tools/RepeatMasker.pm \
	blib/lib/Bio/Tools/RepeatMasker.pm \
	lib/Bio/Tools/Run/Analysis.pm \
	blib/lib/Bio/Tools/Run/Analysis.pm \
	lib/Bio/Tools/Run/AnalysisFactory.pm \
	blib/lib/Bio/Tools/Run/AnalysisFactory.pm \
	lib/Bio/Tools/Run/GenericParameters.pm \
	blib/lib/Bio/Tools/Run/GenericParameters.pm \
	lib/Bio/Tools/Run/ParametersI.pm \
	blib/lib/Bio/Tools/Run/ParametersI.pm \
	lib/Bio/Tools/Run/Phylo/PhyloBase.pm \
	blib/lib/Bio/Tools/Run/Phylo/PhyloBase.pm \
	lib/Bio/Tools/Run/WrapperBase.pm \
	blib/lib/Bio/Tools/Run/WrapperBase.pm \
	lib/Bio/Tools/Run/WrapperBase/CommandExts.pm \
	blib/lib/Bio/Tools/Run/WrapperBase/CommandExts.pm \
	lib/Bio/Tools/Seg.pm \
	blib/lib/Bio/Tools/Seg.pm \
	lib/Bio/Tools/SeqPattern.pm \
	blib/lib/Bio/Tools/SeqPattern.pm \
	lib/Bio/Tools/SeqPattern/Backtranslate.pm \
	blib/lib/Bio/Tools/SeqPattern/Backtranslate.pm \
	lib/Bio/Tools/SeqStats.pm \
	blib/lib/Bio/Tools/SeqStats.pm \
	lib/Bio/Tools/SeqWords.pm \
	blib/lib/Bio/Tools/SeqWords.pm \
	lib/Bio/Tools/Sigcleave.pm \
	blib/lib/Bio/Tools/Sigcleave.pm \
	lib/Bio/Tools/Signalp.pm \
	blib/lib/Bio/Tools/Signalp.pm \
	lib/Bio/Tools/Signalp/ExtendedSignalp.pm \
	blib/lib/Bio/Tools/Signalp/ExtendedSignalp.pm \
	lib/Bio/Tools/Sim4/Exon.pm \
	blib/lib/Bio/Tools/Sim4/Exon.pm \
	lib/Bio/Tools/Sim4/Results.pm \
	blib/lib/Bio/Tools/Sim4/Results.pm \
	lib/Bio/Tools/Spidey/Exon.pm \
	blib/lib/Bio/Tools/Spidey/Exon.pm \
	lib/Bio/Tools/Spidey/Results.pm \
	blib/lib/Bio/Tools/Spidey/Results.pm \
	lib/Bio/Tools/TandemRepeatsFinder.pm \
	blib/lib/Bio/Tools/TandemRepeatsFinder.pm \
	lib/Bio/Tools/TargetP.pm \
	blib/lib/Bio/Tools/TargetP.pm \
	lib/Bio/Tools/Tmhmm.pm \
	blib/lib/Bio/Tools/Tmhmm.pm \
	lib/Bio/Tools/ipcress.pm \
	blib/lib/Bio/Tools/ipcress.pm \
	lib/Bio/Tools/isPcr.pm \
	blib/lib/Bio/Tools/isPcr.pm \
	lib/Bio/Tools/pICalculator.pm \
	blib/lib/Bio/Tools/pICalculator.pm \
	lib/Bio/Tools/tRNAscanSE.pm \
	blib/lib/Bio/Tools/tRNAscanSE.pm \
	lib/Bio/Tree/AnnotatableNode.pm \
	blib/lib/Bio/Tree/AnnotatableNode.pm \
	lib/Bio/Tree/Compatible.pm \
	blib/lib/Bio/Tree/Compatible.pm \
	lib/Bio/Tree/DistanceFactory.pm \
	blib/lib/Bio/Tree/DistanceFactory.pm \
	lib/Bio/Tree/Node.pm \
	blib/lib/Bio/Tree/Node.pm \
	lib/Bio/Tree/NodeI.pm \
	blib/lib/Bio/Tree/NodeI.pm \
	lib/Bio/Tree/NodeNHX.pm \
	blib/lib/Bio/Tree/NodeNHX.pm \
	lib/Bio/Tree/RandomFactory.pm \
	blib/lib/Bio/Tree/RandomFactory.pm \
	lib/Bio/Tree/Statistics.pm \
	blib/lib/Bio/Tree/Statistics.pm \
	lib/Bio/Tree/Tree.pm \
	blib/lib/Bio/Tree/Tree.pm \
	lib/Bio/Tree/TreeFunctionsI.pm \
	blib/lib/Bio/Tree/TreeFunctionsI.pm \
	lib/Bio/Tree/TreeI.pm \
	blib/lib/Bio/Tree/TreeI.pm \
	lib/Bio/TreeIO.pm \
	blib/lib/Bio/TreeIO.pm \
	lib/Bio/TreeIO/NewickParser.pm \
	blib/lib/Bio/TreeIO/NewickParser.pm \
	lib/Bio/TreeIO/TreeEventBuilder.pm \
	blib/lib/Bio/TreeIO/TreeEventBuilder.pm \
	lib/Bio/TreeIO/cluster.pm \
	blib/lib/Bio/TreeIO/cluster.pm \
	lib/Bio/TreeIO/lintree.pm \
	blib/lib/Bio/TreeIO/lintree.pm \
	lib/Bio/TreeIO/newick.pm \
	blib/lib/Bio/TreeIO/newick.pm \
	lib/Bio/TreeIO/nexus.pm \
	blib/lib/Bio/TreeIO/nexus.pm \
	lib/Bio/TreeIO/nhx.pm \
	blib/lib/Bio/TreeIO/nhx.pm \
	lib/Bio/TreeIO/pag.pm \
	blib/lib/Bio/TreeIO/pag.pm \
	lib/Bio/TreeIO/phyloxml.pm \
	blib/lib/Bio/TreeIO/phyloxml.pm \
	lib/Bio/TreeIO/tabtree.pm \
	blib/lib/Bio/TreeIO/tabtree.pm \
	lib/Bio/UpdateableSeqI.pm \
	blib/lib/Bio/UpdateableSeqI.pm \
	lib/Bio/WebAgent.pm \
	blib/lib/Bio/WebAgent.pm \
	lib/BioPerl.pm \
	blib/lib/BioPerl.pm


# --- MakeMaker platform_constants section:
MM_Unix_VERSION = 7.0401
PERL_MALLOC_DEF = -DPERL_EXTMALLOC_DEF -Dmalloc=Perl_malloc -Dfree=Perl_mfree -Drealloc=Perl_realloc -Dcalloc=Perl_calloc


# --- MakeMaker tool_autosplit section:
# Usage: $(AUTOSPLITFILE) FileToSplit AutoDirToSplitInto
AUTOSPLITFILE = $(ABSPERLRUN)  -e 'use AutoSplit;  autosplit($$$$ARGV[0], $$$$ARGV[1], 0, 1, 1)' --



# --- MakeMaker tool_xsubpp section:


# --- MakeMaker tools_other section:
SHELL = /bin/sh
CHMOD = chmod
CP = cp
MV = mv
NOOP = $(TRUE)
NOECHO = @
RM_F = rm -f
RM_RF = rm -rf
TEST_F = test -f
TOUCH = touch
UMASK_NULL = umask 0
DEV_NULL = > /dev/null 2>&1
MKPATH = $(ABSPERLRUN) -MExtUtils::Command -e 'mkpath' --
EQUALIZE_TIMESTAMP = $(ABSPERLRUN) -MExtUtils::Command -e 'eqtime' --
FALSE = false
TRUE = true
ECHO = echo
ECHO_N = echo -n
UNINST = 0
VERBINST = 0
MOD_INSTALL = $(ABSPERLRUN) -MExtUtils::Install -e 'install([ from_to => {@ARGV}, verbose => '\''$(VERBINST)'\'', uninstall_shadows => '\''$(UNINST)'\'', dir_mode => '\''$(PERM_DIR)'\'' ]);' --
DOC_INSTALL = $(ABSPERLRUN) -MExtUtils::Command::MM -e 'perllocal_install' --
UNINSTALL = $(ABSPERLRUN) -MExtUtils::Command::MM -e 'uninstall' --
WARN_IF_OLD_PACKLIST = $(ABSPERLRUN) -MExtUtils::Command::MM -e 'warn_if_old_packlist' --
MACROSTART = 
MACROEND = 
USEMAKEFILE = -f
FIXIN = $(ABSPERLRUN) -MExtUtils::MY -e 'MY->fixin(shift)' --
CP_NONEMPTY = $(ABSPERLRUN) -MExtUtils::Command::MM -e 'cp_nonempty' --


# --- MakeMaker makemakerdflt section:
makemakerdflt : all
	$(NOECHO) $(NOOP)


# --- MakeMaker dist section:
TAR = tar
TARFLAGS = cvf
ZIP = zip
ZIPFLAGS = -r
COMPRESS = gzip --best
SUFFIX = .gz
SHAR = shar
PREOP = $(NOECHO) $(NOOP)
POSTOP = $(NOECHO) $(NOOP)
TO_UNIX = $(NOECHO) $(NOOP)
CI = ci -u
RCS_LABEL = rcs -Nv$(VERSION_SYM): -q
DIST_CP = best
DIST_DEFAULT = tardist
DISTNAME = BioPerl
DISTVNAME = BioPerl-1.7.8


# --- MakeMaker macro section:


# --- MakeMaker depend section:


# --- MakeMaker cflags section:


# --- MakeMaker const_loadlibs section:


# --- MakeMaker const_cccmd section:


# --- MakeMaker post_constants section:


# --- MakeMaker pasthru section:

PASTHRU = LIBPERL_A="$(LIBPERL_A)"\
	LINKTYPE="$(LINKTYPE)"\
	PREFIX="$(PREFIX)"\
	INSTALL_BASE="$(INSTALL_BASE)"


# --- MakeMaker special_targets section:
.SUFFIXES : .xs .c .C .cpp .i .s .cxx .cc $(OBJ_EXT)

.PHONY: all config static dynamic test linkext manifest blibdirs clean realclean disttest distdir



# --- MakeMaker c_o section:


# --- MakeMaker xs_c section:


# --- MakeMaker xs_o section:


# --- MakeMaker top_targets section:
all :: pure_all manifypods
	$(NOECHO) $(NOOP)


pure_all :: config pm_to_blib subdirs linkext
	$(NOECHO) $(NOOP)

subdirs :: $(MYEXTLIB)
	$(NOECHO) $(NOOP)

config :: $(FIRST_MAKEFILE) blibdirs
	$(NOECHO) $(NOOP)

help :
	perldoc ExtUtils::MakeMaker


# --- MakeMaker blibdirs section:
blibdirs : $(INST_LIBDIR)$(DFSEP).exists $(INST_ARCHLIB)$(DFSEP).exists $(INST_AUTODIR)$(DFSEP).exists $(INST_ARCHAUTODIR)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists $(INST_SCRIPT)$(DFSEP).exists $(INST_MAN1DIR)$(DFSEP).exists $(INST_MAN3DIR)$(DFSEP).exists
	$(NOECHO) $(NOOP)

# Backwards compat with 6.18 through 6.25
blibdirs.ts : blibdirs
	$(NOECHO) $(NOOP)

$(INST_LIBDIR)$(DFSEP).exists :: Makefile.PL
	$(NOECHO) $(MKPATH) $(INST_LIBDIR)
	$(NOECHO) $(CHMOD) $(PERM_DIR) $(INST_LIBDIR)
	$(NOECHO) $(TOUCH) $(INST_LIBDIR)$(DFSEP).exists

$(INST_ARCHLIB)$(DFSEP).exists :: Makefile.PL
	$(NOECHO) $(MKPATH) $(INST_ARCHLIB)
	$(NOECHO) $(CHMOD) $(PERM_DIR) $(INST_ARCHLIB)
	$(NOECHO) $(TOUCH) $(INST_ARCHLIB)$(DFSEP).exists

$(INST_AUTODIR)$(DFSEP).exists :: Makefile.PL
	$(NOECHO) $(MKPATH) $(INST_AUTODIR)
	$(NOECHO) $(CHMOD) $(PERM_DIR) $(INST_AUTODIR)
	$(NOECHO) $(TOUCH) $(INST_AUTODIR)$(DFSEP).exists

$(INST_ARCHAUTODIR)$(DFSEP).exists :: Makefile.PL
	$(NOECHO) $(MKPATH) $(INST_ARCHAUTODIR)
	$(NOECHO) $(CHMOD) $(PERM_DIR) $(INST_ARCHAUTODIR)
	$(NOECHO) $(TOUCH) $(INST_ARCHAUTODIR)$(DFSEP).exists

$(INST_BIN)$(DFSEP).exists :: Makefile.PL
	$(NOECHO) $(MKPATH) $(INST_BIN)
	$(NOECHO) $(CHMOD) $(PERM_DIR) $(INST_BIN)
	$(NOECHO) $(TOUCH) $(INST_BIN)$(DFSEP).exists

$(INST_SCRIPT)$(DFSEP).exists :: Makefile.PL
	$(NOECHO) $(MKPATH) $(INST_SCRIPT)
	$(NOECHO) $(CHMOD) $(PERM_DIR) $(INST_SCRIPT)
	$(NOECHO) $(TOUCH) $(INST_SCRIPT)$(DFSEP).exists

$(INST_MAN1DIR)$(DFSEP).exists :: Makefile.PL
	$(NOECHO) $(MKPATH) $(INST_MAN1DIR)
	$(NOECHO) $(CHMOD) $(PERM_DIR) $(INST_MAN1DIR)
	$(NOECHO) $(TOUCH) $(INST_MAN1DIR)$(DFSEP).exists

$(INST_MAN3DIR)$(DFSEP).exists :: Makefile.PL
	$(NOECHO) $(MKPATH) $(INST_MAN3DIR)
	$(NOECHO) $(CHMOD) $(PERM_DIR) $(INST_MAN3DIR)
	$(NOECHO) $(TOUCH) $(INST_MAN3DIR)$(DFSEP).exists



# --- MakeMaker linkext section:

linkext :: $(LINKTYPE)
	$(NOECHO) $(NOOP)


# --- MakeMaker dlsyms section:


# --- MakeMaker dynamic_bs section:

BOOTSTRAP =


# --- MakeMaker dynamic section:

dynamic :: $(FIRST_MAKEFILE) $(BOOTSTRAP) $(INST_DYNAMIC)
	$(NOECHO) $(NOOP)


# --- MakeMaker dynamic_lib section:


# --- MakeMaker static section:

## $(INST_PM) has been moved to the all: target.
## It remains here for awhile to allow for old usage: "make static"
static :: $(FIRST_MAKEFILE) $(INST_STATIC)
	$(NOECHO) $(NOOP)


# --- MakeMaker static_lib section:


# --- MakeMaker manifypods section:

POD2MAN_EXE = $(PERLRUN) "-MExtUtils::Command::MM" -e pod2man "--"
POD2MAN = $(POD2MAN_EXE)


manifypods : pure_all  \
	bin/bp_aacomp \
	bin/bp_bioflat_index \
	bin/bp_biogetseq \
	bin/bp_dbsplit \
	bin/bp_extract_feature_seq \
	bin/bp_fastam9_to_table \
	bin/bp_fetch \
	bin/bp_filter_search \
	bin/bp_find-blast-matches \
	bin/bp_gccalc \
	bin/bp_genbank2gff3 \
	bin/bp_index \
	bin/bp_local_taxonomydb_query \
	bin/bp_make_mrna_protein \
	bin/bp_mask_by_search \
	bin/bp_mrtrans \
	bin/bp_mutate \
	bin/bp_nexus2nh \
	bin/bp_nrdb \
	bin/bp_oligo_count \
	bin/bp_process_gadfly \
	bin/bp_process_sgd \
	bin/bp_revtrans-motif \
	bin/bp_search2alnblocks \
	bin/bp_search2gff \
	bin/bp_search2table \
	bin/bp_search2tribe \
	bin/bp_seq_length \
	bin/bp_seqconvert \
	bin/bp_seqcut \
	bin/bp_seqpart \
	bin/bp_seqret \
	bin/bp_seqretsplit \
	bin/bp_split_seq \
	bin/bp_sreformat \
	bin/bp_taxid4species \
	bin/bp_taxonomy2tree \
	bin/bp_translate_seq \
	bin/bp_tree2pag \
	bin/bp_unflatten_seq \
	lib/Bio/Align/AlignI.pm \
	lib/Bio/Align/DNAStatistics.pm \
	lib/Bio/Align/PairwiseStatistics.pm \
	lib/Bio/Align/ProteinStatistics.pm \
	lib/Bio/Align/StatisticsI.pm \
	lib/Bio/Align/Utilities.pm \
	lib/Bio/AlignIO.pm \
	lib/Bio/AlignIO/Handler/GenericAlignHandler.pm \
	lib/Bio/AlignIO/arp.pm \
	lib/Bio/AlignIO/bl2seq.pm \
	lib/Bio/AlignIO/clustalw.pm \
	lib/Bio/AlignIO/emboss.pm \
	lib/Bio/AlignIO/fasta.pm \
	lib/Bio/AlignIO/largemultifasta.pm \
	lib/Bio/AlignIO/maf.pm \
	lib/Bio/AlignIO/mase.pm \
	lib/Bio/AlignIO/mega.pm \
	lib/Bio/AlignIO/meme.pm \
	lib/Bio/AlignIO/metafasta.pm \
	lib/Bio/AlignIO/msf.pm \
	lib/Bio/AlignIO/nexus.pm \
	lib/Bio/AlignIO/pfam.pm \
	lib/Bio/AlignIO/phylip.pm \
	lib/Bio/AlignIO/po.pm \
	lib/Bio/AlignIO/proda.pm \
	lib/Bio/AlignIO/prodom.pm \
	lib/Bio/AlignIO/psi.pm \
	lib/Bio/AlignIO/selex.pm \
	lib/Bio/AlignIO/xmfa.pm \
	lib/Bio/AnalysisI.pm \
	lib/Bio/AnalysisParserI.pm \
	lib/Bio/AnalysisResultI.pm \
	lib/Bio/AnnotatableI.pm \
	lib/Bio/Annotation/AnnotationFactory.pm \
	lib/Bio/Annotation/Collection.pm \
	lib/Bio/Annotation/Comment.pm \
	lib/Bio/Annotation/DBLink.pm \
	lib/Bio/Annotation/OntologyTerm.pm \
	lib/Bio/Annotation/Reference.pm \
	lib/Bio/Annotation/Relation.pm \
	lib/Bio/Annotation/SimpleValue.pm \
	lib/Bio/Annotation/StructuredValue.pm \
	lib/Bio/Annotation/TagTree.pm \
	lib/Bio/Annotation/Target.pm \
	lib/Bio/Annotation/Tree.pm \
	lib/Bio/Annotation/TypeManager.pm \
	lib/Bio/AnnotationCollectionI.pm \
	lib/Bio/AnnotationI.pm \
	lib/Bio/CodonUsage/IO.pm \
	lib/Bio/CodonUsage/Table.pm \
	lib/Bio/DB/DBFetch.pm \
	lib/Bio/DB/Failover.pm \
	lib/Bio/DB/Fasta.pm \
	lib/Bio/DB/FileCache.pm \
	lib/Bio/DB/Flat.pm \
	lib/Bio/DB/Flat/BDB.pm \
	lib/Bio/DB/Flat/BDB/embl.pm \
	lib/Bio/DB/Flat/BDB/fasta.pm \
	lib/Bio/DB/Flat/BDB/genbank.pm \
	lib/Bio/DB/Flat/BDB/swiss.pm \
	lib/Bio/DB/Flat/BinarySearch.pm \
	lib/Bio/DB/GFF/Util/Binning.pm \
	lib/Bio/DB/GFF/Util/Rearrange.pm \
	lib/Bio/DB/GenericWebAgent.pm \
	lib/Bio/DB/InMemoryCache.pm \
	lib/Bio/DB/IndexedBase.pm \
	lib/Bio/DB/LocationI.pm \
	lib/Bio/DB/Qual.pm \
	lib/Bio/DB/Query/WebQuery.pm \
	lib/Bio/DB/QueryI.pm \
	lib/Bio/DB/RandomAccessI.pm \
	lib/Bio/DB/ReferenceI.pm \
	lib/Bio/DB/Registry.pm \
	lib/Bio/DB/SeqI.pm \
	lib/Bio/DB/Taxonomy.pm \
	lib/Bio/DB/Taxonomy/flatfile.pm \
	lib/Bio/DB/Taxonomy/greengenes.pm \
	lib/Bio/DB/Taxonomy/list.pm \
	lib/Bio/DB/Taxonomy/silva.pm \
	lib/Bio/DB/UpdateableSeqI.pm \
	lib/Bio/DB/WebDBSeqI.pm \
	lib/Bio/DBLinkContainerI.pm \
	lib/Bio/Das/FeatureTypeI.pm \
	lib/Bio/Das/SegmentI.pm \
	lib/Bio/DasI.pm \
	lib/Bio/DescribableI.pm \
	lib/Bio/Event/EventGeneratorI.pm \
	lib/Bio/Event/EventHandlerI.pm \
	lib/Bio/Factory/AnalysisI.pm \
	lib/Bio/Factory/ApplicationFactoryI.pm \
	lib/Bio/Factory/DriverFactory.pm \
	lib/Bio/Factory/FTLocationFactory.pm \
	lib/Bio/Factory/LocationFactoryI.pm \
	lib/Bio/Factory/ObjectBuilderI.pm \
	lib/Bio/Factory/ObjectFactory.pm \
	lib/Bio/Factory/ObjectFactoryI.pm \
	lib/Bio/Factory/SeqAnalysisParserFactory.pm \
	lib/Bio/Factory/SeqAnalysisParserFactoryI.pm \
	lib/Bio/Factory/SequenceFactoryI.pm \
	lib/Bio/Factory/SequenceProcessorI.pm \
	lib/Bio/Factory/SequenceStreamI.pm \
	lib/Bio/Factory/TreeFactoryI.pm \
	lib/Bio/FeatureHolderI.pm \
	lib/Bio/HandlerBaseI.pm \
	lib/Bio/IdCollectionI.pm \
	lib/Bio/IdentifiableI.pm \
	lib/Bio/Index/Abstract.pm \
	lib/Bio/Index/AbstractSeq.pm \
	lib/Bio/Index/Blast.pm \
	lib/Bio/Index/BlastTable.pm \
	lib/Bio/Index/EMBL.pm \
	lib/Bio/Index/Fasta.pm \
	lib/Bio/Index/Fastq.pm \
	lib/Bio/Index/GenBank.pm \
	lib/Bio/Index/Qual.pm \
	lib/Bio/Index/SwissPfam.pm \
	lib/Bio/Index/Swissprot.pm \
	lib/Bio/LocatableSeq.pm \
	lib/Bio/Location/Atomic.pm \
	lib/Bio/Location/AvWithinCoordPolicy.pm \
	lib/Bio/Location/CoordinatePolicyI.pm \
	lib/Bio/Location/Fuzzy.pm \
	lib/Bio/Location/FuzzyLocationI.pm \
	lib/Bio/Location/NarrowestCoordPolicy.pm \
	lib/Bio/Location/Simple.pm \
	lib/Bio/Location/Split.pm \
	lib/Bio/Location/SplitLocationI.pm \
	lib/Bio/Location/WidestCoordPolicy.pm \
	lib/Bio/LocationI.pm \
	lib/Bio/Matrix/Generic.pm \
	lib/Bio/Matrix/IO.pm \
	lib/Bio/Matrix/IO/mlagan.pm \
	lib/Bio/Matrix/IO/phylip.pm \
	lib/Bio/Matrix/IO/scoring.pm \
	lib/Bio/Matrix/MatrixI.pm \
	lib/Bio/Matrix/Mlagan.pm \
	lib/Bio/Matrix/PSM/IO.pm \
	lib/Bio/Matrix/PSM/IO/mast.pm \
	lib/Bio/Matrix/PSM/IO/masta.pm \
	lib/Bio/Matrix/PSM/IO/meme.pm \
	lib/Bio/Matrix/PSM/IO/psiblast.pm \
	lib/Bio/Matrix/PSM/IO/transfac.pm \
	lib/Bio/Matrix/PSM/InstanceSite.pm \
	lib/Bio/Matrix/PSM/InstanceSiteI.pm \
	lib/Bio/Matrix/PSM/ProtMatrix.pm \
	lib/Bio/Matrix/PSM/ProtPsm.pm \
	lib/Bio/Matrix/PSM/Psm.pm \
	lib/Bio/Matrix/PSM/PsmHeader.pm \
	lib/Bio/Matrix/PSM/PsmHeaderI.pm \
	lib/Bio/Matrix/PSM/PsmI.pm \
	lib/Bio/Matrix/PSM/SiteMatrix.pm \
	lib/Bio/Matrix/PSM/SiteMatrixI.pm \
	lib/Bio/Matrix/PhylipDist.pm \
	lib/Bio/Matrix/Scoring.pm \
	lib/Bio/Ontology/DocumentRegistry.pm \
	lib/Bio/Ontology/GOterm.pm \
	lib/Bio/Ontology/InterProTerm.pm \
	lib/Bio/Ontology/OBOEngine.pm \
	lib/Bio/Ontology/OBOterm.pm \
	lib/Bio/Ontology/Ontology.pm \
	lib/Bio/Ontology/OntologyEngineI.pm \
	lib/Bio/Ontology/OntologyI.pm \
	lib/Bio/Ontology/OntologyStore.pm \
	lib/Bio/Ontology/Path.pm \
	lib/Bio/Ontology/PathI.pm \
	lib/Bio/Ontology/Relationship.pm \
	lib/Bio/Ontology/RelationshipFactory.pm \
	lib/Bio/Ontology/RelationshipI.pm \
	lib/Bio/Ontology/RelationshipType.pm \
	lib/Bio/Ontology/SimpleGOEngine/GraphAdaptor.pm \
	lib/Bio/Ontology/SimpleOntologyEngine.pm \
	lib/Bio/Ontology/Term.pm \
	lib/Bio/Ontology/TermFactory.pm \
	lib/Bio/Ontology/TermI.pm \
	lib/Bio/OntologyIO.pm \
	lib/Bio/OntologyIO/Handlers/BaseSAXHandler.pm \
	lib/Bio/OntologyIO/Handlers/InterProHandler.pm \
	lib/Bio/OntologyIO/Handlers/InterPro_BioSQL_Handler.pm \
	lib/Bio/OntologyIO/InterProParser.pm \
	lib/Bio/OntologyIO/dagflat.pm \
	lib/Bio/OntologyIO/goflat.pm \
	lib/Bio/OntologyIO/obo.pm \
	lib/Bio/OntologyIO/simplehierarchy.pm \
	lib/Bio/OntologyIO/soflat.pm \
	lib/Bio/ParameterBaseI.pm \
	lib/Bio/PrimarySeq.pm \
	lib/Bio/PrimarySeqI.pm \
	lib/Bio/PullParserI.pm \
	lib/Bio/Range.pm \
	lib/Bio/RangeI.pm \
	lib/Bio/Root/Exception.pm \
	lib/Bio/Root/HTTPget.pm \
	lib/Bio/Root/IO.pm \
	lib/Bio/Root/Root.pm \
	lib/Bio/Root/RootI.pm \
	lib/Bio/Root/Storable.pm \
	lib/Bio/Root/Test.pm \
	lib/Bio/Root/TestObject.pm \
	lib/Bio/Root/Utilities.pm \
	lib/Bio/Root/Version.pm \
	lib/Bio/Search/BlastStatistics.pm \
	lib/Bio/Search/BlastUtils.pm \
	lib/Bio/Search/DatabaseI.pm \
	lib/Bio/Search/GenericDatabase.pm \
	lib/Bio/Search/GenericStatistics.pm \
	lib/Bio/Search/HSP/BlastHSP.pm \
	lib/Bio/Search/HSP/BlastPullHSP.pm \
	lib/Bio/Search/HSP/FastaHSP.pm \
	lib/Bio/Search/HSP/GenericHSP.pm \
	lib/Bio/Search/HSP/HSPFactory.pm \
	lib/Bio/Search/HSP/HSPI.pm \
	lib/Bio/Search/HSP/ModelHSP.pm \
	lib/Bio/Search/HSP/PSLHSP.pm \
	lib/Bio/Search/HSP/PsiBlastHSP.pm \
	lib/Bio/Search/HSP/PullHSPI.pm \
	lib/Bio/Search/HSP/WABAHSP.pm \
	lib/Bio/Search/Hit/BlastHit.pm \
	lib/Bio/Search/Hit/BlastPullHit.pm \
	lib/Bio/Search/Hit/Fasta.pm \
	lib/Bio/Search/Hit/GenericHit.pm \
	lib/Bio/Search/Hit/HitFactory.pm \
	lib/Bio/Search/Hit/HitI.pm \
	lib/Bio/Search/Hit/ModelHit.pm \
	lib/Bio/Search/Hit/PsiBlastHit.pm \
	lib/Bio/Search/Hit/PullHitI.pm \
	lib/Bio/Search/Iteration/GenericIteration.pm \
	lib/Bio/Search/Iteration/IterationI.pm \
	lib/Bio/Search/Processor.pm \
	lib/Bio/Search/Result/BlastPullResult.pm \
	lib/Bio/Search/Result/BlastResult.pm \
	lib/Bio/Search/Result/CrossMatchResult.pm \
	lib/Bio/Search/Result/GenericResult.pm \
	lib/Bio/Search/Result/INFERNALResult.pm \
	lib/Bio/Search/Result/PullResultI.pm \
	lib/Bio/Search/Result/ResultFactory.pm \
	lib/Bio/Search/Result/ResultI.pm \
	lib/Bio/Search/Result/WABAResult.pm \
	lib/Bio/Search/SearchUtils.pm \
	lib/Bio/Search/StatisticsI.pm \
	lib/Bio/Search/Tiling/MapTileUtils.pm \
	lib/Bio/Search/Tiling/MapTiling.pm \
	lib/Bio/Search/Tiling/TilingI.pm \
	lib/Bio/SearchIO.pm \
	lib/Bio/SearchIO/EventHandlerI.pm \
	lib/Bio/SearchIO/FastHitEventBuilder.pm \
	lib/Bio/SearchIO/IteratedSearchResultEventBuilder.pm \
	lib/Bio/SearchIO/SearchResultEventBuilder.pm \
	lib/Bio/SearchIO/SearchWriterI.pm \
	lib/Bio/SearchIO/Writer/GbrowseGFF.pm \
	lib/Bio/SearchIO/Writer/HSPTableWriter.pm \
	lib/Bio/SearchIO/Writer/HTMLResultWriter.pm \
	lib/Bio/SearchIO/Writer/HitTableWriter.pm \
	lib/Bio/SearchIO/Writer/ResultTableWriter.pm \
	lib/Bio/SearchIO/Writer/TextResultWriter.pm \
	lib/Bio/SearchIO/axt.pm \
	lib/Bio/SearchIO/blast.pm \
	lib/Bio/SearchIO/blast_pull.pm \
	lib/Bio/SearchIO/blasttable.pm \
	lib/Bio/SearchIO/cross_match.pm \
	lib/Bio/SearchIO/erpin.pm \
	lib/Bio/SearchIO/exonerate.pm \
	lib/Bio/SearchIO/fasta.pm \
	lib/Bio/SearchIO/gmap_f9.pm \
	lib/Bio/SearchIO/infernal.pm \
	lib/Bio/SearchIO/megablast.pm \
	lib/Bio/SearchIO/psl.pm \
	lib/Bio/SearchIO/rnamotif.pm \
	lib/Bio/SearchIO/sim4.pm \
	lib/Bio/SearchIO/waba.pm \
	lib/Bio/SearchIO/wise.pm \
	lib/Bio/Seq.pm \
	lib/Bio/Seq/BaseSeqProcessor.pm \
	lib/Bio/Seq/EncodedSeq.pm \
	lib/Bio/Seq/LargeLocatableSeq.pm \
	lib/Bio/Seq/LargePrimarySeq.pm \
	lib/Bio/Seq/LargeSeq.pm \
	lib/Bio/Seq/LargeSeqI.pm \
	lib/Bio/Seq/Meta.pm \
	lib/Bio/Seq/Meta/Array.pm \
	lib/Bio/Seq/MetaI.pm \
	lib/Bio/Seq/PrimaryQual.pm \
	lib/Bio/Seq/PrimedSeq.pm \
	lib/Bio/Seq/QualI.pm \
	lib/Bio/Seq/Quality.pm \
	lib/Bio/Seq/RichSeq.pm \
	lib/Bio/Seq/RichSeqI.pm \
	lib/Bio/Seq/SeqBuilder.pm \
	lib/Bio/Seq/SeqFactory.pm \
	lib/Bio/Seq/SeqFastaSpeedFactory.pm \
	lib/Bio/Seq/SequenceTrace.pm \
	lib/Bio/Seq/SimulatedRead.pm \
	lib/Bio/Seq/TraceI.pm \
	lib/Bio/SeqAnalysisParserI.pm \
	lib/Bio/SeqFeature/Amplicon.pm \
	lib/Bio/SeqFeature/AnnotationAdaptor.pm \
	lib/Bio/SeqFeature/Collection.pm \
	lib/Bio/SeqFeature/CollectionI.pm \
	lib/Bio/SeqFeature/Computation.pm \
	lib/Bio/SeqFeature/FeaturePair.pm \
	lib/Bio/SeqFeature/Gene/Exon.pm \
	lib/Bio/SeqFeature/Gene/ExonI.pm \
	lib/Bio/SeqFeature/Gene/GeneStructure.pm \
	lib/Bio/SeqFeature/Gene/GeneStructureI.pm \
	lib/Bio/SeqFeature/Gene/Intron.pm \
	lib/Bio/SeqFeature/Gene/NC_Feature.pm \
	lib/Bio/SeqFeature/Gene/Poly_A_site.pm \
	lib/Bio/SeqFeature/Gene/Promoter.pm \
	lib/Bio/SeqFeature/Gene/Transcript.pm \
	lib/Bio/SeqFeature/Gene/TranscriptI.pm \
	lib/Bio/SeqFeature/Gene/UTR.pm \
	lib/Bio/SeqFeature/Generic.pm \
	lib/Bio/SeqFeature/Lite.pm \
	lib/Bio/SeqFeature/PositionProxy.pm \
	lib/Bio/SeqFeature/Primer.pm \
	lib/Bio/SeqFeature/Similarity.pm \
	lib/Bio/SeqFeature/SimilarityPair.pm \
	lib/Bio/SeqFeature/SubSeq.pm \
	lib/Bio/SeqFeature/Tools/FeatureNamer.pm \
	lib/Bio/SeqFeature/Tools/IDHandler.pm \
	lib/Bio/SeqFeature/Tools/TypeMapper.pm \
	lib/Bio/SeqFeature/Tools/Unflattener.pm \
	lib/Bio/SeqFeature/TypedSeqFeatureI.pm \
	lib/Bio/SeqFeatureI.pm \
	lib/Bio/SeqI.pm \
	lib/Bio/SeqIO.pm \
	lib/Bio/SeqIO/FTHelper.pm \
	lib/Bio/SeqIO/Handler/GenericRichSeqHandler.pm \
	lib/Bio/SeqIO/MultiFile.pm \
	lib/Bio/SeqIO/ace.pm \
	lib/Bio/SeqIO/asciitree.pm \
	lib/Bio/SeqIO/bsml.pm \
	lib/Bio/SeqIO/bsml_sax.pm \
	lib/Bio/SeqIO/embl.pm \
	lib/Bio/SeqIO/embldriver.pm \
	lib/Bio/SeqIO/fasta.pm \
	lib/Bio/SeqIO/fastq.pm \
	lib/Bio/SeqIO/game.pm \
	lib/Bio/SeqIO/game/featHandler.pm \
	lib/Bio/SeqIO/game/gameHandler.pm \
	lib/Bio/SeqIO/game/gameSubs.pm \
	lib/Bio/SeqIO/game/gameWriter.pm \
	lib/Bio/SeqIO/game/seqHandler.pm \
	lib/Bio/SeqIO/gbdriver.pm \
	lib/Bio/SeqIO/gbxml.pm \
	lib/Bio/SeqIO/gcg.pm \
	lib/Bio/SeqIO/genbank.pm \
	lib/Bio/SeqIO/kegg.pm \
	lib/Bio/SeqIO/largefasta.pm \
	lib/Bio/SeqIO/locuslink.pm \
	lib/Bio/SeqIO/mbsout.pm \
	lib/Bio/SeqIO/metafasta.pm \
	lib/Bio/SeqIO/msout.pm \
	lib/Bio/SeqIO/phd.pm \
	lib/Bio/SeqIO/pir.pm \
	lib/Bio/SeqIO/qual.pm \
	lib/Bio/SeqIO/raw.pm \
	lib/Bio/SeqIO/scf.pm \
	lib/Bio/SeqIO/seqxml.pm \
	lib/Bio/SeqIO/swiss.pm \
	lib/Bio/SeqIO/swissdriver.pm \
	lib/Bio/SeqIO/tab.pm \
	lib/Bio/SeqIO/table.pm \
	lib/Bio/SeqIO/tigr.pm \
	lib/Bio/SeqIO/tigrxml.pm \
	lib/Bio/SeqIO/tinyseq.pm \
	lib/Bio/SeqIO/tinyseq/tinyseqHandler.pm \
	lib/Bio/SeqUtils.pm \
	lib/Bio/SimpleAlign.pm \
	lib/Bio/SimpleAnalysisI.pm \
	lib/Bio/Species.pm \
	lib/Bio/Taxon.pm \
	lib/Bio/Tools/Alignment/Consed.pm \
	lib/Bio/Tools/Alignment/Trim.pm \
	lib/Bio/Tools/AmpliconSearch.pm \
	lib/Bio/Tools/Analysis/SimpleAnalysisBase.pm \
	lib/Bio/Tools/AnalysisResult.pm \
	lib/Bio/Tools/Blat.pm \
	lib/Bio/Tools/CodonTable.pm \
	lib/Bio/Tools/Coil.pm \
	lib/Bio/Tools/ECnumber.pm \
	lib/Bio/Tools/EMBOSS/Palindrome.pm \
	lib/Bio/Tools/EPCR.pm \
	lib/Bio/Tools/ESTScan.pm \
	lib/Bio/Tools/Eponine.pm \
	lib/Bio/Tools/Est2Genome.pm \
	lib/Bio/Tools/Fgenesh.pm \
	lib/Bio/Tools/FootPrinter.pm \
	lib/Bio/Tools/GFF.pm \
	lib/Bio/Tools/Geneid.pm \
	lib/Bio/Tools/Genemark.pm \
	lib/Bio/Tools/Genewise.pm \
	lib/Bio/Tools/Genomewise.pm \
	lib/Bio/Tools/Genscan.pm \
	lib/Bio/Tools/Glimmer.pm \
	lib/Bio/Tools/Grail.pm \
	lib/Bio/Tools/GuessSeqFormat.pm \
	lib/Bio/Tools/IUPAC.pm \
	lib/Bio/Tools/Lucy.pm \
	lib/Bio/Tools/MZEF.pm \
	lib/Bio/Tools/Match.pm \
	lib/Bio/Tools/OddCodes.pm \
	lib/Bio/Tools/Phylo/Gerp.pm \
	lib/Bio/Tools/Phylo/Molphy.pm \
	lib/Bio/Tools/Phylo/Molphy/Result.pm \
	lib/Bio/Tools/Phylo/Phylip/ProtDist.pm \
	lib/Bio/Tools/Prediction/Exon.pm \
	lib/Bio/Tools/Prediction/Gene.pm \
	lib/Bio/Tools/Primer/Assessor/Base.pm \
	lib/Bio/Tools/Primer/AssessorI.pm \
	lib/Bio/Tools/Primer/Feature.pm \
	lib/Bio/Tools/Primer/Pair.pm \
	lib/Bio/Tools/Primer3.pm \
	lib/Bio/Tools/Prints.pm \
	lib/Bio/Tools/Profile.pm \
	lib/Bio/Tools/Promoterwise.pm \
	lib/Bio/Tools/PrositeScan.pm \
	lib/Bio/Tools/Pseudowise.pm \
	lib/Bio/Tools/QRNA.pm \
	lib/Bio/Tools/RandomDistFunctions.pm \
	lib/Bio/Tools/RepeatMasker.pm \
	lib/Bio/Tools/Run/Analysis.pm \
	lib/Bio/Tools/Run/AnalysisFactory.pm \
	lib/Bio/Tools/Run/GenericParameters.pm \
	lib/Bio/Tools/Run/ParametersI.pm \
	lib/Bio/Tools/Run/Phylo/PhyloBase.pm \
	lib/Bio/Tools/Run/WrapperBase.pm \
	lib/Bio/Tools/Run/WrapperBase/CommandExts.pm \
	lib/Bio/Tools/Seg.pm \
	lib/Bio/Tools/SeqPattern.pm \
	lib/Bio/Tools/SeqPattern/Backtranslate.pm \
	lib/Bio/Tools/SeqStats.pm \
	lib/Bio/Tools/SeqWords.pm \
	lib/Bio/Tools/Sigcleave.pm \
	lib/Bio/Tools/Signalp.pm \
	lib/Bio/Tools/Signalp/ExtendedSignalp.pm \
	lib/Bio/Tools/Sim4/Exon.pm \
	lib/Bio/Tools/Sim4/Results.pm \
	lib/Bio/Tools/Spidey/Exon.pm \
	lib/Bio/Tools/Spidey/Results.pm \
	lib/Bio/Tools/TandemRepeatsFinder.pm \
	lib/Bio/Tools/TargetP.pm \
	lib/Bio/Tools/Tmhmm.pm \
	lib/Bio/Tools/ipcress.pm \
	lib/Bio/Tools/isPcr.pm \
	lib/Bio/Tools/pICalculator.pm \
	lib/Bio/Tools/tRNAscanSE.pm \
	lib/Bio/Tree/AnnotatableNode.pm \
	lib/Bio/Tree/Compatible.pm \
	lib/Bio/Tree/DistanceFactory.pm \
	lib/Bio/Tree/Node.pm \
	lib/Bio/Tree/NodeI.pm \
	lib/Bio/Tree/NodeNHX.pm \
	lib/Bio/Tree/RandomFactory.pm \
	lib/Bio/Tree/Statistics.pm \
	lib/Bio/Tree/Tree.pm \
	lib/Bio/Tree/TreeFunctionsI.pm \
	lib/Bio/Tree/TreeI.pm \
	lib/Bio/TreeIO.pm \
	lib/Bio/TreeIO/NewickParser.pm \
	lib/Bio/TreeIO/TreeEventBuilder.pm \
	lib/Bio/TreeIO/cluster.pm \
	lib/Bio/TreeIO/lintree.pm \
	lib/Bio/TreeIO/newick.pm \
	lib/Bio/TreeIO/nexus.pm \
	lib/Bio/TreeIO/nhx.pm \
	lib/Bio/TreeIO/pag.pm \
	lib/Bio/TreeIO/phyloxml.pm \
	lib/Bio/TreeIO/tabtree.pm \
	lib/Bio/UpdateableSeqI.pm \
	lib/Bio/WebAgent.pm \
	lib/BioPerl.pm
	$(NOECHO) $(POD2MAN) --section=1 --perm_rw=$(PERM_RW) -u \
	  bin/bp_aacomp $(INST_MAN1DIR)/bp_aacomp.$(MAN1EXT) \
	  bin/bp_bioflat_index $(INST_MAN1DIR)/bp_bioflat_index.$(MAN1EXT) \
	  bin/bp_biogetseq $(INST_MAN1DIR)/bp_biogetseq.$(MAN1EXT) \
	  bin/bp_dbsplit $(INST_MAN1DIR)/bp_dbsplit.$(MAN1EXT) \
	  bin/bp_extract_feature_seq $(INST_MAN1DIR)/bp_extract_feature_seq.$(MAN1EXT) \
	  bin/bp_fastam9_to_table $(INST_MAN1DIR)/bp_fastam9_to_table.$(MAN1EXT) \
	  bin/bp_fetch $(INST_MAN1DIR)/bp_fetch.$(MAN1EXT) \
	  bin/bp_filter_search $(INST_MAN1DIR)/bp_filter_search.$(MAN1EXT) \
	  bin/bp_find-blast-matches $(INST_MAN1DIR)/bp_find-blast-matches.$(MAN1EXT) \
	  bin/bp_gccalc $(INST_MAN1DIR)/bp_gccalc.$(MAN1EXT) \
	  bin/bp_genbank2gff3 $(INST_MAN1DIR)/bp_genbank2gff3.$(MAN1EXT) \
	  bin/bp_index $(INST_MAN1DIR)/bp_index.$(MAN1EXT) \
	  bin/bp_local_taxonomydb_query $(INST_MAN1DIR)/bp_local_taxonomydb_query.$(MAN1EXT) \
	  bin/bp_make_mrna_protein $(INST_MAN1DIR)/bp_make_mrna_protein.$(MAN1EXT) \
	  bin/bp_mask_by_search $(INST_MAN1DIR)/bp_mask_by_search.$(MAN1EXT) \
	  bin/bp_mrtrans $(INST_MAN1DIR)/bp_mrtrans.$(MAN1EXT) \
	  bin/bp_mutate $(INST_MAN1DIR)/bp_mutate.$(MAN1EXT) \
	  bin/bp_nexus2nh $(INST_MAN1DIR)/bp_nexus2nh.$(MAN1EXT) \
	  bin/bp_nrdb $(INST_MAN1DIR)/bp_nrdb.$(MAN1EXT) \
	  bin/bp_oligo_count $(INST_MAN1DIR)/bp_oligo_count.$(MAN1EXT) \
	  bin/bp_process_gadfly $(INST_MAN1DIR)/bp_process_gadfly.$(MAN1EXT) \
	  bin/bp_process_sgd $(INST_MAN1DIR)/bp_process_sgd.$(MAN1EXT) \
	  bin/bp_revtrans-motif $(INST_MAN1DIR)/bp_revtrans-motif.$(MAN1EXT) \
	  bin/bp_search2alnblocks $(INST_MAN1DIR)/bp_search2alnblocks.$(MAN1EXT) \
	  bin/bp_search2gff $(INST_MAN1DIR)/bp_search2gff.$(MAN1EXT) \
	  bin/bp_search2table $(INST_MAN1DIR)/bp_search2table.$(MAN1EXT) \
	  bin/bp_search2tribe $(INST_MAN1DIR)/bp_search2tribe.$(MAN1EXT) \
	  bin/bp_seq_length $(INST_MAN1DIR)/bp_seq_length.$(MAN1EXT) \
	  bin/bp_seqconvert $(INST_MAN1DIR)/bp_seqconvert.$(MAN1EXT) \
	  bin/bp_seqcut $(INST_MAN1DIR)/bp_seqcut.$(MAN1EXT) \
	  bin/bp_seqpart $(INST_MAN1DIR)/bp_seqpart.$(MAN1EXT) \
	  bin/bp_seqret $(INST_MAN1DIR)/bp_seqret.$(MAN1EXT) \
	  bin/bp_seqretsplit $(INST_MAN1DIR)/bp_seqretsplit.$(MAN1EXT) \
	  bin/bp_split_seq $(INST_MAN1DIR)/bp_split_seq.$(MAN1EXT) \
	  bin/bp_sreformat $(INST_MAN1DIR)/bp_sreformat.$(MAN1EXT) \
	  bin/bp_taxid4species $(INST_MAN1DIR)/bp_taxid4species.$(MAN1EXT) \
	  bin/bp_taxonomy2tree $(INST_MAN1DIR)/bp_taxonomy2tree.$(MAN1EXT) \
	  bin/bp_translate_seq $(INST_MAN1DIR)/bp_translate_seq.$(MAN1EXT) \
	  bin/bp_tree2pag $(INST_MAN1DIR)/bp_tree2pag.$(MAN1EXT) \
	  bin/bp_unflatten_seq $(INST_MAN1DIR)/bp_unflatten_seq.$(MAN1EXT) 
	$(NOECHO) $(POD2MAN) --section=3 --perm_rw=$(PERM_RW) -u \
	  lib/Bio/Align/AlignI.pm $(INST_MAN3DIR)/Bio::Align::AlignI.$(MAN3EXT) \
	  lib/Bio/Align/DNAStatistics.pm $(INST_MAN3DIR)/Bio::Align::DNAStatistics.$(MAN3EXT) \
	  lib/Bio/Align/PairwiseStatistics.pm $(INST_MAN3DIR)/Bio::Align::PairwiseStatistics.$(MAN3EXT) \
	  lib/Bio/Align/ProteinStatistics.pm $(INST_MAN3DIR)/Bio::Align::ProteinStatistics.$(MAN3EXT) \
	  lib/Bio/Align/StatisticsI.pm $(INST_MAN3DIR)/Bio::Align::StatisticsI.$(MAN3EXT) \
	  lib/Bio/Align/Utilities.pm $(INST_MAN3DIR)/Bio::Align::Utilities.$(MAN3EXT) \
	  lib/Bio/AlignIO.pm $(INST_MAN3DIR)/Bio::AlignIO.$(MAN3EXT) \
	  lib/Bio/AlignIO/Handler/GenericAlignHandler.pm $(INST_MAN3DIR)/Bio::AlignIO::Handler::GenericAlignHandler.$(MAN3EXT) \
	  lib/Bio/AlignIO/arp.pm $(INST_MAN3DIR)/Bio::AlignIO::arp.$(MAN3EXT) \
	  lib/Bio/AlignIO/bl2seq.pm $(INST_MAN3DIR)/Bio::AlignIO::bl2seq.$(MAN3EXT) \
	  lib/Bio/AlignIO/clustalw.pm $(INST_MAN3DIR)/Bio::AlignIO::clustalw.$(MAN3EXT) \
	  lib/Bio/AlignIO/emboss.pm $(INST_MAN3DIR)/Bio::AlignIO::emboss.$(MAN3EXT) \
	  lib/Bio/AlignIO/fasta.pm $(INST_MAN3DIR)/Bio::AlignIO::fasta.$(MAN3EXT) \
	  lib/Bio/AlignIO/largemultifasta.pm $(INST_MAN3DIR)/Bio::AlignIO::largemultifasta.$(MAN3EXT) \
	  lib/Bio/AlignIO/maf.pm $(INST_MAN3DIR)/Bio::AlignIO::maf.$(MAN3EXT) \
	  lib/Bio/AlignIO/mase.pm $(INST_MAN3DIR)/Bio::AlignIO::mase.$(MAN3EXT) \
	  lib/Bio/AlignIO/mega.pm $(INST_MAN3DIR)/Bio::AlignIO::mega.$(MAN3EXT) \
	  lib/Bio/AlignIO/meme.pm $(INST_MAN3DIR)/Bio::AlignIO::meme.$(MAN3EXT) \
	  lib/Bio/AlignIO/metafasta.pm $(INST_MAN3DIR)/Bio::AlignIO::metafasta.$(MAN3EXT) \
	  lib/Bio/AlignIO/msf.pm $(INST_MAN3DIR)/Bio::AlignIO::msf.$(MAN3EXT) \
	  lib/Bio/AlignIO/nexus.pm $(INST_MAN3DIR)/Bio::AlignIO::nexus.$(MAN3EXT) \
	  lib/Bio/AlignIO/pfam.pm $(INST_MAN3DIR)/Bio::AlignIO::pfam.$(MAN3EXT) \
	  lib/Bio/AlignIO/phylip.pm $(INST_MAN3DIR)/Bio::AlignIO::phylip.$(MAN3EXT) \
	  lib/Bio/AlignIO/po.pm $(INST_MAN3DIR)/Bio::AlignIO::po.$(MAN3EXT) \
	  lib/Bio/AlignIO/proda.pm $(INST_MAN3DIR)/Bio::AlignIO::proda.$(MAN3EXT) \
	  lib/Bio/AlignIO/prodom.pm $(INST_MAN3DIR)/Bio::AlignIO::prodom.$(MAN3EXT) \
	  lib/Bio/AlignIO/psi.pm $(INST_MAN3DIR)/Bio::AlignIO::psi.$(MAN3EXT) \
	  lib/Bio/AlignIO/selex.pm $(INST_MAN3DIR)/Bio::AlignIO::selex.$(MAN3EXT) \
	  lib/Bio/AlignIO/xmfa.pm $(INST_MAN3DIR)/Bio::AlignIO::xmfa.$(MAN3EXT) \
	  lib/Bio/AnalysisI.pm $(INST_MAN3DIR)/Bio::AnalysisI.$(MAN3EXT) \
	  lib/Bio/AnalysisParserI.pm $(INST_MAN3DIR)/Bio::AnalysisParserI.$(MAN3EXT) \
	  lib/Bio/AnalysisResultI.pm $(INST_MAN3DIR)/Bio::AnalysisResultI.$(MAN3EXT) \
	  lib/Bio/AnnotatableI.pm $(INST_MAN3DIR)/Bio::AnnotatableI.$(MAN3EXT) \
	  lib/Bio/Annotation/AnnotationFactory.pm $(INST_MAN3DIR)/Bio::Annotation::AnnotationFactory.$(MAN3EXT) \
	  lib/Bio/Annotation/Collection.pm $(INST_MAN3DIR)/Bio::Annotation::Collection.$(MAN3EXT) 
	$(NOECHO) $(POD2MAN) --section=3 --perm_rw=$(PERM_RW) -u \
	  lib/Bio/Annotation/Comment.pm $(INST_MAN3DIR)/Bio::Annotation::Comment.$(MAN3EXT) \
	  lib/Bio/Annotation/DBLink.pm $(INST_MAN3DIR)/Bio::Annotation::DBLink.$(MAN3EXT) \
	  lib/Bio/Annotation/OntologyTerm.pm $(INST_MAN3DIR)/Bio::Annotation::OntologyTerm.$(MAN3EXT) \
	  lib/Bio/Annotation/Reference.pm $(INST_MAN3DIR)/Bio::Annotation::Reference.$(MAN3EXT) \
	  lib/Bio/Annotation/Relation.pm $(INST_MAN3DIR)/Bio::Annotation::Relation.$(MAN3EXT) \
	  lib/Bio/Annotation/SimpleValue.pm $(INST_MAN3DIR)/Bio::Annotation::SimpleValue.$(MAN3EXT) \
	  lib/Bio/Annotation/StructuredValue.pm $(INST_MAN3DIR)/Bio::Annotation::StructuredValue.$(MAN3EXT) \
	  lib/Bio/Annotation/TagTree.pm $(INST_MAN3DIR)/Bio::Annotation::TagTree.$(MAN3EXT) \
	  lib/Bio/Annotation/Target.pm $(INST_MAN3DIR)/Bio::Annotation::Target.$(MAN3EXT) \
	  lib/Bio/Annotation/Tree.pm $(INST_MAN3DIR)/Bio::Annotation::Tree.$(MAN3EXT) \
	  lib/Bio/Annotation/TypeManager.pm $(INST_MAN3DIR)/Bio::Annotation::TypeManager.$(MAN3EXT) \
	  lib/Bio/AnnotationCollectionI.pm $(INST_MAN3DIR)/Bio::AnnotationCollectionI.$(MAN3EXT) \
	  lib/Bio/AnnotationI.pm $(INST_MAN3DIR)/Bio::AnnotationI.$(MAN3EXT) \
	  lib/Bio/CodonUsage/IO.pm $(INST_MAN3DIR)/Bio::CodonUsage::IO.$(MAN3EXT) \
	  lib/Bio/CodonUsage/Table.pm $(INST_MAN3DIR)/Bio::CodonUsage::Table.$(MAN3EXT) \
	  lib/Bio/DB/DBFetch.pm $(INST_MAN3DIR)/Bio::DB::DBFetch.$(MAN3EXT) \
	  lib/Bio/DB/Failover.pm $(INST_MAN3DIR)/Bio::DB::Failover.$(MAN3EXT) \
	  lib/Bio/DB/Fasta.pm $(INST_MAN3DIR)/Bio::DB::Fasta.$(MAN3EXT) \
	  lib/Bio/DB/FileCache.pm $(INST_MAN3DIR)/Bio::DB::FileCache.$(MAN3EXT) \
	  lib/Bio/DB/Flat.pm $(INST_MAN3DIR)/Bio::DB::Flat.$(MAN3EXT) \
	  lib/Bio/DB/Flat/BDB.pm $(INST_MAN3DIR)/Bio::DB::Flat::BDB.$(MAN3EXT) \
	  lib/Bio/DB/Flat/BDB/embl.pm $(INST_MAN3DIR)/Bio::DB::Flat::BDB::embl.$(MAN3EXT) \
	  lib/Bio/DB/Flat/BDB/fasta.pm $(INST_MAN3DIR)/Bio::DB::Flat::BDB::fasta.$(MAN3EXT) \
	  lib/Bio/DB/Flat/BDB/genbank.pm $(INST_MAN3DIR)/Bio::DB::Flat::BDB::genbank.$(MAN3EXT) \
	  lib/Bio/DB/Flat/BDB/swiss.pm $(INST_MAN3DIR)/Bio::DB::Flat::BDB::swiss.$(MAN3EXT) \
	  lib/Bio/DB/Flat/BinarySearch.pm $(INST_MAN3DIR)/Bio::DB::Flat::BinarySearch.$(MAN3EXT) \
	  lib/Bio/DB/GFF/Util/Binning.pm $(INST_MAN3DIR)/Bio::DB::GFF::Util::Binning.$(MAN3EXT) \
	  lib/Bio/DB/GFF/Util/Rearrange.pm $(INST_MAN3DIR)/Bio::DB::GFF::Util::Rearrange.$(MAN3EXT) \
	  lib/Bio/DB/GenericWebAgent.pm $(INST_MAN3DIR)/Bio::DB::GenericWebAgent.$(MAN3EXT) \
	  lib/Bio/DB/InMemoryCache.pm $(INST_MAN3DIR)/Bio::DB::InMemoryCache.$(MAN3EXT) \
	  lib/Bio/DB/IndexedBase.pm $(INST_MAN3DIR)/Bio::DB::IndexedBase.$(MAN3EXT) \
	  lib/Bio/DB/LocationI.pm $(INST_MAN3DIR)/Bio::DB::LocationI.$(MAN3EXT) \
	  lib/Bio/DB/Qual.pm $(INST_MAN3DIR)/Bio::DB::Qual.$(MAN3EXT) \
	  lib/Bio/DB/Query/WebQuery.pm $(INST_MAN3DIR)/Bio::DB::Query::WebQuery.$(MAN3EXT) 
	$(NOECHO) $(POD2MAN) --section=3 --perm_rw=$(PERM_RW) -u \
	  lib/Bio/DB/QueryI.pm $(INST_MAN3DIR)/Bio::DB::QueryI.$(MAN3EXT) \
	  lib/Bio/DB/RandomAccessI.pm $(INST_MAN3DIR)/Bio::DB::RandomAccessI.$(MAN3EXT) \
	  lib/Bio/DB/ReferenceI.pm $(INST_MAN3DIR)/Bio::DB::ReferenceI.$(MAN3EXT) \
	  lib/Bio/DB/Registry.pm $(INST_MAN3DIR)/Bio::DB::Registry.$(MAN3EXT) \
	  lib/Bio/DB/SeqI.pm $(INST_MAN3DIR)/Bio::DB::SeqI.$(MAN3EXT) \
	  lib/Bio/DB/Taxonomy.pm $(INST_MAN3DIR)/Bio::DB::Taxonomy.$(MAN3EXT) \
	  lib/Bio/DB/Taxonomy/flatfile.pm $(INST_MAN3DIR)/Bio::DB::Taxonomy::flatfile.$(MAN3EXT) \
	  lib/Bio/DB/Taxonomy/greengenes.pm $(INST_MAN3DIR)/Bio::DB::Taxonomy::greengenes.$(MAN3EXT) \
	  lib/Bio/DB/Taxonomy/list.pm $(INST_MAN3DIR)/Bio::DB::Taxonomy::list.$(MAN3EXT) \
	  lib/Bio/DB/Taxonomy/silva.pm $(INST_MAN3DIR)/Bio::DB::Taxonomy::silva.$(MAN3EXT) \
	  lib/Bio/DB/UpdateableSeqI.pm $(INST_MAN3DIR)/Bio::DB::UpdateableSeqI.$(MAN3EXT) \
	  lib/Bio/DB/WebDBSeqI.pm $(INST_MAN3DIR)/Bio::DB::WebDBSeqI.$(MAN3EXT) \
	  lib/Bio/DBLinkContainerI.pm $(INST_MAN3DIR)/Bio::DBLinkContainerI.$(MAN3EXT) \
	  lib/Bio/Das/FeatureTypeI.pm $(INST_MAN3DIR)/Bio::Das::FeatureTypeI.$(MAN3EXT) \
	  lib/Bio/Das/SegmentI.pm $(INST_MAN3DIR)/Bio::Das::SegmentI.$(MAN3EXT) \
	  lib/Bio/DasI.pm $(INST_MAN3DIR)/Bio::DasI.$(MAN3EXT) \
	  lib/Bio/DescribableI.pm $(INST_MAN3DIR)/Bio::DescribableI.$(MAN3EXT) \
	  lib/Bio/Event/EventGeneratorI.pm $(INST_MAN3DIR)/Bio::Event::EventGeneratorI.$(MAN3EXT) \
	  lib/Bio/Event/EventHandlerI.pm $(INST_MAN3DIR)/Bio::Event::EventHandlerI.$(MAN3EXT) \
	  lib/Bio/Factory/AnalysisI.pm $(INST_MAN3DIR)/Bio::Factory::AnalysisI.$(MAN3EXT) \
	  lib/Bio/Factory/ApplicationFactoryI.pm $(INST_MAN3DIR)/Bio::Factory::ApplicationFactoryI.$(MAN3EXT) \
	  lib/Bio/Factory/DriverFactory.pm $(INST_MAN3DIR)/Bio::Factory::DriverFactory.$(MAN3EXT) \
	  lib/Bio/Factory/FTLocationFactory.pm $(INST_MAN3DIR)/Bio::Factory::FTLocationFactory.$(MAN3EXT) \
	  lib/Bio/Factory/LocationFactoryI.pm $(INST_MAN3DIR)/Bio::Factory::LocationFactoryI.$(MAN3EXT) \
	  lib/Bio/Factory/ObjectBuilderI.pm $(INST_MAN3DIR)/Bio::Factory::ObjectBuilderI.$(MAN3EXT) \
	  lib/Bio/Factory/ObjectFactory.pm $(INST_MAN3DIR)/Bio::Factory::ObjectFactory.$(MAN3EXT) \
	  lib/Bio/Factory/ObjectFactoryI.pm $(INST_MAN3DIR)/Bio::Factory::ObjectFactoryI.$(MAN3EXT) \
	  lib/Bio/Factory/SeqAnalysisParserFactory.pm $(INST_MAN3DIR)/Bio::Factory::SeqAnalysisParserFactory.$(MAN3EXT) \
	  lib/Bio/Factory/SeqAnalysisParserFactoryI.pm $(INST_MAN3DIR)/Bio::Factory::SeqAnalysisParserFactoryI.$(MAN3EXT) \
	  lib/Bio/Factory/SequenceFactoryI.pm $(INST_MAN3DIR)/Bio::Factory::SequenceFactoryI.$(MAN3EXT) \
	  lib/Bio/Factory/SequenceProcessorI.pm $(INST_MAN3DIR)/Bio::Factory::SequenceProcessorI.$(MAN3EXT) \
	  lib/Bio/Factory/SequenceStreamI.pm $(INST_MAN3DIR)/Bio::Factory::SequenceStreamI.$(MAN3EXT) 
	$(NOECHO) $(POD2MAN) --section=3 --perm_rw=$(PERM_RW) -u \
	  lib/Bio/Factory/TreeFactoryI.pm $(INST_MAN3DIR)/Bio::Factory::TreeFactoryI.$(MAN3EXT) \
	  lib/Bio/FeatureHolderI.pm $(INST_MAN3DIR)/Bio::FeatureHolderI.$(MAN3EXT) \
	  lib/Bio/HandlerBaseI.pm $(INST_MAN3DIR)/Bio::HandlerBaseI.$(MAN3EXT) \
	  lib/Bio/IdCollectionI.pm $(INST_MAN3DIR)/Bio::IdCollectionI.$(MAN3EXT) \
	  lib/Bio/IdentifiableI.pm $(INST_MAN3DIR)/Bio::IdentifiableI.$(MAN3EXT) \
	  lib/Bio/Index/Abstract.pm $(INST_MAN3DIR)/Bio::Index::Abstract.$(MAN3EXT) \
	  lib/Bio/Index/AbstractSeq.pm $(INST_MAN3DIR)/Bio::Index::AbstractSeq.$(MAN3EXT) \
	  lib/Bio/Index/Blast.pm $(INST_MAN3DIR)/Bio::Index::Blast.$(MAN3EXT) \
	  lib/Bio/Index/BlastTable.pm $(INST_MAN3DIR)/Bio::Index::BlastTable.$(MAN3EXT) \
	  lib/Bio/Index/EMBL.pm $(INST_MAN3DIR)/Bio::Index::EMBL.$(MAN3EXT) \
	  lib/Bio/Index/Fasta.pm $(INST_MAN3DIR)/Bio::Index::Fasta.$(MAN3EXT) \
	  lib/Bio/Index/Fastq.pm $(INST_MAN3DIR)/Bio::Index::Fastq.$(MAN3EXT) \
	  lib/Bio/Index/GenBank.pm $(INST_MAN3DIR)/Bio::Index::GenBank.$(MAN3EXT) \
	  lib/Bio/Index/Qual.pm $(INST_MAN3DIR)/Bio::Index::Qual.$(MAN3EXT) \
	  lib/Bio/Index/SwissPfam.pm $(INST_MAN3DIR)/Bio::Index::SwissPfam.$(MAN3EXT) \
	  lib/Bio/Index/Swissprot.pm $(INST_MAN3DIR)/Bio::Index::Swissprot.$(MAN3EXT) \
	  lib/Bio/LocatableSeq.pm $(INST_MAN3DIR)/Bio::LocatableSeq.$(MAN3EXT) \
	  lib/Bio/Location/Atomic.pm $(INST_MAN3DIR)/Bio::Location::Atomic.$(MAN3EXT) \
	  lib/Bio/Location/AvWithinCoordPolicy.pm $(INST_MAN3DIR)/Bio::Location::AvWithinCoordPolicy.$(MAN3EXT) \
	  lib/Bio/Location/CoordinatePolicyI.pm $(INST_MAN3DIR)/Bio::Location::CoordinatePolicyI.$(MAN3EXT) \
	  lib/Bio/Location/Fuzzy.pm $(INST_MAN3DIR)/Bio::Location::Fuzzy.$(MAN3EXT) \
	  lib/Bio/Location/FuzzyLocationI.pm $(INST_MAN3DIR)/Bio::Location::FuzzyLocationI.$(MAN3EXT) \
	  lib/Bio/Location/NarrowestCoordPolicy.pm $(INST_MAN3DIR)/Bio::Location::NarrowestCoordPolicy.$(MAN3EXT) \
	  lib/Bio/Location/Simple.pm $(INST_MAN3DIR)/Bio::Location::Simple.$(MAN3EXT) \
	  lib/Bio/Location/Split.pm $(INST_MAN3DIR)/Bio::Location::Split.$(MAN3EXT) \
	  lib/Bio/Location/SplitLocationI.pm $(INST_MAN3DIR)/Bio::Location::SplitLocationI.$(MAN3EXT) \
	  lib/Bio/Location/WidestCoordPolicy.pm $(INST_MAN3DIR)/Bio::Location::WidestCoordPolicy.$(MAN3EXT) \
	  lib/Bio/LocationI.pm $(INST_MAN3DIR)/Bio::LocationI.$(MAN3EXT) \
	  lib/Bio/Matrix/Generic.pm $(INST_MAN3DIR)/Bio::Matrix::Generic.$(MAN3EXT) \
	  lib/Bio/Matrix/IO.pm $(INST_MAN3DIR)/Bio::Matrix::IO.$(MAN3EXT) \
	  lib/Bio/Matrix/IO/mlagan.pm $(INST_MAN3DIR)/Bio::Matrix::IO::mlagan.$(MAN3EXT) \
	  lib/Bio/Matrix/IO/phylip.pm $(INST_MAN3DIR)/Bio::Matrix::IO::phylip.$(MAN3EXT) \
	  lib/Bio/Matrix/IO/scoring.pm $(INST_MAN3DIR)/Bio::Matrix::IO::scoring.$(MAN3EXT) \
	  lib/Bio/Matrix/MatrixI.pm $(INST_MAN3DIR)/Bio::Matrix::MatrixI.$(MAN3EXT) 
	$(NOECHO) $(POD2MAN) --section=3 --perm_rw=$(PERM_RW) -u \
	  lib/Bio/Matrix/Mlagan.pm $(INST_MAN3DIR)/Bio::Matrix::Mlagan.$(MAN3EXT) \
	  lib/Bio/Matrix/PSM/IO.pm $(INST_MAN3DIR)/Bio::Matrix::PSM::IO.$(MAN3EXT) \
	  lib/Bio/Matrix/PSM/IO/mast.pm $(INST_MAN3DIR)/Bio::Matrix::PSM::IO::mast.$(MAN3EXT) \
	  lib/Bio/Matrix/PSM/IO/masta.pm $(INST_MAN3DIR)/Bio::Matrix::PSM::IO::masta.$(MAN3EXT) \
	  lib/Bio/Matrix/PSM/IO/meme.pm $(INST_MAN3DIR)/Bio::Matrix::PSM::IO::meme.$(MAN3EXT) \
	  lib/Bio/Matrix/PSM/IO/psiblast.pm $(INST_MAN3DIR)/Bio::Matrix::PSM::IO::psiblast.$(MAN3EXT) \
	  lib/Bio/Matrix/PSM/IO/transfac.pm $(INST_MAN3DIR)/Bio::Matrix::PSM::IO::transfac.$(MAN3EXT) \
	  lib/Bio/Matrix/PSM/InstanceSite.pm $(INST_MAN3DIR)/Bio::Matrix::PSM::InstanceSite.$(MAN3EXT) \
	  lib/Bio/Matrix/PSM/InstanceSiteI.pm $(INST_MAN3DIR)/Bio::Matrix::PSM::InstanceSiteI.$(MAN3EXT) \
	  lib/Bio/Matrix/PSM/ProtMatrix.pm $(INST_MAN3DIR)/Bio::Matrix::PSM::ProtMatrix.$(MAN3EXT) \
	  lib/Bio/Matrix/PSM/ProtPsm.pm $(INST_MAN3DIR)/Bio::Matrix::PSM::ProtPsm.$(MAN3EXT) \
	  lib/Bio/Matrix/PSM/Psm.pm $(INST_MAN3DIR)/Bio::Matrix::PSM::Psm.$(MAN3EXT) \
	  lib/Bio/Matrix/PSM/PsmHeader.pm $(INST_MAN3DIR)/Bio::Matrix::PSM::PsmHeader.$(MAN3EXT) \
	  lib/Bio/Matrix/PSM/PsmHeaderI.pm $(INST_MAN3DIR)/Bio::Matrix::PSM::PsmHeaderI.$(MAN3EXT) \
	  lib/Bio/Matrix/PSM/PsmI.pm $(INST_MAN3DIR)/Bio::Matrix::PSM::PsmI.$(MAN3EXT) \
	  lib/Bio/Matrix/PSM/SiteMatrix.pm $(INST_MAN3DIR)/Bio::Matrix::PSM::SiteMatrix.$(MAN3EXT) \
	  lib/Bio/Matrix/PSM/SiteMatrixI.pm $(INST_MAN3DIR)/Bio::Matrix::PSM::SiteMatrixI.$(MAN3EXT) \
	  lib/Bio/Matrix/PhylipDist.pm $(INST_MAN3DIR)/Bio::Matrix::PhylipDist.$(MAN3EXT) \
	  lib/Bio/Matrix/Scoring.pm $(INST_MAN3DIR)/Bio::Matrix::Scoring.$(MAN3EXT) \
	  lib/Bio/Ontology/DocumentRegistry.pm $(INST_MAN3DIR)/Bio::Ontology::DocumentRegistry.$(MAN3EXT) \
	  lib/Bio/Ontology/GOterm.pm $(INST_MAN3DIR)/Bio::Ontology::GOterm.$(MAN3EXT) \
	  lib/Bio/Ontology/InterProTerm.pm $(INST_MAN3DIR)/Bio::Ontology::InterProTerm.$(MAN3EXT) \
	  lib/Bio/Ontology/OBOEngine.pm $(INST_MAN3DIR)/Bio::Ontology::OBOEngine.$(MAN3EXT) \
	  lib/Bio/Ontology/OBOterm.pm $(INST_MAN3DIR)/Bio::Ontology::OBOterm.$(MAN3EXT) \
	  lib/Bio/Ontology/Ontology.pm $(INST_MAN3DIR)/Bio::Ontology::Ontology.$(MAN3EXT) \
	  lib/Bio/Ontology/OntologyEngineI.pm $(INST_MAN3DIR)/Bio::Ontology::OntologyEngineI.$(MAN3EXT) \
	  lib/Bio/Ontology/OntologyI.pm $(INST_MAN3DIR)/Bio::Ontology::OntologyI.$(MAN3EXT) \
	  lib/Bio/Ontology/OntologyStore.pm $(INST_MAN3DIR)/Bio::Ontology::OntologyStore.$(MAN3EXT) \
	  lib/Bio/Ontology/Path.pm $(INST_MAN3DIR)/Bio::Ontology::Path.$(MAN3EXT) \
	  lib/Bio/Ontology/PathI.pm $(INST_MAN3DIR)/Bio::Ontology::PathI.$(MAN3EXT) \
	  lib/Bio/Ontology/Relationship.pm $(INST_MAN3DIR)/Bio::Ontology::Relationship.$(MAN3EXT) 
	$(NOECHO) $(POD2MAN) --section=3 --perm_rw=$(PERM_RW) -u \
	  lib/Bio/Ontology/RelationshipFactory.pm $(INST_MAN3DIR)/Bio::Ontology::RelationshipFactory.$(MAN3EXT) \
	  lib/Bio/Ontology/RelationshipI.pm $(INST_MAN3DIR)/Bio::Ontology::RelationshipI.$(MAN3EXT) \
	  lib/Bio/Ontology/RelationshipType.pm $(INST_MAN3DIR)/Bio::Ontology::RelationshipType.$(MAN3EXT) \
	  lib/Bio/Ontology/SimpleGOEngine/GraphAdaptor.pm $(INST_MAN3DIR)/Bio::Ontology::SimpleGOEngine::GraphAdaptor.$(MAN3EXT) \
	  lib/Bio/Ontology/SimpleOntologyEngine.pm $(INST_MAN3DIR)/Bio::Ontology::SimpleOntologyEngine.$(MAN3EXT) \
	  lib/Bio/Ontology/Term.pm $(INST_MAN3DIR)/Bio::Ontology::Term.$(MAN3EXT) \
	  lib/Bio/Ontology/TermFactory.pm $(INST_MAN3DIR)/Bio::Ontology::TermFactory.$(MAN3EXT) \
	  lib/Bio/Ontology/TermI.pm $(INST_MAN3DIR)/Bio::Ontology::TermI.$(MAN3EXT) \
	  lib/Bio/OntologyIO.pm $(INST_MAN3DIR)/Bio::OntologyIO.$(MAN3EXT) \
	  lib/Bio/OntologyIO/Handlers/BaseSAXHandler.pm $(INST_MAN3DIR)/Bio::OntologyIO::Handlers::BaseSAXHandler.$(MAN3EXT) \
	  lib/Bio/OntologyIO/Handlers/InterProHandler.pm $(INST_MAN3DIR)/Bio::OntologyIO::Handlers::InterProHandler.$(MAN3EXT) \
	  lib/Bio/OntologyIO/Handlers/InterPro_BioSQL_Handler.pm $(INST_MAN3DIR)/Bio::OntologyIO::Handlers::InterPro_BioSQL_Handler.$(MAN3EXT) \
	  lib/Bio/OntologyIO/InterProParser.pm $(INST_MAN3DIR)/Bio::OntologyIO::InterProParser.$(MAN3EXT) \
	  lib/Bio/OntologyIO/dagflat.pm $(INST_MAN3DIR)/Bio::OntologyIO::dagflat.$(MAN3EXT) \
	  lib/Bio/OntologyIO/goflat.pm $(INST_MAN3DIR)/Bio::OntologyIO::goflat.$(MAN3EXT) \
	  lib/Bio/OntologyIO/obo.pm $(INST_MAN3DIR)/Bio::OntologyIO::obo.$(MAN3EXT) \
	  lib/Bio/OntologyIO/simplehierarchy.pm $(INST_MAN3DIR)/Bio::OntologyIO::simplehierarchy.$(MAN3EXT) \
	  lib/Bio/OntologyIO/soflat.pm $(INST_MAN3DIR)/Bio::OntologyIO::soflat.$(MAN3EXT) \
	  lib/Bio/ParameterBaseI.pm $(INST_MAN3DIR)/Bio::ParameterBaseI.$(MAN3EXT) \
	  lib/Bio/PrimarySeq.pm $(INST_MAN3DIR)/Bio::PrimarySeq.$(MAN3EXT) \
	  lib/Bio/PrimarySeqI.pm $(INST_MAN3DIR)/Bio::PrimarySeqI.$(MAN3EXT) \
	  lib/Bio/PullParserI.pm $(INST_MAN3DIR)/Bio::PullParserI.$(MAN3EXT) \
	  lib/Bio/Range.pm $(INST_MAN3DIR)/Bio::Range.$(MAN3EXT) \
	  lib/Bio/RangeI.pm $(INST_MAN3DIR)/Bio::RangeI.$(MAN3EXT) \
	  lib/Bio/Root/Exception.pm $(INST_MAN3DIR)/Bio::Root::Exception.$(MAN3EXT) \
	  lib/Bio/Root/HTTPget.pm $(INST_MAN3DIR)/Bio::Root::HTTPget.$(MAN3EXT) \
	  lib/Bio/Root/IO.pm $(INST_MAN3DIR)/Bio::Root::IO.$(MAN3EXT) \
	  lib/Bio/Root/Root.pm $(INST_MAN3DIR)/Bio::Root::Root.$(MAN3EXT) \
	  lib/Bio/Root/RootI.pm $(INST_MAN3DIR)/Bio::Root::RootI.$(MAN3EXT) \
	  lib/Bio/Root/Storable.pm $(INST_MAN3DIR)/Bio::Root::Storable.$(MAN3EXT) \
	  lib/Bio/Root/Test.pm $(INST_MAN3DIR)/Bio::Root::Test.$(MAN3EXT) \
	  lib/Bio/Root/TestObject.pm $(INST_MAN3DIR)/Bio::Root::TestObject.$(MAN3EXT) 
	$(NOECHO) $(POD2MAN) --section=3 --perm_rw=$(PERM_RW) -u \
	  lib/Bio/Root/Utilities.pm $(INST_MAN3DIR)/Bio::Root::Utilities.$(MAN3EXT) \
	  lib/Bio/Root/Version.pm $(INST_MAN3DIR)/Bio::Root::Version.$(MAN3EXT) \
	  lib/Bio/Search/BlastStatistics.pm $(INST_MAN3DIR)/Bio::Search::BlastStatistics.$(MAN3EXT) \
	  lib/Bio/Search/BlastUtils.pm $(INST_MAN3DIR)/Bio::Search::BlastUtils.$(MAN3EXT) \
	  lib/Bio/Search/DatabaseI.pm $(INST_MAN3DIR)/Bio::Search::DatabaseI.$(MAN3EXT) \
	  lib/Bio/Search/GenericDatabase.pm $(INST_MAN3DIR)/Bio::Search::GenericDatabase.$(MAN3EXT) \
	  lib/Bio/Search/GenericStatistics.pm $(INST_MAN3DIR)/Bio::Search::GenericStatistics.$(MAN3EXT) \
	  lib/Bio/Search/HSP/BlastHSP.pm $(INST_MAN3DIR)/Bio::Search::HSP::BlastHSP.$(MAN3EXT) \
	  lib/Bio/Search/HSP/BlastPullHSP.pm $(INST_MAN3DIR)/Bio::Search::HSP::BlastPullHSP.$(MAN3EXT) \
	  lib/Bio/Search/HSP/FastaHSP.pm $(INST_MAN3DIR)/Bio::Search::HSP::FastaHSP.$(MAN3EXT) \
	  lib/Bio/Search/HSP/GenericHSP.pm $(INST_MAN3DIR)/Bio::Search::HSP::GenericHSP.$(MAN3EXT) \
	  lib/Bio/Search/HSP/HSPFactory.pm $(INST_MAN3DIR)/Bio::Search::HSP::HSPFactory.$(MAN3EXT) \
	  lib/Bio/Search/HSP/HSPI.pm $(INST_MAN3DIR)/Bio::Search::HSP::HSPI.$(MAN3EXT) \
	  lib/Bio/Search/HSP/ModelHSP.pm $(INST_MAN3DIR)/Bio::Search::HSP::ModelHSP.$(MAN3EXT) \
	  lib/Bio/Search/HSP/PSLHSP.pm $(INST_MAN3DIR)/Bio::Search::HSP::PSLHSP.$(MAN3EXT) \
	  lib/Bio/Search/HSP/PsiBlastHSP.pm $(INST_MAN3DIR)/Bio::Search::HSP::PsiBlastHSP.$(MAN3EXT) \
	  lib/Bio/Search/HSP/PullHSPI.pm $(INST_MAN3DIR)/Bio::Search::HSP::PullHSPI.$(MAN3EXT) \
	  lib/Bio/Search/HSP/WABAHSP.pm $(INST_MAN3DIR)/Bio::Search::HSP::WABAHSP.$(MAN3EXT) \
	  lib/Bio/Search/Hit/BlastHit.pm $(INST_MAN3DIR)/Bio::Search::Hit::BlastHit.$(MAN3EXT) \
	  lib/Bio/Search/Hit/BlastPullHit.pm $(INST_MAN3DIR)/Bio::Search::Hit::BlastPullHit.$(MAN3EXT) \
	  lib/Bio/Search/Hit/Fasta.pm $(INST_MAN3DIR)/Bio::Search::Hit::Fasta.$(MAN3EXT) \
	  lib/Bio/Search/Hit/GenericHit.pm $(INST_MAN3DIR)/Bio::Search::Hit::GenericHit.$(MAN3EXT) \
	  lib/Bio/Search/Hit/HitFactory.pm $(INST_MAN3DIR)/Bio::Search::Hit::HitFactory.$(MAN3EXT) \
	  lib/Bio/Search/Hit/HitI.pm $(INST_MAN3DIR)/Bio::Search::Hit::HitI.$(MAN3EXT) \
	  lib/Bio/Search/Hit/ModelHit.pm $(INST_MAN3DIR)/Bio::Search::Hit::ModelHit.$(MAN3EXT) \
	  lib/Bio/Search/Hit/PsiBlastHit.pm $(INST_MAN3DIR)/Bio::Search::Hit::PsiBlastHit.$(MAN3EXT) \
	  lib/Bio/Search/Hit/PullHitI.pm $(INST_MAN3DIR)/Bio::Search::Hit::PullHitI.$(MAN3EXT) \
	  lib/Bio/Search/Iteration/GenericIteration.pm $(INST_MAN3DIR)/Bio::Search::Iteration::GenericIteration.$(MAN3EXT) \
	  lib/Bio/Search/Iteration/IterationI.pm $(INST_MAN3DIR)/Bio::Search::Iteration::IterationI.$(MAN3EXT) \
	  lib/Bio/Search/Processor.pm $(INST_MAN3DIR)/Bio::Search::Processor.$(MAN3EXT) \
	  lib/Bio/Search/Result/BlastPullResult.pm $(INST_MAN3DIR)/Bio::Search::Result::BlastPullResult.$(MAN3EXT) 
	$(NOECHO) $(POD2MAN) --section=3 --perm_rw=$(PERM_RW) -u \
	  lib/Bio/Search/Result/BlastResult.pm $(INST_MAN3DIR)/Bio::Search::Result::BlastResult.$(MAN3EXT) \
	  lib/Bio/Search/Result/CrossMatchResult.pm $(INST_MAN3DIR)/Bio::Search::Result::CrossMatchResult.$(MAN3EXT) \
	  lib/Bio/Search/Result/GenericResult.pm $(INST_MAN3DIR)/Bio::Search::Result::GenericResult.$(MAN3EXT) \
	  lib/Bio/Search/Result/INFERNALResult.pm $(INST_MAN3DIR)/Bio::Search::Result::INFERNALResult.$(MAN3EXT) \
	  lib/Bio/Search/Result/PullResultI.pm $(INST_MAN3DIR)/Bio::Search::Result::PullResultI.$(MAN3EXT) \
	  lib/Bio/Search/Result/ResultFactory.pm $(INST_MAN3DIR)/Bio::Search::Result::ResultFactory.$(MAN3EXT) \
	  lib/Bio/Search/Result/ResultI.pm $(INST_MAN3DIR)/Bio::Search::Result::ResultI.$(MAN3EXT) \
	  lib/Bio/Search/Result/WABAResult.pm $(INST_MAN3DIR)/Bio::Search::Result::WABAResult.$(MAN3EXT) \
	  lib/Bio/Search/SearchUtils.pm $(INST_MAN3DIR)/Bio::Search::SearchUtils.$(MAN3EXT) \
	  lib/Bio/Search/StatisticsI.pm $(INST_MAN3DIR)/Bio::Search::StatisticsI.$(MAN3EXT) \
	  lib/Bio/Search/Tiling/MapTileUtils.pm $(INST_MAN3DIR)/Bio::Search::Tiling::MapTileUtils.$(MAN3EXT) \
	  lib/Bio/Search/Tiling/MapTiling.pm $(INST_MAN3DIR)/Bio::Search::Tiling::MapTiling.$(MAN3EXT) \
	  lib/Bio/Search/Tiling/TilingI.pm $(INST_MAN3DIR)/Bio::Search::Tiling::TilingI.$(MAN3EXT) \
	  lib/Bio/SearchIO.pm $(INST_MAN3DIR)/Bio::SearchIO.$(MAN3EXT) \
	  lib/Bio/SearchIO/EventHandlerI.pm $(INST_MAN3DIR)/Bio::SearchIO::EventHandlerI.$(MAN3EXT) \
	  lib/Bio/SearchIO/FastHitEventBuilder.pm $(INST_MAN3DIR)/Bio::SearchIO::FastHitEventBuilder.$(MAN3EXT) \
	  lib/Bio/SearchIO/IteratedSearchResultEventBuilder.pm $(INST_MAN3DIR)/Bio::SearchIO::IteratedSearchResultEventBuilder.$(MAN3EXT) \
	  lib/Bio/SearchIO/SearchResultEventBuilder.pm $(INST_MAN3DIR)/Bio::SearchIO::SearchResultEventBuilder.$(MAN3EXT) \
	  lib/Bio/SearchIO/SearchWriterI.pm $(INST_MAN3DIR)/Bio::SearchIO::SearchWriterI.$(MAN3EXT) \
	  lib/Bio/SearchIO/Writer/GbrowseGFF.pm $(INST_MAN3DIR)/Bio::SearchIO::Writer::GbrowseGFF.$(MAN3EXT) \
	  lib/Bio/SearchIO/Writer/HSPTableWriter.pm $(INST_MAN3DIR)/Bio::SearchIO::Writer::HSPTableWriter.$(MAN3EXT) \
	  lib/Bio/SearchIO/Writer/HTMLResultWriter.pm $(INST_MAN3DIR)/Bio::SearchIO::Writer::HTMLResultWriter.$(MAN3EXT) \
	  lib/Bio/SearchIO/Writer/HitTableWriter.pm $(INST_MAN3DIR)/Bio::SearchIO::Writer::HitTableWriter.$(MAN3EXT) \
	  lib/Bio/SearchIO/Writer/ResultTableWriter.pm $(INST_MAN3DIR)/Bio::SearchIO::Writer::ResultTableWriter.$(MAN3EXT) \
	  lib/Bio/SearchIO/Writer/TextResultWriter.pm $(INST_MAN3DIR)/Bio::SearchIO::Writer::TextResultWriter.$(MAN3EXT) \
	  lib/Bio/SearchIO/axt.pm $(INST_MAN3DIR)/Bio::SearchIO::axt.$(MAN3EXT) \
	  lib/Bio/SearchIO/blast.pm $(INST_MAN3DIR)/Bio::SearchIO::blast.$(MAN3EXT) \
	  lib/Bio/SearchIO/blast_pull.pm $(INST_MAN3DIR)/Bio::SearchIO::blast_pull.$(MAN3EXT) 
	$(NOECHO) $(POD2MAN) --section=3 --perm_rw=$(PERM_RW) -u \
	  lib/Bio/SearchIO/blasttable.pm $(INST_MAN3DIR)/Bio::SearchIO::blasttable.$(MAN3EXT) \
	  lib/Bio/SearchIO/cross_match.pm $(INST_MAN3DIR)/Bio::SearchIO::cross_match.$(MAN3EXT) \
	  lib/Bio/SearchIO/erpin.pm $(INST_MAN3DIR)/Bio::SearchIO::erpin.$(MAN3EXT) \
	  lib/Bio/SearchIO/exonerate.pm $(INST_MAN3DIR)/Bio::SearchIO::exonerate.$(MAN3EXT) \
	  lib/Bio/SearchIO/fasta.pm $(INST_MAN3DIR)/Bio::SearchIO::fasta.$(MAN3EXT) \
	  lib/Bio/SearchIO/gmap_f9.pm $(INST_MAN3DIR)/Bio::SearchIO::gmap_f9.$(MAN3EXT) \
	  lib/Bio/SearchIO/infernal.pm $(INST_MAN3DIR)/Bio::SearchIO::infernal.$(MAN3EXT) \
	  lib/Bio/SearchIO/megablast.pm $(INST_MAN3DIR)/Bio::SearchIO::megablast.$(MAN3EXT) \
	  lib/Bio/SearchIO/psl.pm $(INST_MAN3DIR)/Bio::SearchIO::psl.$(MAN3EXT) \
	  lib/Bio/SearchIO/rnamotif.pm $(INST_MAN3DIR)/Bio::SearchIO::rnamotif.$(MAN3EXT) \
	  lib/Bio/SearchIO/sim4.pm $(INST_MAN3DIR)/Bio::SearchIO::sim4.$(MAN3EXT) \
	  lib/Bio/SearchIO/waba.pm $(INST_MAN3DIR)/Bio::SearchIO::waba.$(MAN3EXT) \
	  lib/Bio/SearchIO/wise.pm $(INST_MAN3DIR)/Bio::SearchIO::wise.$(MAN3EXT) \
	  lib/Bio/Seq.pm $(INST_MAN3DIR)/Bio::Seq.$(MAN3EXT) \
	  lib/Bio/Seq/BaseSeqProcessor.pm $(INST_MAN3DIR)/Bio::Seq::BaseSeqProcessor.$(MAN3EXT) \
	  lib/Bio/Seq/EncodedSeq.pm $(INST_MAN3DIR)/Bio::Seq::EncodedSeq.$(MAN3EXT) \
	  lib/Bio/Seq/LargeLocatableSeq.pm $(INST_MAN3DIR)/Bio::Seq::LargeLocatableSeq.$(MAN3EXT) \
	  lib/Bio/Seq/LargePrimarySeq.pm $(INST_MAN3DIR)/Bio::Seq::LargePrimarySeq.$(MAN3EXT) \
	  lib/Bio/Seq/LargeSeq.pm $(INST_MAN3DIR)/Bio::Seq::LargeSeq.$(MAN3EXT) \
	  lib/Bio/Seq/LargeSeqI.pm $(INST_MAN3DIR)/Bio::Seq::LargeSeqI.$(MAN3EXT) \
	  lib/Bio/Seq/Meta.pm $(INST_MAN3DIR)/Bio::Seq::Meta.$(MAN3EXT) \
	  lib/Bio/Seq/Meta/Array.pm $(INST_MAN3DIR)/Bio::Seq::Meta::Array.$(MAN3EXT) \
	  lib/Bio/Seq/MetaI.pm $(INST_MAN3DIR)/Bio::Seq::MetaI.$(MAN3EXT) \
	  lib/Bio/Seq/PrimaryQual.pm $(INST_MAN3DIR)/Bio::Seq::PrimaryQual.$(MAN3EXT) \
	  lib/Bio/Seq/PrimedSeq.pm $(INST_MAN3DIR)/Bio::Seq::PrimedSeq.$(MAN3EXT) \
	  lib/Bio/Seq/QualI.pm $(INST_MAN3DIR)/Bio::Seq::QualI.$(MAN3EXT) \
	  lib/Bio/Seq/Quality.pm $(INST_MAN3DIR)/Bio::Seq::Quality.$(MAN3EXT) \
	  lib/Bio/Seq/RichSeq.pm $(INST_MAN3DIR)/Bio::Seq::RichSeq.$(MAN3EXT) \
	  lib/Bio/Seq/RichSeqI.pm $(INST_MAN3DIR)/Bio::Seq::RichSeqI.$(MAN3EXT) \
	  lib/Bio/Seq/SeqBuilder.pm $(INST_MAN3DIR)/Bio::Seq::SeqBuilder.$(MAN3EXT) \
	  lib/Bio/Seq/SeqFactory.pm $(INST_MAN3DIR)/Bio::Seq::SeqFactory.$(MAN3EXT) \
	  lib/Bio/Seq/SeqFastaSpeedFactory.pm $(INST_MAN3DIR)/Bio::Seq::SeqFastaSpeedFactory.$(MAN3EXT) \
	  lib/Bio/Seq/SequenceTrace.pm $(INST_MAN3DIR)/Bio::Seq::SequenceTrace.$(MAN3EXT) \
	  lib/Bio/Seq/SimulatedRead.pm $(INST_MAN3DIR)/Bio::Seq::SimulatedRead.$(MAN3EXT) \
	  lib/Bio/Seq/TraceI.pm $(INST_MAN3DIR)/Bio::Seq::TraceI.$(MAN3EXT) 
	$(NOECHO) $(POD2MAN) --section=3 --perm_rw=$(PERM_RW) -u \
	  lib/Bio/SeqAnalysisParserI.pm $(INST_MAN3DIR)/Bio::SeqAnalysisParserI.$(MAN3EXT) \
	  lib/Bio/SeqFeature/Amplicon.pm $(INST_MAN3DIR)/Bio::SeqFeature::Amplicon.$(MAN3EXT) \
	  lib/Bio/SeqFeature/AnnotationAdaptor.pm $(INST_MAN3DIR)/Bio::SeqFeature::AnnotationAdaptor.$(MAN3EXT) \
	  lib/Bio/SeqFeature/Collection.pm $(INST_MAN3DIR)/Bio::SeqFeature::Collection.$(MAN3EXT) \
	  lib/Bio/SeqFeature/CollectionI.pm $(INST_MAN3DIR)/Bio::SeqFeature::CollectionI.$(MAN3EXT) \
	  lib/Bio/SeqFeature/Computation.pm $(INST_MAN3DIR)/Bio::SeqFeature::Computation.$(MAN3EXT) \
	  lib/Bio/SeqFeature/FeaturePair.pm $(INST_MAN3DIR)/Bio::SeqFeature::FeaturePair.$(MAN3EXT) \
	  lib/Bio/SeqFeature/Gene/Exon.pm $(INST_MAN3DIR)/Bio::SeqFeature::Gene::Exon.$(MAN3EXT) \
	  lib/Bio/SeqFeature/Gene/ExonI.pm $(INST_MAN3DIR)/Bio::SeqFeature::Gene::ExonI.$(MAN3EXT) \
	  lib/Bio/SeqFeature/Gene/GeneStructure.pm $(INST_MAN3DIR)/Bio::SeqFeature::Gene::GeneStructure.$(MAN3EXT) \
	  lib/Bio/SeqFeature/Gene/GeneStructureI.pm $(INST_MAN3DIR)/Bio::SeqFeature::Gene::GeneStructureI.$(MAN3EXT) \
	  lib/Bio/SeqFeature/Gene/Intron.pm $(INST_MAN3DIR)/Bio::SeqFeature::Gene::Intron.$(MAN3EXT) \
	  lib/Bio/SeqFeature/Gene/NC_Feature.pm $(INST_MAN3DIR)/Bio::SeqFeature::Gene::NC_Feature.$(MAN3EXT) \
	  lib/Bio/SeqFeature/Gene/Poly_A_site.pm $(INST_MAN3DIR)/Bio::SeqFeature::Gene::Poly_A_site.$(MAN3EXT) \
	  lib/Bio/SeqFeature/Gene/Promoter.pm $(INST_MAN3DIR)/Bio::SeqFeature::Gene::Promoter.$(MAN3EXT) \
	  lib/Bio/SeqFeature/Gene/Transcript.pm $(INST_MAN3DIR)/Bio::SeqFeature::Gene::Transcript.$(MAN3EXT) \
	  lib/Bio/SeqFeature/Gene/TranscriptI.pm $(INST_MAN3DIR)/Bio::SeqFeature::Gene::TranscriptI.$(MAN3EXT) \
	  lib/Bio/SeqFeature/Gene/UTR.pm $(INST_MAN3DIR)/Bio::SeqFeature::Gene::UTR.$(MAN3EXT) \
	  lib/Bio/SeqFeature/Generic.pm $(INST_MAN3DIR)/Bio::SeqFeature::Generic.$(MAN3EXT) \
	  lib/Bio/SeqFeature/Lite.pm $(INST_MAN3DIR)/Bio::SeqFeature::Lite.$(MAN3EXT) \
	  lib/Bio/SeqFeature/PositionProxy.pm $(INST_MAN3DIR)/Bio::SeqFeature::PositionProxy.$(MAN3EXT) \
	  lib/Bio/SeqFeature/Primer.pm $(INST_MAN3DIR)/Bio::SeqFeature::Primer.$(MAN3EXT) \
	  lib/Bio/SeqFeature/Similarity.pm $(INST_MAN3DIR)/Bio::SeqFeature::Similarity.$(MAN3EXT) \
	  lib/Bio/SeqFeature/SimilarityPair.pm $(INST_MAN3DIR)/Bio::SeqFeature::SimilarityPair.$(MAN3EXT) \
	  lib/Bio/SeqFeature/SubSeq.pm $(INST_MAN3DIR)/Bio::SeqFeature::SubSeq.$(MAN3EXT) \
	  lib/Bio/SeqFeature/Tools/FeatureNamer.pm $(INST_MAN3DIR)/Bio::SeqFeature::Tools::FeatureNamer.$(MAN3EXT) \
	  lib/Bio/SeqFeature/Tools/IDHandler.pm $(INST_MAN3DIR)/Bio::SeqFeature::Tools::IDHandler.$(MAN3EXT) \
	  lib/Bio/SeqFeature/Tools/TypeMapper.pm $(INST_MAN3DIR)/Bio::SeqFeature::Tools::TypeMapper.$(MAN3EXT) \
	  lib/Bio/SeqFeature/Tools/Unflattener.pm $(INST_MAN3DIR)/Bio::SeqFeature::Tools::Unflattener.$(MAN3EXT) 
	$(NOECHO) $(POD2MAN) --section=3 --perm_rw=$(PERM_RW) -u \
	  lib/Bio/SeqFeature/TypedSeqFeatureI.pm $(INST_MAN3DIR)/Bio::SeqFeature::TypedSeqFeatureI.$(MAN3EXT) \
	  lib/Bio/SeqFeatureI.pm $(INST_MAN3DIR)/Bio::SeqFeatureI.$(MAN3EXT) \
	  lib/Bio/SeqI.pm $(INST_MAN3DIR)/Bio::SeqI.$(MAN3EXT) \
	  lib/Bio/SeqIO.pm $(INST_MAN3DIR)/Bio::SeqIO.$(MAN3EXT) \
	  lib/Bio/SeqIO/FTHelper.pm $(INST_MAN3DIR)/Bio::SeqIO::FTHelper.$(MAN3EXT) \
	  lib/Bio/SeqIO/Handler/GenericRichSeqHandler.pm $(INST_MAN3DIR)/Bio::SeqIO::Handler::GenericRichSeqHandler.$(MAN3EXT) \
	  lib/Bio/SeqIO/MultiFile.pm $(INST_MAN3DIR)/Bio::SeqIO::MultiFile.$(MAN3EXT) \
	  lib/Bio/SeqIO/ace.pm $(INST_MAN3DIR)/Bio::SeqIO::ace.$(MAN3EXT) \
	  lib/Bio/SeqIO/asciitree.pm $(INST_MAN3DIR)/Bio::SeqIO::asciitree.$(MAN3EXT) \
	  lib/Bio/SeqIO/bsml.pm $(INST_MAN3DIR)/Bio::SeqIO::bsml.$(MAN3EXT) \
	  lib/Bio/SeqIO/bsml_sax.pm $(INST_MAN3DIR)/Bio::SeqIO::bsml_sax.$(MAN3EXT) \
	  lib/Bio/SeqIO/embl.pm $(INST_MAN3DIR)/Bio::SeqIO::embl.$(MAN3EXT) \
	  lib/Bio/SeqIO/embldriver.pm $(INST_MAN3DIR)/Bio::SeqIO::embldriver.$(MAN3EXT) \
	  lib/Bio/SeqIO/fasta.pm $(INST_MAN3DIR)/Bio::SeqIO::fasta.$(MAN3EXT) \
	  lib/Bio/SeqIO/fastq.pm $(INST_MAN3DIR)/Bio::SeqIO::fastq.$(MAN3EXT) \
	  lib/Bio/SeqIO/game.pm $(INST_MAN3DIR)/Bio::SeqIO::game.$(MAN3EXT) \
	  lib/Bio/SeqIO/game/featHandler.pm $(INST_MAN3DIR)/Bio::SeqIO::game::featHandler.$(MAN3EXT) \
	  lib/Bio/SeqIO/game/gameHandler.pm $(INST_MAN3DIR)/Bio::SeqIO::game::gameHandler.$(MAN3EXT) \
	  lib/Bio/SeqIO/game/gameSubs.pm $(INST_MAN3DIR)/Bio::SeqIO::game::gameSubs.$(MAN3EXT) \
	  lib/Bio/SeqIO/game/gameWriter.pm $(INST_MAN3DIR)/Bio::SeqIO::game::gameWriter.$(MAN3EXT) \
	  lib/Bio/SeqIO/game/seqHandler.pm $(INST_MAN3DIR)/Bio::SeqIO::game::seqHandler.$(MAN3EXT) \
	  lib/Bio/SeqIO/gbdriver.pm $(INST_MAN3DIR)/Bio::SeqIO::gbdriver.$(MAN3EXT) \
	  lib/Bio/SeqIO/gbxml.pm $(INST_MAN3DIR)/Bio::SeqIO::gbxml.$(MAN3EXT) \
	  lib/Bio/SeqIO/gcg.pm $(INST_MAN3DIR)/Bio::SeqIO::gcg.$(MAN3EXT) \
	  lib/Bio/SeqIO/genbank.pm $(INST_MAN3DIR)/Bio::SeqIO::genbank.$(MAN3EXT) \
	  lib/Bio/SeqIO/kegg.pm $(INST_MAN3DIR)/Bio::SeqIO::kegg.$(MAN3EXT) \
	  lib/Bio/SeqIO/largefasta.pm $(INST_MAN3DIR)/Bio::SeqIO::largefasta.$(MAN3EXT) \
	  lib/Bio/SeqIO/locuslink.pm $(INST_MAN3DIR)/Bio::SeqIO::locuslink.$(MAN3EXT) \
	  lib/Bio/SeqIO/mbsout.pm $(INST_MAN3DIR)/Bio::SeqIO::mbsout.$(MAN3EXT) \
	  lib/Bio/SeqIO/metafasta.pm $(INST_MAN3DIR)/Bio::SeqIO::metafasta.$(MAN3EXT) \
	  lib/Bio/SeqIO/msout.pm $(INST_MAN3DIR)/Bio::SeqIO::msout.$(MAN3EXT) \
	  lib/Bio/SeqIO/phd.pm $(INST_MAN3DIR)/Bio::SeqIO::phd.$(MAN3EXT) \
	  lib/Bio/SeqIO/pir.pm $(INST_MAN3DIR)/Bio::SeqIO::pir.$(MAN3EXT) \
	  lib/Bio/SeqIO/qual.pm $(INST_MAN3DIR)/Bio::SeqIO::qual.$(MAN3EXT) \
	  lib/Bio/SeqIO/raw.pm $(INST_MAN3DIR)/Bio::SeqIO::raw.$(MAN3EXT) \
	  lib/Bio/SeqIO/scf.pm $(INST_MAN3DIR)/Bio::SeqIO::scf.$(MAN3EXT) 
	$(NOECHO) $(POD2MAN) --section=3 --perm_rw=$(PERM_RW) -u \
	  lib/Bio/SeqIO/seqxml.pm $(INST_MAN3DIR)/Bio::SeqIO::seqxml.$(MAN3EXT) \
	  lib/Bio/SeqIO/swiss.pm $(INST_MAN3DIR)/Bio::SeqIO::swiss.$(MAN3EXT) \
	  lib/Bio/SeqIO/swissdriver.pm $(INST_MAN3DIR)/Bio::SeqIO::swissdriver.$(MAN3EXT) \
	  lib/Bio/SeqIO/tab.pm $(INST_MAN3DIR)/Bio::SeqIO::tab.$(MAN3EXT) \
	  lib/Bio/SeqIO/table.pm $(INST_MAN3DIR)/Bio::SeqIO::table.$(MAN3EXT) \
	  lib/Bio/SeqIO/tigr.pm $(INST_MAN3DIR)/Bio::SeqIO::tigr.$(MAN3EXT) \
	  lib/Bio/SeqIO/tigrxml.pm $(INST_MAN3DIR)/Bio::SeqIO::tigrxml.$(MAN3EXT) \
	  lib/Bio/SeqIO/tinyseq.pm $(INST_MAN3DIR)/Bio::SeqIO::tinyseq.$(MAN3EXT) \
	  lib/Bio/SeqIO/tinyseq/tinyseqHandler.pm $(INST_MAN3DIR)/Bio::SeqIO::tinyseq::tinyseqHandler.$(MAN3EXT) \
	  lib/Bio/SeqUtils.pm $(INST_MAN3DIR)/Bio::SeqUtils.$(MAN3EXT) \
	  lib/Bio/SimpleAlign.pm $(INST_MAN3DIR)/Bio::SimpleAlign.$(MAN3EXT) \
	  lib/Bio/SimpleAnalysisI.pm $(INST_MAN3DIR)/Bio::SimpleAnalysisI.$(MAN3EXT) \
	  lib/Bio/Species.pm $(INST_MAN3DIR)/Bio::Species.$(MAN3EXT) \
	  lib/Bio/Taxon.pm $(INST_MAN3DIR)/Bio::Taxon.$(MAN3EXT) \
	  lib/Bio/Tools/Alignment/Consed.pm $(INST_MAN3DIR)/Bio::Tools::Alignment::Consed.$(MAN3EXT) \
	  lib/Bio/Tools/Alignment/Trim.pm $(INST_MAN3DIR)/Bio::Tools::Alignment::Trim.$(MAN3EXT) \
	  lib/Bio/Tools/AmpliconSearch.pm $(INST_MAN3DIR)/Bio::Tools::AmpliconSearch.$(MAN3EXT) \
	  lib/Bio/Tools/Analysis/SimpleAnalysisBase.pm $(INST_MAN3DIR)/Bio::Tools::Analysis::SimpleAnalysisBase.$(MAN3EXT) \
	  lib/Bio/Tools/AnalysisResult.pm $(INST_MAN3DIR)/Bio::Tools::AnalysisResult.$(MAN3EXT) \
	  lib/Bio/Tools/Blat.pm $(INST_MAN3DIR)/Bio::Tools::Blat.$(MAN3EXT) \
	  lib/Bio/Tools/CodonTable.pm $(INST_MAN3DIR)/Bio::Tools::CodonTable.$(MAN3EXT) \
	  lib/Bio/Tools/Coil.pm $(INST_MAN3DIR)/Bio::Tools::Coil.$(MAN3EXT) \
	  lib/Bio/Tools/ECnumber.pm $(INST_MAN3DIR)/Bio::Tools::ECnumber.$(MAN3EXT) \
	  lib/Bio/Tools/EMBOSS/Palindrome.pm $(INST_MAN3DIR)/Bio::Tools::EMBOSS::Palindrome.$(MAN3EXT) \
	  lib/Bio/Tools/EPCR.pm $(INST_MAN3DIR)/Bio::Tools::EPCR.$(MAN3EXT) \
	  lib/Bio/Tools/ESTScan.pm $(INST_MAN3DIR)/Bio::Tools::ESTScan.$(MAN3EXT) \
	  lib/Bio/Tools/Eponine.pm $(INST_MAN3DIR)/Bio::Tools::Eponine.$(MAN3EXT) \
	  lib/Bio/Tools/Est2Genome.pm $(INST_MAN3DIR)/Bio::Tools::Est2Genome.$(MAN3EXT) \
	  lib/Bio/Tools/Fgenesh.pm $(INST_MAN3DIR)/Bio::Tools::Fgenesh.$(MAN3EXT) \
	  lib/Bio/Tools/FootPrinter.pm $(INST_MAN3DIR)/Bio::Tools::FootPrinter.$(MAN3EXT) \
	  lib/Bio/Tools/GFF.pm $(INST_MAN3DIR)/Bio::Tools::GFF.$(MAN3EXT) \
	  lib/Bio/Tools/Geneid.pm $(INST_MAN3DIR)/Bio::Tools::Geneid.$(MAN3EXT) \
	  lib/Bio/Tools/Genemark.pm $(INST_MAN3DIR)/Bio::Tools::Genemark.$(MAN3EXT) \
	  lib/Bio/Tools/Genewise.pm $(INST_MAN3DIR)/Bio::Tools::Genewise.$(MAN3EXT) \
	  lib/Bio/Tools/Genomewise.pm $(INST_MAN3DIR)/Bio::Tools::Genomewise.$(MAN3EXT) 
	$(NOECHO) $(POD2MAN) --section=3 --perm_rw=$(PERM_RW) -u \
	  lib/Bio/Tools/Genscan.pm $(INST_MAN3DIR)/Bio::Tools::Genscan.$(MAN3EXT) \
	  lib/Bio/Tools/Glimmer.pm $(INST_MAN3DIR)/Bio::Tools::Glimmer.$(MAN3EXT) \
	  lib/Bio/Tools/Grail.pm $(INST_MAN3DIR)/Bio::Tools::Grail.$(MAN3EXT) \
	  lib/Bio/Tools/GuessSeqFormat.pm $(INST_MAN3DIR)/Bio::Tools::GuessSeqFormat.$(MAN3EXT) \
	  lib/Bio/Tools/IUPAC.pm $(INST_MAN3DIR)/Bio::Tools::IUPAC.$(MAN3EXT) \
	  lib/Bio/Tools/Lucy.pm $(INST_MAN3DIR)/Bio::Tools::Lucy.$(MAN3EXT) \
	  lib/Bio/Tools/MZEF.pm $(INST_MAN3DIR)/Bio::Tools::MZEF.$(MAN3EXT) \
	  lib/Bio/Tools/Match.pm $(INST_MAN3DIR)/Bio::Tools::Match.$(MAN3EXT) \
	  lib/Bio/Tools/OddCodes.pm $(INST_MAN3DIR)/Bio::Tools::OddCodes.$(MAN3EXT) \
	  lib/Bio/Tools/Phylo/Gerp.pm $(INST_MAN3DIR)/Bio::Tools::Phylo::Gerp.$(MAN3EXT) \
	  lib/Bio/Tools/Phylo/Molphy.pm $(INST_MAN3DIR)/Bio::Tools::Phylo::Molphy.$(MAN3EXT) \
	  lib/Bio/Tools/Phylo/Molphy/Result.pm $(INST_MAN3DIR)/Bio::Tools::Phylo::Molphy::Result.$(MAN3EXT) \
	  lib/Bio/Tools/Phylo/Phylip/ProtDist.pm $(INST_MAN3DIR)/Bio::Tools::Phylo::Phylip::ProtDist.$(MAN3EXT) \
	  lib/Bio/Tools/Prediction/Exon.pm $(INST_MAN3DIR)/Bio::Tools::Prediction::Exon.$(MAN3EXT) \
	  lib/Bio/Tools/Prediction/Gene.pm $(INST_MAN3DIR)/Bio::Tools::Prediction::Gene.$(MAN3EXT) \
	  lib/Bio/Tools/Primer/Assessor/Base.pm $(INST_MAN3DIR)/Bio::Tools::Primer::Assessor::Base.$(MAN3EXT) \
	  lib/Bio/Tools/Primer/AssessorI.pm $(INST_MAN3DIR)/Bio::Tools::Primer::AssessorI.$(MAN3EXT) \
	  lib/Bio/Tools/Primer/Feature.pm $(INST_MAN3DIR)/Bio::Tools::Primer::Feature.$(MAN3EXT) \
	  lib/Bio/Tools/Primer/Pair.pm $(INST_MAN3DIR)/Bio::Tools::Primer::Pair.$(MAN3EXT) \
	  lib/Bio/Tools/Primer3.pm $(INST_MAN3DIR)/Bio::Tools::Primer3.$(MAN3EXT) \
	  lib/Bio/Tools/Prints.pm $(INST_MAN3DIR)/Bio::Tools::Prints.$(MAN3EXT) \
	  lib/Bio/Tools/Profile.pm $(INST_MAN3DIR)/Bio::Tools::Profile.$(MAN3EXT) \
	  lib/Bio/Tools/Promoterwise.pm $(INST_MAN3DIR)/Bio::Tools::Promoterwise.$(MAN3EXT) \
	  lib/Bio/Tools/PrositeScan.pm $(INST_MAN3DIR)/Bio::Tools::PrositeScan.$(MAN3EXT) \
	  lib/Bio/Tools/Pseudowise.pm $(INST_MAN3DIR)/Bio::Tools::Pseudowise.$(MAN3EXT) \
	  lib/Bio/Tools/QRNA.pm $(INST_MAN3DIR)/Bio::Tools::QRNA.$(MAN3EXT) \
	  lib/Bio/Tools/RandomDistFunctions.pm $(INST_MAN3DIR)/Bio::Tools::RandomDistFunctions.$(MAN3EXT) \
	  lib/Bio/Tools/RepeatMasker.pm $(INST_MAN3DIR)/Bio::Tools::RepeatMasker.$(MAN3EXT) \
	  lib/Bio/Tools/Run/Analysis.pm $(INST_MAN3DIR)/Bio::Tools::Run::Analysis.$(MAN3EXT) \
	  lib/Bio/Tools/Run/AnalysisFactory.pm $(INST_MAN3DIR)/Bio::Tools::Run::AnalysisFactory.$(MAN3EXT) \
	  lib/Bio/Tools/Run/GenericParameters.pm $(INST_MAN3DIR)/Bio::Tools::Run::GenericParameters.$(MAN3EXT) \
	  lib/Bio/Tools/Run/ParametersI.pm $(INST_MAN3DIR)/Bio::Tools::Run::ParametersI.$(MAN3EXT) 
	$(NOECHO) $(POD2MAN) --section=3 --perm_rw=$(PERM_RW) -u \
	  lib/Bio/Tools/Run/Phylo/PhyloBase.pm $(INST_MAN3DIR)/Bio::Tools::Run::Phylo::PhyloBase.$(MAN3EXT) \
	  lib/Bio/Tools/Run/WrapperBase.pm $(INST_MAN3DIR)/Bio::Tools::Run::WrapperBase.$(MAN3EXT) \
	  lib/Bio/Tools/Run/WrapperBase/CommandExts.pm $(INST_MAN3DIR)/Bio::Tools::Run::WrapperBase::CommandExts.$(MAN3EXT) \
	  lib/Bio/Tools/Seg.pm $(INST_MAN3DIR)/Bio::Tools::Seg.$(MAN3EXT) \
	  lib/Bio/Tools/SeqPattern.pm $(INST_MAN3DIR)/Bio::Tools::SeqPattern.$(MAN3EXT) \
	  lib/Bio/Tools/SeqPattern/Backtranslate.pm $(INST_MAN3DIR)/Bio::Tools::SeqPattern::Backtranslate.$(MAN3EXT) \
	  lib/Bio/Tools/SeqStats.pm $(INST_MAN3DIR)/Bio::Tools::SeqStats.$(MAN3EXT) \
	  lib/Bio/Tools/SeqWords.pm $(INST_MAN3DIR)/Bio::Tools::SeqWords.$(MAN3EXT) \
	  lib/Bio/Tools/Sigcleave.pm $(INST_MAN3DIR)/Bio::Tools::Sigcleave.$(MAN3EXT) \
	  lib/Bio/Tools/Signalp.pm $(INST_MAN3DIR)/Bio::Tools::Signalp.$(MAN3EXT) \
	  lib/Bio/Tools/Signalp/ExtendedSignalp.pm $(INST_MAN3DIR)/Bio::Tools::Signalp::ExtendedSignalp.$(MAN3EXT) \
	  lib/Bio/Tools/Sim4/Exon.pm $(INST_MAN3DIR)/Bio::Tools::Sim4::Exon.$(MAN3EXT) \
	  lib/Bio/Tools/Sim4/Results.pm $(INST_MAN3DIR)/Bio::Tools::Sim4::Results.$(MAN3EXT) \
	  lib/Bio/Tools/Spidey/Exon.pm $(INST_MAN3DIR)/Bio::Tools::Spidey::Exon.$(MAN3EXT) \
	  lib/Bio/Tools/Spidey/Results.pm $(INST_MAN3DIR)/Bio::Tools::Spidey::Results.$(MAN3EXT) \
	  lib/Bio/Tools/TandemRepeatsFinder.pm $(INST_MAN3DIR)/Bio::Tools::TandemRepeatsFinder.$(MAN3EXT) \
	  lib/Bio/Tools/TargetP.pm $(INST_MAN3DIR)/Bio::Tools::TargetP.$(MAN3EXT) \
	  lib/Bio/Tools/Tmhmm.pm $(INST_MAN3DIR)/Bio::Tools::Tmhmm.$(MAN3EXT) \
	  lib/Bio/Tools/ipcress.pm $(INST_MAN3DIR)/Bio::Tools::ipcress.$(MAN3EXT) \
	  lib/Bio/Tools/isPcr.pm $(INST_MAN3DIR)/Bio::Tools::isPcr.$(MAN3EXT) \
	  lib/Bio/Tools/pICalculator.pm $(INST_MAN3DIR)/Bio::Tools::pICalculator.$(MAN3EXT) \
	  lib/Bio/Tools/tRNAscanSE.pm $(INST_MAN3DIR)/Bio::Tools::tRNAscanSE.$(MAN3EXT) \
	  lib/Bio/Tree/AnnotatableNode.pm $(INST_MAN3DIR)/Bio::Tree::AnnotatableNode.$(MAN3EXT) \
	  lib/Bio/Tree/Compatible.pm $(INST_MAN3DIR)/Bio::Tree::Compatible.$(MAN3EXT) \
	  lib/Bio/Tree/DistanceFactory.pm $(INST_MAN3DIR)/Bio::Tree::DistanceFactory.$(MAN3EXT) \
	  lib/Bio/Tree/Node.pm $(INST_MAN3DIR)/Bio::Tree::Node.$(MAN3EXT) \
	  lib/Bio/Tree/NodeI.pm $(INST_MAN3DIR)/Bio::Tree::NodeI.$(MAN3EXT) \
	  lib/Bio/Tree/NodeNHX.pm $(INST_MAN3DIR)/Bio::Tree::NodeNHX.$(MAN3EXT) \
	  lib/Bio/Tree/RandomFactory.pm $(INST_MAN3DIR)/Bio::Tree::RandomFactory.$(MAN3EXT) \
	  lib/Bio/Tree/Statistics.pm $(INST_MAN3DIR)/Bio::Tree::Statistics.$(MAN3EXT) \
	  lib/Bio/Tree/Tree.pm $(INST_MAN3DIR)/Bio::Tree::Tree.$(MAN3EXT) \
	  lib/Bio/Tree/TreeFunctionsI.pm $(INST_MAN3DIR)/Bio::Tree::TreeFunctionsI.$(MAN3EXT) \
	  lib/Bio/Tree/TreeI.pm $(INST_MAN3DIR)/Bio::Tree::TreeI.$(MAN3EXT) 
	$(NOECHO) $(POD2MAN) --section=3 --perm_rw=$(PERM_RW) -u \
	  lib/Bio/TreeIO.pm $(INST_MAN3DIR)/Bio::TreeIO.$(MAN3EXT) \
	  lib/Bio/TreeIO/NewickParser.pm $(INST_MAN3DIR)/Bio::TreeIO::NewickParser.$(MAN3EXT) \
	  lib/Bio/TreeIO/TreeEventBuilder.pm $(INST_MAN3DIR)/Bio::TreeIO::TreeEventBuilder.$(MAN3EXT) \
	  lib/Bio/TreeIO/cluster.pm $(INST_MAN3DIR)/Bio::TreeIO::cluster.$(MAN3EXT) \
	  lib/Bio/TreeIO/lintree.pm $(INST_MAN3DIR)/Bio::TreeIO::lintree.$(MAN3EXT) \
	  lib/Bio/TreeIO/newick.pm $(INST_MAN3DIR)/Bio::TreeIO::newick.$(MAN3EXT) \
	  lib/Bio/TreeIO/nexus.pm $(INST_MAN3DIR)/Bio::TreeIO::nexus.$(MAN3EXT) \
	  lib/Bio/TreeIO/nhx.pm $(INST_MAN3DIR)/Bio::TreeIO::nhx.$(MAN3EXT) \
	  lib/Bio/TreeIO/pag.pm $(INST_MAN3DIR)/Bio::TreeIO::pag.$(MAN3EXT) \
	  lib/Bio/TreeIO/phyloxml.pm $(INST_MAN3DIR)/Bio::TreeIO::phyloxml.$(MAN3EXT) \
	  lib/Bio/TreeIO/tabtree.pm $(INST_MAN3DIR)/Bio::TreeIO::tabtree.$(MAN3EXT) \
	  lib/Bio/UpdateableSeqI.pm $(INST_MAN3DIR)/Bio::UpdateableSeqI.$(MAN3EXT) \
	  lib/Bio/WebAgent.pm $(INST_MAN3DIR)/Bio::WebAgent.$(MAN3EXT) \
	  lib/BioPerl.pm $(INST_MAN3DIR)/BioPerl.$(MAN3EXT) 




# --- MakeMaker processPL section:


# --- MakeMaker installbin section:

EXE_FILES = bin/bp_aacomp bin/bp_bioflat_index bin/bp_biogetseq bin/bp_dbsplit bin/bp_extract_feature_seq bin/bp_fastam9_to_table bin/bp_fetch bin/bp_filter_search bin/bp_find-blast-matches bin/bp_gccalc bin/bp_genbank2gff3 bin/bp_index bin/bp_local_taxonomydb_query bin/bp_make_mrna_protein bin/bp_mask_by_search bin/bp_mrtrans bin/bp_mutate bin/bp_nexus2nh bin/bp_nrdb bin/bp_oligo_count bin/bp_process_gadfly bin/bp_process_sgd bin/bp_revtrans-motif bin/bp_search2alnblocks bin/bp_search2gff bin/bp_search2table bin/bp_search2tribe bin/bp_seq_length bin/bp_seqconvert bin/bp_seqcut bin/bp_seqpart bin/bp_seqret bin/bp_seqretsplit bin/bp_split_seq bin/bp_sreformat bin/bp_taxid4species bin/bp_taxonomy2tree bin/bp_translate_seq bin/bp_tree2pag bin/bp_unflatten_seq

pure_all :: $(INST_SCRIPT)/bp_genbank2gff3 $(INST_SCRIPT)/bp_filter_search $(INST_SCRIPT)/bp_taxonomy2tree $(INST_SCRIPT)/bp_search2alnblocks $(INST_SCRIPT)/bp_nrdb $(INST_SCRIPT)/bp_search2table $(INST_SCRIPT)/bp_oligo_count $(INST_SCRIPT)/bp_translate_seq $(INST_SCRIPT)/bp_split_seq $(INST_SCRIPT)/bp_bioflat_index $(INST_SCRIPT)/bp_mask_by_search $(INST_SCRIPT)/bp_seqretsplit $(INST_SCRIPT)/bp_search2gff $(INST_SCRIPT)/bp_gccalc $(INST_SCRIPT)/bp_process_sgd $(INST_SCRIPT)/bp_process_gadfly $(INST_SCRIPT)/bp_nexus2nh $(INST_SCRIPT)/bp_seqconvert $(INST_SCRIPT)/bp_aacomp $(INST_SCRIPT)/bp_mutate $(INST_SCRIPT)/bp_seq_length $(INST_SCRIPT)/bp_seqcut $(INST_SCRIPT)/bp_find-blast-matches $(INST_SCRIPT)/bp_revtrans-motif $(INST_SCRIPT)/bp_seqpart $(INST_SCRIPT)/bp_seqret $(INST_SCRIPT)/bp_mrtrans $(INST_SCRIPT)/bp_make_mrna_protein $(INST_SCRIPT)/bp_tree2pag $(INST_SCRIPT)/bp_extract_feature_seq $(INST_SCRIPT)/bp_fastam9_to_table $(INST_SCRIPT)/bp_unflatten_seq $(INST_SCRIPT)/bp_taxid4species $(INST_SCRIPT)/bp_sreformat $(INST_SCRIPT)/bp_local_taxonomydb_query $(INST_SCRIPT)/bp_dbsplit $(INST_SCRIPT)/bp_index $(INST_SCRIPT)/bp_search2tribe $(INST_SCRIPT)/bp_biogetseq $(INST_SCRIPT)/bp_fetch
	$(NOECHO) $(NOOP)

realclean ::
	$(RM_F) \
	  $(INST_SCRIPT)/bp_genbank2gff3 $(INST_SCRIPT)/bp_filter_search \
	  $(INST_SCRIPT)/bp_taxonomy2tree $(INST_SCRIPT)/bp_search2alnblocks \
	  $(INST_SCRIPT)/bp_nrdb $(INST_SCRIPT)/bp_search2table \
	  $(INST_SCRIPT)/bp_oligo_count $(INST_SCRIPT)/bp_translate_seq \
	  $(INST_SCRIPT)/bp_split_seq $(INST_SCRIPT)/bp_bioflat_index \
	  $(INST_SCRIPT)/bp_mask_by_search $(INST_SCRIPT)/bp_seqretsplit \
	  $(INST_SCRIPT)/bp_search2gff $(INST_SCRIPT)/bp_gccalc \
	  $(INST_SCRIPT)/bp_process_sgd $(INST_SCRIPT)/bp_process_gadfly \
	  $(INST_SCRIPT)/bp_nexus2nh $(INST_SCRIPT)/bp_seqconvert \
	  $(INST_SCRIPT)/bp_aacomp $(INST_SCRIPT)/bp_mutate \
	  $(INST_SCRIPT)/bp_seq_length $(INST_SCRIPT)/bp_seqcut \
	  $(INST_SCRIPT)/bp_find-blast-matches $(INST_SCRIPT)/bp_revtrans-motif \
	  $(INST_SCRIPT)/bp_seqpart $(INST_SCRIPT)/bp_seqret \
	  $(INST_SCRIPT)/bp_mrtrans $(INST_SCRIPT)/bp_make_mrna_protein \
	  $(INST_SCRIPT)/bp_tree2pag $(INST_SCRIPT)/bp_extract_feature_seq \
	  $(INST_SCRIPT)/bp_fastam9_to_table $(INST_SCRIPT)/bp_unflatten_seq \
	  $(INST_SCRIPT)/bp_taxid4species $(INST_SCRIPT)/bp_sreformat \
	  $(INST_SCRIPT)/bp_local_taxonomydb_query $(INST_SCRIPT)/bp_dbsplit \
	  $(INST_SCRIPT)/bp_index $(INST_SCRIPT)/bp_search2tribe \
	  $(INST_SCRIPT)/bp_biogetseq $(INST_SCRIPT)/bp_fetch 

$(INST_SCRIPT)/bp_genbank2gff3 : bin/bp_genbank2gff3 $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_genbank2gff3
	$(CP) bin/bp_genbank2gff3 $(INST_SCRIPT)/bp_genbank2gff3
	$(FIXIN) $(INST_SCRIPT)/bp_genbank2gff3
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_genbank2gff3

$(INST_SCRIPT)/bp_filter_search : bin/bp_filter_search $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_filter_search
	$(CP) bin/bp_filter_search $(INST_SCRIPT)/bp_filter_search
	$(FIXIN) $(INST_SCRIPT)/bp_filter_search
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_filter_search

$(INST_SCRIPT)/bp_taxonomy2tree : bin/bp_taxonomy2tree $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_taxonomy2tree
	$(CP) bin/bp_taxonomy2tree $(INST_SCRIPT)/bp_taxonomy2tree
	$(FIXIN) $(INST_SCRIPT)/bp_taxonomy2tree
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_taxonomy2tree

$(INST_SCRIPT)/bp_search2alnblocks : bin/bp_search2alnblocks $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_search2alnblocks
	$(CP) bin/bp_search2alnblocks $(INST_SCRIPT)/bp_search2alnblocks
	$(FIXIN) $(INST_SCRIPT)/bp_search2alnblocks
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_search2alnblocks

$(INST_SCRIPT)/bp_nrdb : bin/bp_nrdb $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_nrdb
	$(CP) bin/bp_nrdb $(INST_SCRIPT)/bp_nrdb
	$(FIXIN) $(INST_SCRIPT)/bp_nrdb
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_nrdb

$(INST_SCRIPT)/bp_search2table : bin/bp_search2table $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_search2table
	$(CP) bin/bp_search2table $(INST_SCRIPT)/bp_search2table
	$(FIXIN) $(INST_SCRIPT)/bp_search2table
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_search2table

$(INST_SCRIPT)/bp_oligo_count : bin/bp_oligo_count $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_oligo_count
	$(CP) bin/bp_oligo_count $(INST_SCRIPT)/bp_oligo_count
	$(FIXIN) $(INST_SCRIPT)/bp_oligo_count
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_oligo_count

$(INST_SCRIPT)/bp_translate_seq : bin/bp_translate_seq $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_translate_seq
	$(CP) bin/bp_translate_seq $(INST_SCRIPT)/bp_translate_seq
	$(FIXIN) $(INST_SCRIPT)/bp_translate_seq
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_translate_seq

$(INST_SCRIPT)/bp_split_seq : bin/bp_split_seq $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_split_seq
	$(CP) bin/bp_split_seq $(INST_SCRIPT)/bp_split_seq
	$(FIXIN) $(INST_SCRIPT)/bp_split_seq
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_split_seq

$(INST_SCRIPT)/bp_bioflat_index : bin/bp_bioflat_index $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_bioflat_index
	$(CP) bin/bp_bioflat_index $(INST_SCRIPT)/bp_bioflat_index
	$(FIXIN) $(INST_SCRIPT)/bp_bioflat_index
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_bioflat_index

$(INST_SCRIPT)/bp_mask_by_search : bin/bp_mask_by_search $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_mask_by_search
	$(CP) bin/bp_mask_by_search $(INST_SCRIPT)/bp_mask_by_search
	$(FIXIN) $(INST_SCRIPT)/bp_mask_by_search
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_mask_by_search

$(INST_SCRIPT)/bp_seqretsplit : bin/bp_seqretsplit $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_seqretsplit
	$(CP) bin/bp_seqretsplit $(INST_SCRIPT)/bp_seqretsplit
	$(FIXIN) $(INST_SCRIPT)/bp_seqretsplit
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_seqretsplit

$(INST_SCRIPT)/bp_search2gff : bin/bp_search2gff $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_search2gff
	$(CP) bin/bp_search2gff $(INST_SCRIPT)/bp_search2gff
	$(FIXIN) $(INST_SCRIPT)/bp_search2gff
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_search2gff

$(INST_SCRIPT)/bp_gccalc : bin/bp_gccalc $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_gccalc
	$(CP) bin/bp_gccalc $(INST_SCRIPT)/bp_gccalc
	$(FIXIN) $(INST_SCRIPT)/bp_gccalc
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_gccalc

$(INST_SCRIPT)/bp_process_sgd : bin/bp_process_sgd $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_process_sgd
	$(CP) bin/bp_process_sgd $(INST_SCRIPT)/bp_process_sgd
	$(FIXIN) $(INST_SCRIPT)/bp_process_sgd
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_process_sgd

$(INST_SCRIPT)/bp_process_gadfly : bin/bp_process_gadfly $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_process_gadfly
	$(CP) bin/bp_process_gadfly $(INST_SCRIPT)/bp_process_gadfly
	$(FIXIN) $(INST_SCRIPT)/bp_process_gadfly
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_process_gadfly

$(INST_SCRIPT)/bp_nexus2nh : bin/bp_nexus2nh $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_nexus2nh
	$(CP) bin/bp_nexus2nh $(INST_SCRIPT)/bp_nexus2nh
	$(FIXIN) $(INST_SCRIPT)/bp_nexus2nh
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_nexus2nh

$(INST_SCRIPT)/bp_seqconvert : bin/bp_seqconvert $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_seqconvert
	$(CP) bin/bp_seqconvert $(INST_SCRIPT)/bp_seqconvert
	$(FIXIN) $(INST_SCRIPT)/bp_seqconvert
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_seqconvert

$(INST_SCRIPT)/bp_aacomp : bin/bp_aacomp $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_aacomp
	$(CP) bin/bp_aacomp $(INST_SCRIPT)/bp_aacomp
	$(FIXIN) $(INST_SCRIPT)/bp_aacomp
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_aacomp

$(INST_SCRIPT)/bp_mutate : bin/bp_mutate $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_mutate
	$(CP) bin/bp_mutate $(INST_SCRIPT)/bp_mutate
	$(FIXIN) $(INST_SCRIPT)/bp_mutate
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_mutate

$(INST_SCRIPT)/bp_seq_length : bin/bp_seq_length $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_seq_length
	$(CP) bin/bp_seq_length $(INST_SCRIPT)/bp_seq_length
	$(FIXIN) $(INST_SCRIPT)/bp_seq_length
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_seq_length

$(INST_SCRIPT)/bp_seqcut : bin/bp_seqcut $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_seqcut
	$(CP) bin/bp_seqcut $(INST_SCRIPT)/bp_seqcut
	$(FIXIN) $(INST_SCRIPT)/bp_seqcut
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_seqcut

$(INST_SCRIPT)/bp_find-blast-matches : bin/bp_find-blast-matches $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_find-blast-matches
	$(CP) bin/bp_find-blast-matches $(INST_SCRIPT)/bp_find-blast-matches
	$(FIXIN) $(INST_SCRIPT)/bp_find-blast-matches
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_find-blast-matches

$(INST_SCRIPT)/bp_revtrans-motif : bin/bp_revtrans-motif $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_revtrans-motif
	$(CP) bin/bp_revtrans-motif $(INST_SCRIPT)/bp_revtrans-motif
	$(FIXIN) $(INST_SCRIPT)/bp_revtrans-motif
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_revtrans-motif

$(INST_SCRIPT)/bp_seqpart : bin/bp_seqpart $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_seqpart
	$(CP) bin/bp_seqpart $(INST_SCRIPT)/bp_seqpart
	$(FIXIN) $(INST_SCRIPT)/bp_seqpart
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_seqpart

$(INST_SCRIPT)/bp_seqret : bin/bp_seqret $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_seqret
	$(CP) bin/bp_seqret $(INST_SCRIPT)/bp_seqret
	$(FIXIN) $(INST_SCRIPT)/bp_seqret
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_seqret

$(INST_SCRIPT)/bp_mrtrans : bin/bp_mrtrans $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_mrtrans
	$(CP) bin/bp_mrtrans $(INST_SCRIPT)/bp_mrtrans
	$(FIXIN) $(INST_SCRIPT)/bp_mrtrans
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_mrtrans

$(INST_SCRIPT)/bp_make_mrna_protein : bin/bp_make_mrna_protein $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_make_mrna_protein
	$(CP) bin/bp_make_mrna_protein $(INST_SCRIPT)/bp_make_mrna_protein
	$(FIXIN) $(INST_SCRIPT)/bp_make_mrna_protein
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_make_mrna_protein

$(INST_SCRIPT)/bp_tree2pag : bin/bp_tree2pag $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_tree2pag
	$(CP) bin/bp_tree2pag $(INST_SCRIPT)/bp_tree2pag
	$(FIXIN) $(INST_SCRIPT)/bp_tree2pag
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_tree2pag

$(INST_SCRIPT)/bp_extract_feature_seq : bin/bp_extract_feature_seq $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_extract_feature_seq
	$(CP) bin/bp_extract_feature_seq $(INST_SCRIPT)/bp_extract_feature_seq
	$(FIXIN) $(INST_SCRIPT)/bp_extract_feature_seq
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_extract_feature_seq

$(INST_SCRIPT)/bp_fastam9_to_table : bin/bp_fastam9_to_table $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_fastam9_to_table
	$(CP) bin/bp_fastam9_to_table $(INST_SCRIPT)/bp_fastam9_to_table
	$(FIXIN) $(INST_SCRIPT)/bp_fastam9_to_table
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_fastam9_to_table

$(INST_SCRIPT)/bp_unflatten_seq : bin/bp_unflatten_seq $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_unflatten_seq
	$(CP) bin/bp_unflatten_seq $(INST_SCRIPT)/bp_unflatten_seq
	$(FIXIN) $(INST_SCRIPT)/bp_unflatten_seq
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_unflatten_seq

$(INST_SCRIPT)/bp_taxid4species : bin/bp_taxid4species $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_taxid4species
	$(CP) bin/bp_taxid4species $(INST_SCRIPT)/bp_taxid4species
	$(FIXIN) $(INST_SCRIPT)/bp_taxid4species
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_taxid4species

$(INST_SCRIPT)/bp_sreformat : bin/bp_sreformat $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_sreformat
	$(CP) bin/bp_sreformat $(INST_SCRIPT)/bp_sreformat
	$(FIXIN) $(INST_SCRIPT)/bp_sreformat
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_sreformat

$(INST_SCRIPT)/bp_local_taxonomydb_query : bin/bp_local_taxonomydb_query $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_local_taxonomydb_query
	$(CP) bin/bp_local_taxonomydb_query $(INST_SCRIPT)/bp_local_taxonomydb_query
	$(FIXIN) $(INST_SCRIPT)/bp_local_taxonomydb_query
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_local_taxonomydb_query

$(INST_SCRIPT)/bp_dbsplit : bin/bp_dbsplit $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_dbsplit
	$(CP) bin/bp_dbsplit $(INST_SCRIPT)/bp_dbsplit
	$(FIXIN) $(INST_SCRIPT)/bp_dbsplit
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_dbsplit

$(INST_SCRIPT)/bp_index : bin/bp_index $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_index
	$(CP) bin/bp_index $(INST_SCRIPT)/bp_index
	$(FIXIN) $(INST_SCRIPT)/bp_index
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_index

$(INST_SCRIPT)/bp_search2tribe : bin/bp_search2tribe $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_search2tribe
	$(CP) bin/bp_search2tribe $(INST_SCRIPT)/bp_search2tribe
	$(FIXIN) $(INST_SCRIPT)/bp_search2tribe
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_search2tribe

$(INST_SCRIPT)/bp_biogetseq : bin/bp_biogetseq $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_biogetseq
	$(CP) bin/bp_biogetseq $(INST_SCRIPT)/bp_biogetseq
	$(FIXIN) $(INST_SCRIPT)/bp_biogetseq
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_biogetseq

$(INST_SCRIPT)/bp_fetch : bin/bp_fetch $(FIRST_MAKEFILE) $(INST_SCRIPT)$(DFSEP).exists $(INST_BIN)$(DFSEP).exists
	$(NOECHO) $(RM_F) $(INST_SCRIPT)/bp_fetch
	$(CP) bin/bp_fetch $(INST_SCRIPT)/bp_fetch
	$(FIXIN) $(INST_SCRIPT)/bp_fetch
	-$(NOECHO) $(CHMOD) $(PERM_RWX) $(INST_SCRIPT)/bp_fetch



# --- MakeMaker subdirs section:

# none

# --- MakeMaker clean_subdirs section:
clean_subdirs :
	$(NOECHO) $(NOOP)


# --- MakeMaker clean section:

# Delete temporary files but do not touch installed files. We don't delete
# the Makefile here so a later make realclean still has a makefile to use.

clean :: clean_subdirs
	- $(RM_F) \
	  $(BASEEXT).bso $(BASEEXT).def \
	  $(BASEEXT).exp $(BASEEXT).x \
	  $(BOOTSTRAP) $(INST_ARCHAUTODIR)/extralibs.all \
	  $(INST_ARCHAUTODIR)/extralibs.ld $(MAKE_APERL_FILE) \
	  *$(LIB_EXT) *$(OBJ_EXT) \
	  *perl.core MYMETA.json \
	  MYMETA.yml blibdirs.ts \
	  core core.*perl.*.? \
	  core.[0-9] core.[0-9][0-9] \
	  core.[0-9][0-9][0-9] core.[0-9][0-9][0-9][0-9] \
	  core.[0-9][0-9][0-9][0-9][0-9] lib$(BASEEXT).def \
	  mon.out perl \
	  perl$(EXE_EXT) perl.exe \
	  perlmain.c pm_to_blib \
	  pm_to_blib.ts so_locations \
	  tmon.out 
	- $(RM_RF) \
	  blib 
	  $(NOECHO) $(RM_F) $(MAKEFILE_OLD)
	- $(MV) $(FIRST_MAKEFILE) $(MAKEFILE_OLD) $(DEV_NULL)


# --- MakeMaker realclean_subdirs section:
realclean_subdirs :
	$(NOECHO) $(NOOP)


# --- MakeMaker realclean section:
# Delete temporary files (via clean) and also delete dist files
realclean purge ::  clean realclean_subdirs
	- $(RM_F) \
	  $(MAKEFILE_OLD) $(FIRST_MAKEFILE) 
	- $(RM_RF) \
	  $(DISTVNAME) 


# --- MakeMaker metafile section:
metafile : create_distdir
	$(NOECHO) $(ECHO) Generating META.yml
	$(NOECHO) $(ECHO) '---' > META_new.yml
	$(NOECHO) $(ECHO) 'abstract: '\''Perl modules for biology'\''' >> META_new.yml
	$(NOECHO) $(ECHO) 'author:' >> META_new.yml
	$(NOECHO) $(ECHO) '  - '\''See individual modules'\''' >> META_new.yml
	$(NOECHO) $(ECHO) 'build_requires:' >> META_new.yml
	$(NOECHO) $(ECHO) '  ExtUtils::MakeMaker: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  File::Spec: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  FindBin: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  IO::Handle: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  IPC::Open3: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  Test::Memory::Cycle: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  Test::More: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  Test::Weaken: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  lib: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) 'configure_requires:' >> META_new.yml
	$(NOECHO) $(ECHO) '  ExtUtils::MakeMaker: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) 'dynamic_config: 1' >> META_new.yml
	$(NOECHO) $(ECHO) 'generated_by: '\''ExtUtils::MakeMaker version 7.0401, CPAN::Meta::Converter version 2.150001'\''' >> META_new.yml
	$(NOECHO) $(ECHO) 'license: perl' >> META_new.yml
	$(NOECHO) $(ECHO) 'meta-spec:' >> META_new.yml
	$(NOECHO) $(ECHO) '  url: http://module-build.sourceforge.net/META-spec-v1.4.html' >> META_new.yml
	$(NOECHO) $(ECHO) '  version: '\''1.4'\''' >> META_new.yml
	$(NOECHO) $(ECHO) 'name: BioPerl' >> META_new.yml
	$(NOECHO) $(ECHO) 'no_index:' >> META_new.yml
	$(NOECHO) $(ECHO) '  directory:' >> META_new.yml
	$(NOECHO) $(ECHO) '    - t' >> META_new.yml
	$(NOECHO) $(ECHO) '    - inc' >> META_new.yml
	$(NOECHO) $(ECHO) 'requires:' >> META_new.yml
	$(NOECHO) $(ECHO) '  AnyDBM_File: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  Carp: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  Cwd: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  DB_File: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  Data::Dumper: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  Data::Stag: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  Digest::MD5: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  Dumpvalue: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  Error: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  Exporter: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  Fcntl: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  File::Basename: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  File::Copy: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  File::Path: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  File::Spec: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  File::Spec::Functions: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  File::Temp: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  FileHandle: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  Getopt::Long: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  Graph::Directed: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  HTTP::Request::Common: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  HTTP::Response: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  IO::File: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  IO::Handle: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  IO::Pipe: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  IO::Scalar: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  IO::Socket: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  IO::String: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  IPC::Run: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  LWP::UserAgent: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  List::MoreUtils: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  List::Util: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  Math::BigFloat: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  Module::Build: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  POSIX: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  Pod::Usage: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  Scalar::Util: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  Set::Scalar: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  Storable: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  Symbol: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  Test::Builder: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  Test::Builder::Module: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  Test::Most: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  Test::RequiresInternet: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  Text::Balanced: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  Text::Wrap: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  Tie::Handle: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  Tie::RefHash: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  Time::HiRes: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  UNIVERSAL: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  URI: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  XML::DOM: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  XML::LibXML: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  XML::LibXML::Reader: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  XML::Parser::PerlSAX: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  XML::SAX: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  XML::SAX::Base: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  XML::SAX::Writer: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  XML::Twig: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  XML::Writer: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  YAML: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  base: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  constant: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  integer: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  overload: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  parent: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  perl: '\''5.006'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  strict: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  utf8: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  vars: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  version: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) '  warnings: '\''0'\''' >> META_new.yml
	$(NOECHO) $(ECHO) 'version: v1.7.8' >> META_new.yml
	-$(NOECHO) $(MV) META_new.yml $(DISTVNAME)/META.yml
	$(NOECHO) $(ECHO) Generating META.json
	$(NOECHO) $(ECHO) '{' > META_new.json
	$(NOECHO) $(ECHO) '   "abstract" : "Perl modules for biology",' >> META_new.json
	$(NOECHO) $(ECHO) '   "author" : [' >> META_new.json
	$(NOECHO) $(ECHO) '      "See individual modules"' >> META_new.json
	$(NOECHO) $(ECHO) '   ],' >> META_new.json
	$(NOECHO) $(ECHO) '   "dynamic_config" : 1,' >> META_new.json
	$(NOECHO) $(ECHO) '   "generated_by" : "ExtUtils::MakeMaker version 7.0401, CPAN::Meta::Converter version 2.150001",' >> META_new.json
	$(NOECHO) $(ECHO) '   "license" : [' >> META_new.json
	$(NOECHO) $(ECHO) '      "perl_5"' >> META_new.json
	$(NOECHO) $(ECHO) '   ],' >> META_new.json
	$(NOECHO) $(ECHO) '   "meta-spec" : {' >> META_new.json
	$(NOECHO) $(ECHO) '      "url" : "http://search.cpan.org/perldoc?CPAN::Meta::Spec",' >> META_new.json
	$(NOECHO) $(ECHO) '      "version" : "2"' >> META_new.json
	$(NOECHO) $(ECHO) '   },' >> META_new.json
	$(NOECHO) $(ECHO) '   "name" : "BioPerl",' >> META_new.json
	$(NOECHO) $(ECHO) '   "no_index" : {' >> META_new.json
	$(NOECHO) $(ECHO) '      "directory" : [' >> META_new.json
	$(NOECHO) $(ECHO) '         "t",' >> META_new.json
	$(NOECHO) $(ECHO) '         "inc"' >> META_new.json
	$(NOECHO) $(ECHO) '      ]' >> META_new.json
	$(NOECHO) $(ECHO) '   },' >> META_new.json
	$(NOECHO) $(ECHO) '   "prereqs" : {' >> META_new.json
	$(NOECHO) $(ECHO) '      "build" : {' >> META_new.json
	$(NOECHO) $(ECHO) '         "requires" : {' >> META_new.json
	$(NOECHO) $(ECHO) '            "ExtUtils::MakeMaker" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "File::Spec" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "FindBin" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "IO::Handle" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "IPC::Open3" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "Test::Memory::Cycle" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "Test::More" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "Test::Weaken" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "lib" : "0"' >> META_new.json
	$(NOECHO) $(ECHO) '         }' >> META_new.json
	$(NOECHO) $(ECHO) '      },' >> META_new.json
	$(NOECHO) $(ECHO) '      "configure" : {' >> META_new.json
	$(NOECHO) $(ECHO) '         "requires" : {' >> META_new.json
	$(NOECHO) $(ECHO) '            "ExtUtils::MakeMaker" : "0"' >> META_new.json
	$(NOECHO) $(ECHO) '         }' >> META_new.json
	$(NOECHO) $(ECHO) '      },' >> META_new.json
	$(NOECHO) $(ECHO) '      "runtime" : {' >> META_new.json
	$(NOECHO) $(ECHO) '         "requires" : {' >> META_new.json
	$(NOECHO) $(ECHO) '            "AnyDBM_File" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "Carp" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "Cwd" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "DB_File" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "Data::Dumper" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "Data::Stag" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "Digest::MD5" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "Dumpvalue" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "Error" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "Exporter" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "Fcntl" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "File::Basename" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "File::Copy" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "File::Path" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "File::Spec" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "File::Spec::Functions" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "File::Temp" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "FileHandle" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "Getopt::Long" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "Graph::Directed" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "HTTP::Request::Common" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "HTTP::Response" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "IO::File" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "IO::Handle" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "IO::Pipe" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "IO::Scalar" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "IO::Socket" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "IO::String" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "IPC::Run" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "LWP::UserAgent" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "List::MoreUtils" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "List::Util" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "Math::BigFloat" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "Module::Build" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "POSIX" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "Pod::Usage" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "Scalar::Util" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "Set::Scalar" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "Storable" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "Symbol" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "Test::Builder" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "Test::Builder::Module" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "Test::Most" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "Test::RequiresInternet" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "Text::Balanced" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "Text::Wrap" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "Tie::Handle" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "Tie::RefHash" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "Time::HiRes" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "UNIVERSAL" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "URI" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "XML::DOM" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "XML::LibXML" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "XML::LibXML::Reader" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "XML::Parser::PerlSAX" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "XML::SAX" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "XML::SAX::Base" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "XML::SAX::Writer" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "XML::Twig" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "XML::Writer" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "YAML" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "base" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "constant" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "integer" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "overload" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "parent" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "perl" : "5.006",' >> META_new.json
	$(NOECHO) $(ECHO) '            "strict" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "utf8" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "vars" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "version" : "0",' >> META_new.json
	$(NOECHO) $(ECHO) '            "warnings" : "0"' >> META_new.json
	$(NOECHO) $(ECHO) '         }' >> META_new.json
	$(NOECHO) $(ECHO) '      }' >> META_new.json
	$(NOECHO) $(ECHO) '   },' >> META_new.json
	$(NOECHO) $(ECHO) '   "release_status" : "stable",' >> META_new.json
	$(NOECHO) $(ECHO) '   "version" : "v1.7.8"' >> META_new.json
	$(NOECHO) $(ECHO) '}' >> META_new.json
	-$(NOECHO) $(MV) META_new.json $(DISTVNAME)/META.json


# --- MakeMaker signature section:
signature :
	cpansign -s


# --- MakeMaker dist_basics section:
distclean :: realclean distcheck
	$(NOECHO) $(NOOP)

distcheck :
	$(PERLRUN) "-MExtUtils::Manifest=fullcheck" -e fullcheck

skipcheck :
	$(PERLRUN) "-MExtUtils::Manifest=skipcheck" -e skipcheck

manifest :
	$(PERLRUN) "-MExtUtils::Manifest=mkmanifest" -e mkmanifest

veryclean : realclean
	$(RM_F) *~ */*~ *.orig */*.orig *.bak */*.bak *.old */*.old



# --- MakeMaker dist_core section:

dist : $(DIST_DEFAULT) $(FIRST_MAKEFILE)
	$(NOECHO) $(ABSPERLRUN) -l -e 'print '\''Warning: Makefile possibly out of date with $(VERSION_FROM)'\''' \
	  -e '    if -e '\''$(VERSION_FROM)'\'' and -M '\''$(VERSION_FROM)'\'' < -M '\''$(FIRST_MAKEFILE)'\'';' --

tardist : $(DISTVNAME).tar$(SUFFIX)
	$(NOECHO) $(NOOP)

uutardist : $(DISTVNAME).tar$(SUFFIX)
	uuencode $(DISTVNAME).tar$(SUFFIX) $(DISTVNAME).tar$(SUFFIX) > $(DISTVNAME).tar$(SUFFIX)_uu
	$(NOECHO) $(ECHO) 'Created $(DISTVNAME).tar$(SUFFIX)_uu'

$(DISTVNAME).tar$(SUFFIX) : distdir
	$(PREOP)
	$(TO_UNIX)
	$(TAR) $(TARFLAGS) $(DISTVNAME).tar $(DISTVNAME)
	$(RM_RF) $(DISTVNAME)
	$(COMPRESS) $(DISTVNAME).tar
	$(NOECHO) $(ECHO) 'Created $(DISTVNAME).tar$(SUFFIX)'
	$(POSTOP)

zipdist : $(DISTVNAME).zip
	$(NOECHO) $(NOOP)

$(DISTVNAME).zip : distdir
	$(PREOP)
	$(ZIP) $(ZIPFLAGS) $(DISTVNAME).zip $(DISTVNAME)
	$(RM_RF) $(DISTVNAME)
	$(NOECHO) $(ECHO) 'Created $(DISTVNAME).zip'
	$(POSTOP)

shdist : distdir
	$(PREOP)
	$(SHAR) $(DISTVNAME) > $(DISTVNAME).shar
	$(RM_RF) $(DISTVNAME)
	$(NOECHO) $(ECHO) 'Created $(DISTVNAME).shar'
	$(POSTOP)


# --- MakeMaker distdir section:
create_distdir :
	$(RM_RF) $(DISTVNAME)
	$(PERLRUN) "-MExtUtils::Manifest=manicopy,maniread" \
		-e "manicopy(maniread(),'$(DISTVNAME)', '$(DIST_CP)');"

distdir : create_distdir distmeta 
	$(NOECHO) $(NOOP)



# --- MakeMaker dist_test section:
disttest : distdir
	cd $(DISTVNAME) && $(ABSPERLRUN) Makefile.PL "INSTALL_BASE=/bios-store5/lyt/python_demo_try/SeriesAnalysis/perl5"
	cd $(DISTVNAME) && $(MAKE) $(PASTHRU)
	cd $(DISTVNAME) && $(MAKE) test $(PASTHRU)



# --- MakeMaker dist_ci section:

ci :
	$(PERLRUN) "-MExtUtils::Manifest=maniread" \
	  -e "@all = keys %{ maniread() };" \
	  -e "print(qq{Executing $(CI) @all\n}); system(qq{$(CI) @all});" \
	  -e "print(qq{Executing $(RCS_LABEL) ...\n}); system(qq{$(RCS_LABEL) @all});"


# --- MakeMaker distmeta section:
distmeta : create_distdir metafile
	$(NOECHO) cd $(DISTVNAME) && $(ABSPERLRUN) -MExtUtils::Manifest=maniadd -e 'exit unless -e q{META.yml};' \
	  -e 'eval { maniadd({q{META.yml} => q{Module YAML meta-data (added by MakeMaker)}}) }' \
	  -e '    or print "Could not add META.yml to MANIFEST: $$$${'\''@'\''}\n"' --
	$(NOECHO) cd $(DISTVNAME) && $(ABSPERLRUN) -MExtUtils::Manifest=maniadd -e 'exit unless -f q{META.json};' \
	  -e 'eval { maniadd({q{META.json} => q{Module JSON meta-data (added by MakeMaker)}}) }' \
	  -e '    or print "Could not add META.json to MANIFEST: $$$${'\''@'\''}\n"' --



# --- MakeMaker distsignature section:
distsignature : create_distdir
	$(NOECHO) cd $(DISTVNAME) && $(ABSPERLRUN) -MExtUtils::Manifest=maniadd -e 'eval { maniadd({q{SIGNATURE} => q{Public-key signature (added by MakeMaker)}}) }' \
	  -e '    or print "Could not add SIGNATURE to MANIFEST: $$$${'\''@'\''}\n"' --
	$(NOECHO) cd $(DISTVNAME) && $(TOUCH) SIGNATURE
	cd $(DISTVNAME) && cpansign -s



# --- MakeMaker install section:

install :: pure_install doc_install
	$(NOECHO) $(NOOP)

install_perl :: pure_perl_install doc_perl_install
	$(NOECHO) $(NOOP)

install_site :: pure_site_install doc_site_install
	$(NOECHO) $(NOOP)

install_vendor :: pure_vendor_install doc_vendor_install
	$(NOECHO) $(NOOP)

pure_install :: pure_$(INSTALLDIRS)_install
	$(NOECHO) $(NOOP)

doc_install :: doc_$(INSTALLDIRS)_install
	$(NOECHO) $(NOOP)

pure__install : pure_site_install
	$(NOECHO) $(ECHO) INSTALLDIRS not defined, defaulting to INSTALLDIRS=site

doc__install : doc_site_install
	$(NOECHO) $(ECHO) INSTALLDIRS not defined, defaulting to INSTALLDIRS=site

pure_perl_install :: all
	$(NOECHO) $(MOD_INSTALL) \
		read "$(PERL_ARCHLIB)/auto/$(FULLEXT)/.packlist" \
		write "$(DESTINSTALLARCHLIB)/auto/$(FULLEXT)/.packlist" \
		"$(INST_LIB)" "$(DESTINSTALLPRIVLIB)" \
		"$(INST_ARCHLIB)" "$(DESTINSTALLARCHLIB)" \
		"$(INST_BIN)" "$(DESTINSTALLBIN)" \
		"$(INST_SCRIPT)" "$(DESTINSTALLSCRIPT)" \
		"$(INST_MAN1DIR)" "$(DESTINSTALLMAN1DIR)" \
		"$(INST_MAN3DIR)" "$(DESTINSTALLMAN3DIR)"
	$(NOECHO) $(WARN_IF_OLD_PACKLIST) \
		"$(SITEARCHEXP)/auto/$(FULLEXT)"


pure_site_install :: all
	$(NOECHO) $(MOD_INSTALL) \
		read "$(SITEARCHEXP)/auto/$(FULLEXT)/.packlist" \
		write "$(DESTINSTALLSITEARCH)/auto/$(FULLEXT)/.packlist" \
		"$(INST_LIB)" "$(DESTINSTALLSITELIB)" \
		"$(INST_ARCHLIB)" "$(DESTINSTALLSITEARCH)" \
		"$(INST_BIN)" "$(DESTINSTALLSITEBIN)" \
		"$(INST_SCRIPT)" "$(DESTINSTALLSITESCRIPT)" \
		"$(INST_MAN1DIR)" "$(DESTINSTALLSITEMAN1DIR)" \
		"$(INST_MAN3DIR)" "$(DESTINSTALLSITEMAN3DIR)"
	$(NOECHO) $(WARN_IF_OLD_PACKLIST) \
		"$(PERL_ARCHLIB)/auto/$(FULLEXT)"

pure_vendor_install :: all
	$(NOECHO) $(MOD_INSTALL) \
		read "$(VENDORARCHEXP)/auto/$(FULLEXT)/.packlist" \
		write "$(DESTINSTALLVENDORARCH)/auto/$(FULLEXT)/.packlist" \
		"$(INST_LIB)" "$(DESTINSTALLVENDORLIB)" \
		"$(INST_ARCHLIB)" "$(DESTINSTALLVENDORARCH)" \
		"$(INST_BIN)" "$(DESTINSTALLVENDORBIN)" \
		"$(INST_SCRIPT)" "$(DESTINSTALLVENDORSCRIPT)" \
		"$(INST_MAN1DIR)" "$(DESTINSTALLVENDORMAN1DIR)" \
		"$(INST_MAN3DIR)" "$(DESTINSTALLVENDORMAN3DIR)"


doc_perl_install :: all
	$(NOECHO) $(ECHO) Appending installation info to "$(DESTINSTALLARCHLIB)/perllocal.pod"
	-$(NOECHO) $(MKPATH) "$(DESTINSTALLARCHLIB)"
	-$(NOECHO) $(DOC_INSTALL) \
		"Module" "$(NAME)" \
		"installed into" $(INSTALLPRIVLIB) \
		LINKTYPE "$(LINKTYPE)" \
		VERSION "$(VERSION)" \
		EXE_FILES "$(EXE_FILES)" \
		>> "$(DESTINSTALLARCHLIB)/perllocal.pod"

doc_site_install :: all
	$(NOECHO) $(ECHO) Appending installation info to "$(DESTINSTALLARCHLIB)/perllocal.pod"
	-$(NOECHO) $(MKPATH) "$(DESTINSTALLARCHLIB)"
	-$(NOECHO) $(DOC_INSTALL) \
		"Module" "$(NAME)" \
		"installed into" $(INSTALLSITELIB) \
		LINKTYPE "$(LINKTYPE)" \
		VERSION "$(VERSION)" \
		EXE_FILES "$(EXE_FILES)" \
		>> "$(DESTINSTALLARCHLIB)/perllocal.pod"

doc_vendor_install :: all
	$(NOECHO) $(ECHO) Appending installation info to "$(DESTINSTALLARCHLIB)/perllocal.pod"
	-$(NOECHO) $(MKPATH) "$(DESTINSTALLARCHLIB)"
	-$(NOECHO) $(DOC_INSTALL) \
		"Module" "$(NAME)" \
		"installed into" $(INSTALLVENDORLIB) \
		LINKTYPE "$(LINKTYPE)" \
		VERSION "$(VERSION)" \
		EXE_FILES "$(EXE_FILES)" \
		>> "$(DESTINSTALLARCHLIB)/perllocal.pod"


uninstall :: uninstall_from_$(INSTALLDIRS)dirs
	$(NOECHO) $(NOOP)

uninstall_from_perldirs ::
	$(NOECHO) $(UNINSTALL) "$(PERL_ARCHLIB)/auto/$(FULLEXT)/.packlist"

uninstall_from_sitedirs ::
	$(NOECHO) $(UNINSTALL) "$(SITEARCHEXP)/auto/$(FULLEXT)/.packlist"

uninstall_from_vendordirs ::
	$(NOECHO) $(UNINSTALL) "$(VENDORARCHEXP)/auto/$(FULLEXT)/.packlist"


# --- MakeMaker force section:
# Phony target to force checking subdirectories.
FORCE :
	$(NOECHO) $(NOOP)


# --- MakeMaker perldepend section:


# --- MakeMaker makefile section:
# We take a very conservative approach here, but it's worth it.
# We move Makefile to Makefile.old here to avoid gnu make looping.
$(FIRST_MAKEFILE) : Makefile.PL $(CONFIGDEP)
	$(NOECHO) $(ECHO) "Makefile out-of-date with respect to $?"
	$(NOECHO) $(ECHO) "Cleaning current config before rebuilding Makefile..."
	-$(NOECHO) $(RM_F) $(MAKEFILE_OLD)
	-$(NOECHO) $(MV)   $(FIRST_MAKEFILE) $(MAKEFILE_OLD)
	- $(MAKE) $(USEMAKEFILE) $(MAKEFILE_OLD) clean $(DEV_NULL)
	$(PERLRUN) Makefile.PL "INSTALL_BASE=/bios-store5/lyt/python_demo_try/SeriesAnalysis/perl5"
	$(NOECHO) $(ECHO) "==> Your Makefile has been rebuilt. <=="
	$(NOECHO) $(ECHO) "==> Please rerun the $(MAKE) command.  <=="
	$(FALSE)



# --- MakeMaker staticmake section:

# --- MakeMaker makeaperl section ---
MAP_TARGET    = perl
FULLPERL      = "/bios-store1/home/yutongli/.conda/envs/try_619_new/bin/perl"

$(MAP_TARGET) :: static $(MAKE_APERL_FILE)
	$(MAKE) $(USEMAKEFILE) $(MAKE_APERL_FILE) $@

$(MAKE_APERL_FILE) : $(FIRST_MAKEFILE) pm_to_blib
	$(NOECHO) $(ECHO) Writing \"$(MAKE_APERL_FILE)\" for this $(MAP_TARGET)
	$(NOECHO) $(PERLRUNINST) \
		Makefile.PL DIR="" \
		MAKEFILE=$(MAKE_APERL_FILE) LINKTYPE=static \
		MAKEAPERL=1 NORECURS=1 CCCDLFLAGS= \
		INSTALL_BASE=/bios-store5/lyt/python_demo_try/SeriesAnalysis/perl5


# --- MakeMaker test section:

TEST_VERBOSE=0
TEST_TYPE=test_$(LINKTYPE)
TEST_FILE = test.pl
TEST_FILES = t/*.t t/Align/*.t t/AlignIO/*.t t/Annotation/*.t t/LocalDB/*.t t/LocalDB/Index/*.t t/LocalDB/Taxonomy/*.t t/Matrix/*.t t/Matrix/IO/*.t t/Ontology/*.t t/Ontology/IO/*.t t/RemoteDB/*.t t/Root/*.t t/SearchIO/*.t t/SearchIO/Writer/*.t t/Seq/*.t t/SeqFeature/*.t t/SeqIO/*.t t/SeqTools/*.t t/Tools/*.t t/Tools/Alignment/*.t t/Tools/EMBOSS/*.t t/Tools/Phylo/*.t t/Tools/Phylo/Phylip/*.t t/Tools/Signalp/*.t t/Tools/Spidey/*.t t/Tree/*.t t/Tree/TreeIO/*.t
TESTDB_SW = -d

testdb :: testdb_$(LINKTYPE)

test :: $(TEST_TYPE) subdirs-test

subdirs-test ::
	$(NOECHO) $(NOOP)


test_dynamic :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERLRUN) "-MExtUtils::Command::MM" "-MTest::Harness" "-e" "undef *Test::Harness::Switches; test_harness($(TEST_VERBOSE), '$(INST_LIB)', '$(INST_ARCHLIB)')" $(TEST_FILES)

testdb_dynamic :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERLRUN) $(TESTDB_SW) "-I$(INST_LIB)" "-I$(INST_ARCHLIB)" $(TEST_FILE)

test_ : test_dynamic

test_static :: test_dynamic
testdb_static :: testdb_dynamic


# --- MakeMaker ppd section:
# Creates a PPD (Perl Package Description) for a binary distribution.
ppd :
	$(NOECHO) $(ECHO) '<SOFTPKG NAME="$(DISTNAME)" VERSION="$(VERSION)">' > $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '    <ABSTRACT>Perl modules for biology</ABSTRACT>' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '    <AUTHOR>See individual modules</AUTHOR>' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '    <IMPLEMENTATION>' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <PERLCORE VERSION="5,006,0,0" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="AnyDBM_File::" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="Carp::" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="Cwd::" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="DB_File::" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="Data::Dumper" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="Data::Stag" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="Digest::MD5" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="Dumpvalue::" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="Error::" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="Exporter::" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="Fcntl::" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="File::Basename" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="File::Copy" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="File::Path" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="File::Spec" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="File::Spec::Functions" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="File::Temp" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="FileHandle::" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="Getopt::Long" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="Graph::Directed" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="HTTP::Request::Common" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="HTTP::Response" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="IO::File" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="IO::Handle" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="IO::Pipe" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="IO::Scalar" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="IO::Socket" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="IO::String" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="IPC::Run" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="LWP::UserAgent" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="List::MoreUtils" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="List::Util" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="Math::BigFloat" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="Module::Build" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="POSIX::" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="Pod::Usage" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="Scalar::Util" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="Set::Scalar" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="Storable::" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="Symbol::" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="Test::Builder" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="Test::Builder::Module" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="Test::Most" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="Test::RequiresInternet" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="Text::Balanced" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="Text::Wrap" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="Tie::Handle" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="Tie::RefHash" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="Time::HiRes" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="UNIVERSAL::" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="URI::" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="XML::DOM" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="XML::LibXML" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="XML::LibXML::Reader" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="XML::Parser::PerlSAX" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="XML::SAX" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="XML::SAX::Base" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="XML::SAX::Writer" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="XML::Twig" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="XML::Writer" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="YAML::" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="base::" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="constant::" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="integer::" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="overload::" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="parent::" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="strict::" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="utf8::" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="vars::" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="version::" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <REQUIRE NAME="warnings::" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <ARCHITECTURE NAME="x86_64-linux-thread-multi-5.22" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '        <CODEBASE HREF="" />' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '    </IMPLEMENTATION>' >> $(DISTNAME).ppd
	$(NOECHO) $(ECHO) '</SOFTPKG>' >> $(DISTNAME).ppd


# --- MakeMaker pm_to_blib section:

pm_to_blib : $(FIRST_MAKEFILE) $(TO_INST_PM)
	$(NOECHO) $(ABSPERLRUN) -MExtUtils::Install -e 'pm_to_blib({@ARGV}, '\''$(INST_LIB)/auto'\'', q[$(PM_FILTER)], '\''$(PERM_DIR)'\'')' -- \
	  lib/Bio/Align/AlignI.pm blib/lib/Bio/Align/AlignI.pm \
	  lib/Bio/Align/DNAStatistics.pm blib/lib/Bio/Align/DNAStatistics.pm \
	  lib/Bio/Align/PairwiseStatistics.pm blib/lib/Bio/Align/PairwiseStatistics.pm \
	  lib/Bio/Align/ProteinStatistics.pm blib/lib/Bio/Align/ProteinStatistics.pm \
	  lib/Bio/Align/StatisticsI.pm blib/lib/Bio/Align/StatisticsI.pm \
	  lib/Bio/Align/Utilities.pm blib/lib/Bio/Align/Utilities.pm \
	  lib/Bio/AlignIO.pm blib/lib/Bio/AlignIO.pm \
	  lib/Bio/AlignIO/Handler/GenericAlignHandler.pm blib/lib/Bio/AlignIO/Handler/GenericAlignHandler.pm \
	  lib/Bio/AlignIO/arp.pm blib/lib/Bio/AlignIO/arp.pm \
	  lib/Bio/AlignIO/bl2seq.pm blib/lib/Bio/AlignIO/bl2seq.pm \
	  lib/Bio/AlignIO/clustalw.pm blib/lib/Bio/AlignIO/clustalw.pm \
	  lib/Bio/AlignIO/emboss.pm blib/lib/Bio/AlignIO/emboss.pm \
	  lib/Bio/AlignIO/fasta.pm blib/lib/Bio/AlignIO/fasta.pm \
	  lib/Bio/AlignIO/largemultifasta.pm blib/lib/Bio/AlignIO/largemultifasta.pm \
	  lib/Bio/AlignIO/maf.pm blib/lib/Bio/AlignIO/maf.pm \
	  lib/Bio/AlignIO/mase.pm blib/lib/Bio/AlignIO/mase.pm \
	  lib/Bio/AlignIO/mega.pm blib/lib/Bio/AlignIO/mega.pm \
	  lib/Bio/AlignIO/meme.pm blib/lib/Bio/AlignIO/meme.pm \
	  lib/Bio/AlignIO/metafasta.pm blib/lib/Bio/AlignIO/metafasta.pm \
	  lib/Bio/AlignIO/msf.pm blib/lib/Bio/AlignIO/msf.pm \
	  lib/Bio/AlignIO/nexus.pm blib/lib/Bio/AlignIO/nexus.pm \
	  lib/Bio/AlignIO/pfam.pm blib/lib/Bio/AlignIO/pfam.pm \
	  lib/Bio/AlignIO/phylip.pm blib/lib/Bio/AlignIO/phylip.pm \
	  lib/Bio/AlignIO/po.pm blib/lib/Bio/AlignIO/po.pm \
	  lib/Bio/AlignIO/proda.pm blib/lib/Bio/AlignIO/proda.pm \
	  lib/Bio/AlignIO/prodom.pm blib/lib/Bio/AlignIO/prodom.pm \
	  lib/Bio/AlignIO/psi.pm blib/lib/Bio/AlignIO/psi.pm \
	  lib/Bio/AlignIO/selex.pm blib/lib/Bio/AlignIO/selex.pm \
	  lib/Bio/AlignIO/xmfa.pm blib/lib/Bio/AlignIO/xmfa.pm \
	  lib/Bio/AnalysisI.pm blib/lib/Bio/AnalysisI.pm \
	  lib/Bio/AnalysisParserI.pm blib/lib/Bio/AnalysisParserI.pm \
	  lib/Bio/AnalysisResultI.pm blib/lib/Bio/AnalysisResultI.pm \
	  lib/Bio/AnnotatableI.pm blib/lib/Bio/AnnotatableI.pm \
	  lib/Bio/Annotation/AnnotationFactory.pm blib/lib/Bio/Annotation/AnnotationFactory.pm \
	  lib/Bio/Annotation/Collection.pm blib/lib/Bio/Annotation/Collection.pm \
	  lib/Bio/Annotation/Comment.pm blib/lib/Bio/Annotation/Comment.pm \
	  lib/Bio/Annotation/DBLink.pm blib/lib/Bio/Annotation/DBLink.pm \
	  lib/Bio/Annotation/OntologyTerm.pm blib/lib/Bio/Annotation/OntologyTerm.pm \
	  lib/Bio/Annotation/Reference.pm blib/lib/Bio/Annotation/Reference.pm \
	  lib/Bio/Annotation/Relation.pm blib/lib/Bio/Annotation/Relation.pm \
	  lib/Bio/Annotation/SimpleValue.pm blib/lib/Bio/Annotation/SimpleValue.pm \
	  lib/Bio/Annotation/StructuredValue.pm blib/lib/Bio/Annotation/StructuredValue.pm 
	$(NOECHO) $(ABSPERLRUN) -MExtUtils::Install -e 'pm_to_blib({@ARGV}, '\''$(INST_LIB)/auto'\'', q[$(PM_FILTER)], '\''$(PERM_DIR)'\'')' -- \
	  lib/Bio/Annotation/TagTree.pm blib/lib/Bio/Annotation/TagTree.pm \
	  lib/Bio/Annotation/Target.pm blib/lib/Bio/Annotation/Target.pm \
	  lib/Bio/Annotation/Tree.pm blib/lib/Bio/Annotation/Tree.pm \
	  lib/Bio/Annotation/TypeManager.pm blib/lib/Bio/Annotation/TypeManager.pm \
	  lib/Bio/AnnotationCollectionI.pm blib/lib/Bio/AnnotationCollectionI.pm \
	  lib/Bio/AnnotationI.pm blib/lib/Bio/AnnotationI.pm \
	  lib/Bio/CodonUsage/IO.pm blib/lib/Bio/CodonUsage/IO.pm \
	  lib/Bio/CodonUsage/Table.pm blib/lib/Bio/CodonUsage/Table.pm \
	  lib/Bio/DB/DBFetch.pm blib/lib/Bio/DB/DBFetch.pm \
	  lib/Bio/DB/Failover.pm blib/lib/Bio/DB/Failover.pm \
	  lib/Bio/DB/Fasta.pm blib/lib/Bio/DB/Fasta.pm \
	  lib/Bio/DB/FileCache.pm blib/lib/Bio/DB/FileCache.pm \
	  lib/Bio/DB/Flat.pm blib/lib/Bio/DB/Flat.pm \
	  lib/Bio/DB/Flat/BDB.pm blib/lib/Bio/DB/Flat/BDB.pm \
	  lib/Bio/DB/Flat/BDB/embl.pm blib/lib/Bio/DB/Flat/BDB/embl.pm \
	  lib/Bio/DB/Flat/BDB/fasta.pm blib/lib/Bio/DB/Flat/BDB/fasta.pm \
	  lib/Bio/DB/Flat/BDB/genbank.pm blib/lib/Bio/DB/Flat/BDB/genbank.pm \
	  lib/Bio/DB/Flat/BDB/swiss.pm blib/lib/Bio/DB/Flat/BDB/swiss.pm \
	  lib/Bio/DB/Flat/BinarySearch.pm blib/lib/Bio/DB/Flat/BinarySearch.pm \
	  lib/Bio/DB/GFF/Util/Binning.pm blib/lib/Bio/DB/GFF/Util/Binning.pm \
	  lib/Bio/DB/GFF/Util/Rearrange.pm blib/lib/Bio/DB/GFF/Util/Rearrange.pm \
	  lib/Bio/DB/GenericWebAgent.pm blib/lib/Bio/DB/GenericWebAgent.pm \
	  lib/Bio/DB/InMemoryCache.pm blib/lib/Bio/DB/InMemoryCache.pm \
	  lib/Bio/DB/IndexedBase.pm blib/lib/Bio/DB/IndexedBase.pm \
	  lib/Bio/DB/LocationI.pm blib/lib/Bio/DB/LocationI.pm \
	  lib/Bio/DB/Qual.pm blib/lib/Bio/DB/Qual.pm \
	  lib/Bio/DB/Query/WebQuery.pm blib/lib/Bio/DB/Query/WebQuery.pm \
	  lib/Bio/DB/QueryI.pm blib/lib/Bio/DB/QueryI.pm \
	  lib/Bio/DB/RandomAccessI.pm blib/lib/Bio/DB/RandomAccessI.pm \
	  lib/Bio/DB/ReferenceI.pm blib/lib/Bio/DB/ReferenceI.pm \
	  lib/Bio/DB/Registry.pm blib/lib/Bio/DB/Registry.pm \
	  lib/Bio/DB/SeqI.pm blib/lib/Bio/DB/SeqI.pm \
	  lib/Bio/DB/Taxonomy.pm blib/lib/Bio/DB/Taxonomy.pm \
	  lib/Bio/DB/Taxonomy/flatfile.pm blib/lib/Bio/DB/Taxonomy/flatfile.pm \
	  lib/Bio/DB/Taxonomy/greengenes.pm blib/lib/Bio/DB/Taxonomy/greengenes.pm \
	  lib/Bio/DB/Taxonomy/list.pm blib/lib/Bio/DB/Taxonomy/list.pm \
	  lib/Bio/DB/Taxonomy/silva.pm blib/lib/Bio/DB/Taxonomy/silva.pm \
	  lib/Bio/DB/UpdateableSeqI.pm blib/lib/Bio/DB/UpdateableSeqI.pm \
	  lib/Bio/DB/WebDBSeqI.pm blib/lib/Bio/DB/WebDBSeqI.pm \
	  lib/Bio/DBLinkContainerI.pm blib/lib/Bio/DBLinkContainerI.pm \
	  lib/Bio/Das/FeatureTypeI.pm blib/lib/Bio/Das/FeatureTypeI.pm \
	  lib/Bio/Das/SegmentI.pm blib/lib/Bio/Das/SegmentI.pm \
	  lib/Bio/DasI.pm blib/lib/Bio/DasI.pm \
	  lib/Bio/DescribableI.pm blib/lib/Bio/DescribableI.pm 
	$(NOECHO) $(ABSPERLRUN) -MExtUtils::Install -e 'pm_to_blib({@ARGV}, '\''$(INST_LIB)/auto'\'', q[$(PM_FILTER)], '\''$(PERM_DIR)'\'')' -- \
	  lib/Bio/Event/EventGeneratorI.pm blib/lib/Bio/Event/EventGeneratorI.pm \
	  lib/Bio/Event/EventHandlerI.pm blib/lib/Bio/Event/EventHandlerI.pm \
	  lib/Bio/Factory/AnalysisI.pm blib/lib/Bio/Factory/AnalysisI.pm \
	  lib/Bio/Factory/ApplicationFactoryI.pm blib/lib/Bio/Factory/ApplicationFactoryI.pm \
	  lib/Bio/Factory/DriverFactory.pm blib/lib/Bio/Factory/DriverFactory.pm \
	  lib/Bio/Factory/FTLocationFactory.pm blib/lib/Bio/Factory/FTLocationFactory.pm \
	  lib/Bio/Factory/LocationFactoryI.pm blib/lib/Bio/Factory/LocationFactoryI.pm \
	  lib/Bio/Factory/ObjectBuilderI.pm blib/lib/Bio/Factory/ObjectBuilderI.pm \
	  lib/Bio/Factory/ObjectFactory.pm blib/lib/Bio/Factory/ObjectFactory.pm \
	  lib/Bio/Factory/ObjectFactoryI.pm blib/lib/Bio/Factory/ObjectFactoryI.pm \
	  lib/Bio/Factory/SeqAnalysisParserFactory.pm blib/lib/Bio/Factory/SeqAnalysisParserFactory.pm \
	  lib/Bio/Factory/SeqAnalysisParserFactoryI.pm blib/lib/Bio/Factory/SeqAnalysisParserFactoryI.pm \
	  lib/Bio/Factory/SequenceFactoryI.pm blib/lib/Bio/Factory/SequenceFactoryI.pm \
	  lib/Bio/Factory/SequenceProcessorI.pm blib/lib/Bio/Factory/SequenceProcessorI.pm \
	  lib/Bio/Factory/SequenceStreamI.pm blib/lib/Bio/Factory/SequenceStreamI.pm \
	  lib/Bio/Factory/TreeFactoryI.pm blib/lib/Bio/Factory/TreeFactoryI.pm \
	  lib/Bio/FeatureHolderI.pm blib/lib/Bio/FeatureHolderI.pm \
	  lib/Bio/HandlerBaseI.pm blib/lib/Bio/HandlerBaseI.pm \
	  lib/Bio/IdCollectionI.pm blib/lib/Bio/IdCollectionI.pm \
	  lib/Bio/IdentifiableI.pm blib/lib/Bio/IdentifiableI.pm \
	  lib/Bio/Index/Abstract.pm blib/lib/Bio/Index/Abstract.pm \
	  lib/Bio/Index/AbstractSeq.pm blib/lib/Bio/Index/AbstractSeq.pm \
	  lib/Bio/Index/Blast.pm blib/lib/Bio/Index/Blast.pm \
	  lib/Bio/Index/BlastTable.pm blib/lib/Bio/Index/BlastTable.pm \
	  lib/Bio/Index/EMBL.pm blib/lib/Bio/Index/EMBL.pm \
	  lib/Bio/Index/Fasta.pm blib/lib/Bio/Index/Fasta.pm \
	  lib/Bio/Index/Fastq.pm blib/lib/Bio/Index/Fastq.pm \
	  lib/Bio/Index/GenBank.pm blib/lib/Bio/Index/GenBank.pm \
	  lib/Bio/Index/Qual.pm blib/lib/Bio/Index/Qual.pm \
	  lib/Bio/Index/SwissPfam.pm blib/lib/Bio/Index/SwissPfam.pm \
	  lib/Bio/Index/Swissprot.pm blib/lib/Bio/Index/Swissprot.pm \
	  lib/Bio/LocatableSeq.pm blib/lib/Bio/LocatableSeq.pm \
	  lib/Bio/Location/Atomic.pm blib/lib/Bio/Location/Atomic.pm \
	  lib/Bio/Location/AvWithinCoordPolicy.pm blib/lib/Bio/Location/AvWithinCoordPolicy.pm \
	  lib/Bio/Location/CoordinatePolicyI.pm blib/lib/Bio/Location/CoordinatePolicyI.pm \
	  lib/Bio/Location/Fuzzy.pm blib/lib/Bio/Location/Fuzzy.pm \
	  lib/Bio/Location/FuzzyLocationI.pm blib/lib/Bio/Location/FuzzyLocationI.pm \
	  lib/Bio/Location/NarrowestCoordPolicy.pm blib/lib/Bio/Location/NarrowestCoordPolicy.pm 
	$(NOECHO) $(ABSPERLRUN) -MExtUtils::Install -e 'pm_to_blib({@ARGV}, '\''$(INST_LIB)/auto'\'', q[$(PM_FILTER)], '\''$(PERM_DIR)'\'')' -- \
	  lib/Bio/Location/Simple.pm blib/lib/Bio/Location/Simple.pm \
	  lib/Bio/Location/Split.pm blib/lib/Bio/Location/Split.pm \
	  lib/Bio/Location/SplitLocationI.pm blib/lib/Bio/Location/SplitLocationI.pm \
	  lib/Bio/Location/WidestCoordPolicy.pm blib/lib/Bio/Location/WidestCoordPolicy.pm \
	  lib/Bio/LocationI.pm blib/lib/Bio/LocationI.pm \
	  lib/Bio/Matrix/Generic.pm blib/lib/Bio/Matrix/Generic.pm \
	  lib/Bio/Matrix/IO.pm blib/lib/Bio/Matrix/IO.pm \
	  lib/Bio/Matrix/IO/mlagan.pm blib/lib/Bio/Matrix/IO/mlagan.pm \
	  lib/Bio/Matrix/IO/phylip.pm blib/lib/Bio/Matrix/IO/phylip.pm \
	  lib/Bio/Matrix/IO/scoring.pm blib/lib/Bio/Matrix/IO/scoring.pm \
	  lib/Bio/Matrix/MatrixI.pm blib/lib/Bio/Matrix/MatrixI.pm \
	  lib/Bio/Matrix/Mlagan.pm blib/lib/Bio/Matrix/Mlagan.pm \
	  lib/Bio/Matrix/PSM/IO.pm blib/lib/Bio/Matrix/PSM/IO.pm \
	  lib/Bio/Matrix/PSM/IO/mast.pm blib/lib/Bio/Matrix/PSM/IO/mast.pm \
	  lib/Bio/Matrix/PSM/IO/masta.pm blib/lib/Bio/Matrix/PSM/IO/masta.pm \
	  lib/Bio/Matrix/PSM/IO/meme.pm blib/lib/Bio/Matrix/PSM/IO/meme.pm \
	  lib/Bio/Matrix/PSM/IO/psiblast.pm blib/lib/Bio/Matrix/PSM/IO/psiblast.pm \
	  lib/Bio/Matrix/PSM/IO/transfac.pm blib/lib/Bio/Matrix/PSM/IO/transfac.pm \
	  lib/Bio/Matrix/PSM/InstanceSite.pm blib/lib/Bio/Matrix/PSM/InstanceSite.pm \
	  lib/Bio/Matrix/PSM/InstanceSiteI.pm blib/lib/Bio/Matrix/PSM/InstanceSiteI.pm \
	  lib/Bio/Matrix/PSM/ProtMatrix.pm blib/lib/Bio/Matrix/PSM/ProtMatrix.pm \
	  lib/Bio/Matrix/PSM/ProtPsm.pm blib/lib/Bio/Matrix/PSM/ProtPsm.pm \
	  lib/Bio/Matrix/PSM/Psm.pm blib/lib/Bio/Matrix/PSM/Psm.pm \
	  lib/Bio/Matrix/PSM/PsmHeader.pm blib/lib/Bio/Matrix/PSM/PsmHeader.pm \
	  lib/Bio/Matrix/PSM/PsmHeaderI.pm blib/lib/Bio/Matrix/PSM/PsmHeaderI.pm \
	  lib/Bio/Matrix/PSM/PsmI.pm blib/lib/Bio/Matrix/PSM/PsmI.pm \
	  lib/Bio/Matrix/PSM/SiteMatrix.pm blib/lib/Bio/Matrix/PSM/SiteMatrix.pm \
	  lib/Bio/Matrix/PSM/SiteMatrixI.pm blib/lib/Bio/Matrix/PSM/SiteMatrixI.pm \
	  lib/Bio/Matrix/PhylipDist.pm blib/lib/Bio/Matrix/PhylipDist.pm \
	  lib/Bio/Matrix/Scoring.pm blib/lib/Bio/Matrix/Scoring.pm \
	  lib/Bio/Ontology/DocumentRegistry.pm blib/lib/Bio/Ontology/DocumentRegistry.pm \
	  lib/Bio/Ontology/GOterm.pm blib/lib/Bio/Ontology/GOterm.pm \
	  lib/Bio/Ontology/InterProTerm.pm blib/lib/Bio/Ontology/InterProTerm.pm \
	  lib/Bio/Ontology/OBOEngine.pm blib/lib/Bio/Ontology/OBOEngine.pm \
	  lib/Bio/Ontology/OBOterm.pm blib/lib/Bio/Ontology/OBOterm.pm \
	  lib/Bio/Ontology/Ontology.pm blib/lib/Bio/Ontology/Ontology.pm \
	  lib/Bio/Ontology/OntologyEngineI.pm blib/lib/Bio/Ontology/OntologyEngineI.pm \
	  lib/Bio/Ontology/OntologyI.pm blib/lib/Bio/Ontology/OntologyI.pm \
	  lib/Bio/Ontology/OntologyStore.pm blib/lib/Bio/Ontology/OntologyStore.pm 
	$(NOECHO) $(ABSPERLRUN) -MExtUtils::Install -e 'pm_to_blib({@ARGV}, '\''$(INST_LIB)/auto'\'', q[$(PM_FILTER)], '\''$(PERM_DIR)'\'')' -- \
	  lib/Bio/Ontology/Path.pm blib/lib/Bio/Ontology/Path.pm \
	  lib/Bio/Ontology/PathI.pm blib/lib/Bio/Ontology/PathI.pm \
	  lib/Bio/Ontology/Relationship.pm blib/lib/Bio/Ontology/Relationship.pm \
	  lib/Bio/Ontology/RelationshipFactory.pm blib/lib/Bio/Ontology/RelationshipFactory.pm \
	  lib/Bio/Ontology/RelationshipI.pm blib/lib/Bio/Ontology/RelationshipI.pm \
	  lib/Bio/Ontology/RelationshipType.pm blib/lib/Bio/Ontology/RelationshipType.pm \
	  lib/Bio/Ontology/SimpleGOEngine/GraphAdaptor.pm blib/lib/Bio/Ontology/SimpleGOEngine/GraphAdaptor.pm \
	  lib/Bio/Ontology/SimpleOntologyEngine.pm blib/lib/Bio/Ontology/SimpleOntologyEngine.pm \
	  lib/Bio/Ontology/Term.pm blib/lib/Bio/Ontology/Term.pm \
	  lib/Bio/Ontology/TermFactory.pm blib/lib/Bio/Ontology/TermFactory.pm \
	  lib/Bio/Ontology/TermI.pm blib/lib/Bio/Ontology/TermI.pm \
	  lib/Bio/OntologyIO.pm blib/lib/Bio/OntologyIO.pm \
	  lib/Bio/OntologyIO/Handlers/BaseSAXHandler.pm blib/lib/Bio/OntologyIO/Handlers/BaseSAXHandler.pm \
	  lib/Bio/OntologyIO/Handlers/InterProHandler.pm blib/lib/Bio/OntologyIO/Handlers/InterProHandler.pm \
	  lib/Bio/OntologyIO/Handlers/InterPro_BioSQL_Handler.pm blib/lib/Bio/OntologyIO/Handlers/InterPro_BioSQL_Handler.pm \
	  lib/Bio/OntologyIO/InterProParser.pm blib/lib/Bio/OntologyIO/InterProParser.pm \
	  lib/Bio/OntologyIO/dagflat.pm blib/lib/Bio/OntologyIO/dagflat.pm \
	  lib/Bio/OntologyIO/goflat.pm blib/lib/Bio/OntologyIO/goflat.pm \
	  lib/Bio/OntologyIO/obo.pm blib/lib/Bio/OntologyIO/obo.pm \
	  lib/Bio/OntologyIO/simplehierarchy.pm blib/lib/Bio/OntologyIO/simplehierarchy.pm \
	  lib/Bio/OntologyIO/soflat.pm blib/lib/Bio/OntologyIO/soflat.pm \
	  lib/Bio/ParameterBaseI.pm blib/lib/Bio/ParameterBaseI.pm \
	  lib/Bio/PrimarySeq.pm blib/lib/Bio/PrimarySeq.pm \
	  lib/Bio/PrimarySeqI.pm blib/lib/Bio/PrimarySeqI.pm \
	  lib/Bio/PullParserI.pm blib/lib/Bio/PullParserI.pm \
	  lib/Bio/Range.pm blib/lib/Bio/Range.pm \
	  lib/Bio/RangeI.pm blib/lib/Bio/RangeI.pm \
	  lib/Bio/Root/Exception.pm blib/lib/Bio/Root/Exception.pm \
	  lib/Bio/Root/HTTPget.pm blib/lib/Bio/Root/HTTPget.pm \
	  lib/Bio/Root/IO.pm blib/lib/Bio/Root/IO.pm \
	  lib/Bio/Root/Root.pm blib/lib/Bio/Root/Root.pm \
	  lib/Bio/Root/RootI.pm blib/lib/Bio/Root/RootI.pm \
	  lib/Bio/Root/Storable.pm blib/lib/Bio/Root/Storable.pm \
	  lib/Bio/Root/Test.pm blib/lib/Bio/Root/Test.pm \
	  lib/Bio/Root/TestObject.pm blib/lib/Bio/Root/TestObject.pm \
	  lib/Bio/Root/Utilities.pm blib/lib/Bio/Root/Utilities.pm \
	  lib/Bio/Root/Version.pm blib/lib/Bio/Root/Version.pm \
	  lib/Bio/Search/BlastStatistics.pm blib/lib/Bio/Search/BlastStatistics.pm \
	  lib/Bio/Search/BlastUtils.pm blib/lib/Bio/Search/BlastUtils.pm \
	  lib/Bio/Search/DatabaseI.pm blib/lib/Bio/Search/DatabaseI.pm 
	$(NOECHO) $(ABSPERLRUN) -MExtUtils::Install -e 'pm_to_blib({@ARGV}, '\''$(INST_LIB)/auto'\'', q[$(PM_FILTER)], '\''$(PERM_DIR)'\'')' -- \
	  lib/Bio/Search/GenericDatabase.pm blib/lib/Bio/Search/GenericDatabase.pm \
	  lib/Bio/Search/GenericStatistics.pm blib/lib/Bio/Search/GenericStatistics.pm \
	  lib/Bio/Search/HSP/BlastHSP.pm blib/lib/Bio/Search/HSP/BlastHSP.pm \
	  lib/Bio/Search/HSP/BlastPullHSP.pm blib/lib/Bio/Search/HSP/BlastPullHSP.pm \
	  lib/Bio/Search/HSP/FastaHSP.pm blib/lib/Bio/Search/HSP/FastaHSP.pm \
	  lib/Bio/Search/HSP/GenericHSP.pm blib/lib/Bio/Search/HSP/GenericHSP.pm \
	  lib/Bio/Search/HSP/HSPFactory.pm blib/lib/Bio/Search/HSP/HSPFactory.pm \
	  lib/Bio/Search/HSP/HSPI.pm blib/lib/Bio/Search/HSP/HSPI.pm \
	  lib/Bio/Search/HSP/ModelHSP.pm blib/lib/Bio/Search/HSP/ModelHSP.pm \
	  lib/Bio/Search/HSP/PSLHSP.pm blib/lib/Bio/Search/HSP/PSLHSP.pm \
	  lib/Bio/Search/HSP/PsiBlastHSP.pm blib/lib/Bio/Search/HSP/PsiBlastHSP.pm \
	  lib/Bio/Search/HSP/PullHSPI.pm blib/lib/Bio/Search/HSP/PullHSPI.pm \
	  lib/Bio/Search/HSP/WABAHSP.pm blib/lib/Bio/Search/HSP/WABAHSP.pm \
	  lib/Bio/Search/Hit/BlastHit.pm blib/lib/Bio/Search/Hit/BlastHit.pm \
	  lib/Bio/Search/Hit/BlastPullHit.pm blib/lib/Bio/Search/Hit/BlastPullHit.pm \
	  lib/Bio/Search/Hit/Fasta.pm blib/lib/Bio/Search/Hit/Fasta.pm \
	  lib/Bio/Search/Hit/GenericHit.pm blib/lib/Bio/Search/Hit/GenericHit.pm \
	  lib/Bio/Search/Hit/HitFactory.pm blib/lib/Bio/Search/Hit/HitFactory.pm \
	  lib/Bio/Search/Hit/HitI.pm blib/lib/Bio/Search/Hit/HitI.pm \
	  lib/Bio/Search/Hit/ModelHit.pm blib/lib/Bio/Search/Hit/ModelHit.pm \
	  lib/Bio/Search/Hit/PsiBlastHit.pm blib/lib/Bio/Search/Hit/PsiBlastHit.pm \
	  lib/Bio/Search/Hit/PullHitI.pm blib/lib/Bio/Search/Hit/PullHitI.pm \
	  lib/Bio/Search/Iteration/GenericIteration.pm blib/lib/Bio/Search/Iteration/GenericIteration.pm \
	  lib/Bio/Search/Iteration/IterationI.pm blib/lib/Bio/Search/Iteration/IterationI.pm \
	  lib/Bio/Search/Processor.pm blib/lib/Bio/Search/Processor.pm \
	  lib/Bio/Search/Result/BlastPullResult.pm blib/lib/Bio/Search/Result/BlastPullResult.pm \
	  lib/Bio/Search/Result/BlastResult.pm blib/lib/Bio/Search/Result/BlastResult.pm \
	  lib/Bio/Search/Result/CrossMatchResult.pm blib/lib/Bio/Search/Result/CrossMatchResult.pm \
	  lib/Bio/Search/Result/GenericResult.pm blib/lib/Bio/Search/Result/GenericResult.pm \
	  lib/Bio/Search/Result/INFERNALResult.pm blib/lib/Bio/Search/Result/INFERNALResult.pm \
	  lib/Bio/Search/Result/PullResultI.pm blib/lib/Bio/Search/Result/PullResultI.pm \
	  lib/Bio/Search/Result/ResultFactory.pm blib/lib/Bio/Search/Result/ResultFactory.pm \
	  lib/Bio/Search/Result/ResultI.pm blib/lib/Bio/Search/Result/ResultI.pm \
	  lib/Bio/Search/Result/WABAResult.pm blib/lib/Bio/Search/Result/WABAResult.pm \
	  lib/Bio/Search/SearchUtils.pm blib/lib/Bio/Search/SearchUtils.pm 
	$(NOECHO) $(ABSPERLRUN) -MExtUtils::Install -e 'pm_to_blib({@ARGV}, '\''$(INST_LIB)/auto'\'', q[$(PM_FILTER)], '\''$(PERM_DIR)'\'')' -- \
	  lib/Bio/Search/StatisticsI.pm blib/lib/Bio/Search/StatisticsI.pm \
	  lib/Bio/Search/Tiling/MapTileUtils.pm blib/lib/Bio/Search/Tiling/MapTileUtils.pm \
	  lib/Bio/Search/Tiling/MapTiling.pm blib/lib/Bio/Search/Tiling/MapTiling.pm \
	  lib/Bio/Search/Tiling/TilingI.pm blib/lib/Bio/Search/Tiling/TilingI.pm \
	  lib/Bio/SearchIO.pm blib/lib/Bio/SearchIO.pm \
	  lib/Bio/SearchIO/EventHandlerI.pm blib/lib/Bio/SearchIO/EventHandlerI.pm \
	  lib/Bio/SearchIO/FastHitEventBuilder.pm blib/lib/Bio/SearchIO/FastHitEventBuilder.pm \
	  lib/Bio/SearchIO/IteratedSearchResultEventBuilder.pm blib/lib/Bio/SearchIO/IteratedSearchResultEventBuilder.pm \
	  lib/Bio/SearchIO/SearchResultEventBuilder.pm blib/lib/Bio/SearchIO/SearchResultEventBuilder.pm \
	  lib/Bio/SearchIO/SearchWriterI.pm blib/lib/Bio/SearchIO/SearchWriterI.pm \
	  lib/Bio/SearchIO/Writer/GbrowseGFF.pm blib/lib/Bio/SearchIO/Writer/GbrowseGFF.pm \
	  lib/Bio/SearchIO/Writer/HSPTableWriter.pm blib/lib/Bio/SearchIO/Writer/HSPTableWriter.pm \
	  lib/Bio/SearchIO/Writer/HTMLResultWriter.pm blib/lib/Bio/SearchIO/Writer/HTMLResultWriter.pm \
	  lib/Bio/SearchIO/Writer/HitTableWriter.pm blib/lib/Bio/SearchIO/Writer/HitTableWriter.pm \
	  lib/Bio/SearchIO/Writer/ResultTableWriter.pm blib/lib/Bio/SearchIO/Writer/ResultTableWriter.pm \
	  lib/Bio/SearchIO/Writer/TextResultWriter.pm blib/lib/Bio/SearchIO/Writer/TextResultWriter.pm \
	  lib/Bio/SearchIO/axt.pm blib/lib/Bio/SearchIO/axt.pm \
	  lib/Bio/SearchIO/blast.pm blib/lib/Bio/SearchIO/blast.pm \
	  lib/Bio/SearchIO/blast_pull.pm blib/lib/Bio/SearchIO/blast_pull.pm \
	  lib/Bio/SearchIO/blasttable.pm blib/lib/Bio/SearchIO/blasttable.pm \
	  lib/Bio/SearchIO/cross_match.pm blib/lib/Bio/SearchIO/cross_match.pm \
	  lib/Bio/SearchIO/erpin.pm blib/lib/Bio/SearchIO/erpin.pm \
	  lib/Bio/SearchIO/exonerate.pm blib/lib/Bio/SearchIO/exonerate.pm \
	  lib/Bio/SearchIO/fasta.pm blib/lib/Bio/SearchIO/fasta.pm \
	  lib/Bio/SearchIO/gmap_f9.pm blib/lib/Bio/SearchIO/gmap_f9.pm \
	  lib/Bio/SearchIO/infernal.pm blib/lib/Bio/SearchIO/infernal.pm \
	  lib/Bio/SearchIO/megablast.pm blib/lib/Bio/SearchIO/megablast.pm \
	  lib/Bio/SearchIO/psl.pm blib/lib/Bio/SearchIO/psl.pm \
	  lib/Bio/SearchIO/rnamotif.pm blib/lib/Bio/SearchIO/rnamotif.pm \
	  lib/Bio/SearchIO/sim4.pm blib/lib/Bio/SearchIO/sim4.pm \
	  lib/Bio/SearchIO/waba.pm blib/lib/Bio/SearchIO/waba.pm \
	  lib/Bio/SearchIO/wise.pm blib/lib/Bio/SearchIO/wise.pm \
	  lib/Bio/Seq.pm blib/lib/Bio/Seq.pm \
	  lib/Bio/Seq/BaseSeqProcessor.pm blib/lib/Bio/Seq/BaseSeqProcessor.pm \
	  lib/Bio/Seq/EncodedSeq.pm blib/lib/Bio/Seq/EncodedSeq.pm \
	  lib/Bio/Seq/LargeLocatableSeq.pm blib/lib/Bio/Seq/LargeLocatableSeq.pm \
	  lib/Bio/Seq/LargePrimarySeq.pm blib/lib/Bio/Seq/LargePrimarySeq.pm 
	$(NOECHO) $(ABSPERLRUN) -MExtUtils::Install -e 'pm_to_blib({@ARGV}, '\''$(INST_LIB)/auto'\'', q[$(PM_FILTER)], '\''$(PERM_DIR)'\'')' -- \
	  lib/Bio/Seq/LargeSeq.pm blib/lib/Bio/Seq/LargeSeq.pm \
	  lib/Bio/Seq/LargeSeqI.pm blib/lib/Bio/Seq/LargeSeqI.pm \
	  lib/Bio/Seq/Meta.pm blib/lib/Bio/Seq/Meta.pm \
	  lib/Bio/Seq/Meta/Array.pm blib/lib/Bio/Seq/Meta/Array.pm \
	  lib/Bio/Seq/MetaI.pm blib/lib/Bio/Seq/MetaI.pm \
	  lib/Bio/Seq/PrimaryQual.pm blib/lib/Bio/Seq/PrimaryQual.pm \
	  lib/Bio/Seq/PrimedSeq.pm blib/lib/Bio/Seq/PrimedSeq.pm \
	  lib/Bio/Seq/QualI.pm blib/lib/Bio/Seq/QualI.pm \
	  lib/Bio/Seq/Quality.pm blib/lib/Bio/Seq/Quality.pm \
	  lib/Bio/Seq/RichSeq.pm blib/lib/Bio/Seq/RichSeq.pm \
	  lib/Bio/Seq/RichSeqI.pm blib/lib/Bio/Seq/RichSeqI.pm \
	  lib/Bio/Seq/SeqBuilder.pm blib/lib/Bio/Seq/SeqBuilder.pm \
	  lib/Bio/Seq/SeqFactory.pm blib/lib/Bio/Seq/SeqFactory.pm \
	  lib/Bio/Seq/SeqFastaSpeedFactory.pm blib/lib/Bio/Seq/SeqFastaSpeedFactory.pm \
	  lib/Bio/Seq/SequenceTrace.pm blib/lib/Bio/Seq/SequenceTrace.pm \
	  lib/Bio/Seq/SimulatedRead.pm blib/lib/Bio/Seq/SimulatedRead.pm \
	  lib/Bio/Seq/TraceI.pm blib/lib/Bio/Seq/TraceI.pm \
	  lib/Bio/SeqAnalysisParserI.pm blib/lib/Bio/SeqAnalysisParserI.pm \
	  lib/Bio/SeqFeature/Amplicon.pm blib/lib/Bio/SeqFeature/Amplicon.pm \
	  lib/Bio/SeqFeature/AnnotationAdaptor.pm blib/lib/Bio/SeqFeature/AnnotationAdaptor.pm \
	  lib/Bio/SeqFeature/Collection.pm blib/lib/Bio/SeqFeature/Collection.pm \
	  lib/Bio/SeqFeature/CollectionI.pm blib/lib/Bio/SeqFeature/CollectionI.pm \
	  lib/Bio/SeqFeature/Computation.pm blib/lib/Bio/SeqFeature/Computation.pm \
	  lib/Bio/SeqFeature/FeaturePair.pm blib/lib/Bio/SeqFeature/FeaturePair.pm \
	  lib/Bio/SeqFeature/Gene/Exon.pm blib/lib/Bio/SeqFeature/Gene/Exon.pm \
	  lib/Bio/SeqFeature/Gene/ExonI.pm blib/lib/Bio/SeqFeature/Gene/ExonI.pm \
	  lib/Bio/SeqFeature/Gene/GeneStructure.pm blib/lib/Bio/SeqFeature/Gene/GeneStructure.pm \
	  lib/Bio/SeqFeature/Gene/GeneStructureI.pm blib/lib/Bio/SeqFeature/Gene/GeneStructureI.pm \
	  lib/Bio/SeqFeature/Gene/Intron.pm blib/lib/Bio/SeqFeature/Gene/Intron.pm \
	  lib/Bio/SeqFeature/Gene/NC_Feature.pm blib/lib/Bio/SeqFeature/Gene/NC_Feature.pm \
	  lib/Bio/SeqFeature/Gene/Poly_A_site.pm blib/lib/Bio/SeqFeature/Gene/Poly_A_site.pm \
	  lib/Bio/SeqFeature/Gene/Promoter.pm blib/lib/Bio/SeqFeature/Gene/Promoter.pm \
	  lib/Bio/SeqFeature/Gene/Transcript.pm blib/lib/Bio/SeqFeature/Gene/Transcript.pm \
	  lib/Bio/SeqFeature/Gene/TranscriptI.pm blib/lib/Bio/SeqFeature/Gene/TranscriptI.pm \
	  lib/Bio/SeqFeature/Gene/UTR.pm blib/lib/Bio/SeqFeature/Gene/UTR.pm \
	  lib/Bio/SeqFeature/Generic.pm blib/lib/Bio/SeqFeature/Generic.pm \
	  lib/Bio/SeqFeature/Lite.pm blib/lib/Bio/SeqFeature/Lite.pm \
	  lib/Bio/SeqFeature/PositionProxy.pm blib/lib/Bio/SeqFeature/PositionProxy.pm \
	  lib/Bio/SeqFeature/Primer.pm blib/lib/Bio/SeqFeature/Primer.pm 
	$(NOECHO) $(ABSPERLRUN) -MExtUtils::Install -e 'pm_to_blib({@ARGV}, '\''$(INST_LIB)/auto'\'', q[$(PM_FILTER)], '\''$(PERM_DIR)'\'')' -- \
	  lib/Bio/SeqFeature/Similarity.pm blib/lib/Bio/SeqFeature/Similarity.pm \
	  lib/Bio/SeqFeature/SimilarityPair.pm blib/lib/Bio/SeqFeature/SimilarityPair.pm \
	  lib/Bio/SeqFeature/SubSeq.pm blib/lib/Bio/SeqFeature/SubSeq.pm \
	  lib/Bio/SeqFeature/Tools/FeatureNamer.pm blib/lib/Bio/SeqFeature/Tools/FeatureNamer.pm \
	  lib/Bio/SeqFeature/Tools/IDHandler.pm blib/lib/Bio/SeqFeature/Tools/IDHandler.pm \
	  lib/Bio/SeqFeature/Tools/TypeMapper.pm blib/lib/Bio/SeqFeature/Tools/TypeMapper.pm \
	  lib/Bio/SeqFeature/Tools/Unflattener.pm blib/lib/Bio/SeqFeature/Tools/Unflattener.pm \
	  lib/Bio/SeqFeature/TypedSeqFeatureI.pm blib/lib/Bio/SeqFeature/TypedSeqFeatureI.pm \
	  lib/Bio/SeqFeatureI.pm blib/lib/Bio/SeqFeatureI.pm \
	  lib/Bio/SeqI.pm blib/lib/Bio/SeqI.pm \
	  lib/Bio/SeqIO.pm blib/lib/Bio/SeqIO.pm \
	  lib/Bio/SeqIO/FTHelper.pm blib/lib/Bio/SeqIO/FTHelper.pm \
	  lib/Bio/SeqIO/Handler/GenericRichSeqHandler.pm blib/lib/Bio/SeqIO/Handler/GenericRichSeqHandler.pm \
	  lib/Bio/SeqIO/MultiFile.pm blib/lib/Bio/SeqIO/MultiFile.pm \
	  lib/Bio/SeqIO/ace.pm blib/lib/Bio/SeqIO/ace.pm \
	  lib/Bio/SeqIO/asciitree.pm blib/lib/Bio/SeqIO/asciitree.pm \
	  lib/Bio/SeqIO/bsml.pm blib/lib/Bio/SeqIO/bsml.pm \
	  lib/Bio/SeqIO/bsml_sax.pm blib/lib/Bio/SeqIO/bsml_sax.pm \
	  lib/Bio/SeqIO/embl.pm blib/lib/Bio/SeqIO/embl.pm \
	  lib/Bio/SeqIO/embldriver.pm blib/lib/Bio/SeqIO/embldriver.pm \
	  lib/Bio/SeqIO/fasta.pm blib/lib/Bio/SeqIO/fasta.pm \
	  lib/Bio/SeqIO/fastq.pm blib/lib/Bio/SeqIO/fastq.pm \
	  lib/Bio/SeqIO/game.pm blib/lib/Bio/SeqIO/game.pm \
	  lib/Bio/SeqIO/game/featHandler.pm blib/lib/Bio/SeqIO/game/featHandler.pm \
	  lib/Bio/SeqIO/game/gameHandler.pm blib/lib/Bio/SeqIO/game/gameHandler.pm \
	  lib/Bio/SeqIO/game/gameSubs.pm blib/lib/Bio/SeqIO/game/gameSubs.pm \
	  lib/Bio/SeqIO/game/gameWriter.pm blib/lib/Bio/SeqIO/game/gameWriter.pm \
	  lib/Bio/SeqIO/game/seqHandler.pm blib/lib/Bio/SeqIO/game/seqHandler.pm \
	  lib/Bio/SeqIO/gbdriver.pm blib/lib/Bio/SeqIO/gbdriver.pm \
	  lib/Bio/SeqIO/gbxml.pm blib/lib/Bio/SeqIO/gbxml.pm \
	  lib/Bio/SeqIO/gcg.pm blib/lib/Bio/SeqIO/gcg.pm \
	  lib/Bio/SeqIO/genbank.pm blib/lib/Bio/SeqIO/genbank.pm \
	  lib/Bio/SeqIO/kegg.pm blib/lib/Bio/SeqIO/kegg.pm \
	  lib/Bio/SeqIO/largefasta.pm blib/lib/Bio/SeqIO/largefasta.pm \
	  lib/Bio/SeqIO/locuslink.pm blib/lib/Bio/SeqIO/locuslink.pm \
	  lib/Bio/SeqIO/mbsout.pm blib/lib/Bio/SeqIO/mbsout.pm \
	  lib/Bio/SeqIO/metafasta.pm blib/lib/Bio/SeqIO/metafasta.pm \
	  lib/Bio/SeqIO/msout.pm blib/lib/Bio/SeqIO/msout.pm \
	  lib/Bio/SeqIO/phd.pm blib/lib/Bio/SeqIO/phd.pm \
	  lib/Bio/SeqIO/pir.pm blib/lib/Bio/SeqIO/pir.pm \
	  lib/Bio/SeqIO/qual.pm blib/lib/Bio/SeqIO/qual.pm \
	  lib/Bio/SeqIO/raw.pm blib/lib/Bio/SeqIO/raw.pm 
	$(NOECHO) $(ABSPERLRUN) -MExtUtils::Install -e 'pm_to_blib({@ARGV}, '\''$(INST_LIB)/auto'\'', q[$(PM_FILTER)], '\''$(PERM_DIR)'\'')' -- \
	  lib/Bio/SeqIO/scf.pm blib/lib/Bio/SeqIO/scf.pm \
	  lib/Bio/SeqIO/seqxml.pm blib/lib/Bio/SeqIO/seqxml.pm \
	  lib/Bio/SeqIO/swiss.pm blib/lib/Bio/SeqIO/swiss.pm \
	  lib/Bio/SeqIO/swissdriver.pm blib/lib/Bio/SeqIO/swissdriver.pm \
	  lib/Bio/SeqIO/tab.pm blib/lib/Bio/SeqIO/tab.pm \
	  lib/Bio/SeqIO/table.pm blib/lib/Bio/SeqIO/table.pm \
	  lib/Bio/SeqIO/tigr.pm blib/lib/Bio/SeqIO/tigr.pm \
	  lib/Bio/SeqIO/tigrxml.pm blib/lib/Bio/SeqIO/tigrxml.pm \
	  lib/Bio/SeqIO/tinyseq.pm blib/lib/Bio/SeqIO/tinyseq.pm \
	  lib/Bio/SeqIO/tinyseq/tinyseqHandler.pm blib/lib/Bio/SeqIO/tinyseq/tinyseqHandler.pm \
	  lib/Bio/SeqUtils.pm blib/lib/Bio/SeqUtils.pm \
	  lib/Bio/SimpleAlign.pm blib/lib/Bio/SimpleAlign.pm \
	  lib/Bio/SimpleAnalysisI.pm blib/lib/Bio/SimpleAnalysisI.pm \
	  lib/Bio/Species.pm blib/lib/Bio/Species.pm \
	  lib/Bio/Taxon.pm blib/lib/Bio/Taxon.pm \
	  lib/Bio/Tools/Alignment/Consed.pm blib/lib/Bio/Tools/Alignment/Consed.pm \
	  lib/Bio/Tools/Alignment/Trim.pm blib/lib/Bio/Tools/Alignment/Trim.pm \
	  lib/Bio/Tools/AmpliconSearch.pm blib/lib/Bio/Tools/AmpliconSearch.pm \
	  lib/Bio/Tools/Analysis/SimpleAnalysisBase.pm blib/lib/Bio/Tools/Analysis/SimpleAnalysisBase.pm \
	  lib/Bio/Tools/AnalysisResult.pm blib/lib/Bio/Tools/AnalysisResult.pm \
	  lib/Bio/Tools/Blat.pm blib/lib/Bio/Tools/Blat.pm \
	  lib/Bio/Tools/CodonTable.pm blib/lib/Bio/Tools/CodonTable.pm \
	  lib/Bio/Tools/Coil.pm blib/lib/Bio/Tools/Coil.pm \
	  lib/Bio/Tools/ECnumber.pm blib/lib/Bio/Tools/ECnumber.pm \
	  lib/Bio/Tools/EMBOSS/Palindrome.pm blib/lib/Bio/Tools/EMBOSS/Palindrome.pm \
	  lib/Bio/Tools/EPCR.pm blib/lib/Bio/Tools/EPCR.pm \
	  lib/Bio/Tools/ESTScan.pm blib/lib/Bio/Tools/ESTScan.pm \
	  lib/Bio/Tools/Eponine.pm blib/lib/Bio/Tools/Eponine.pm \
	  lib/Bio/Tools/Est2Genome.pm blib/lib/Bio/Tools/Est2Genome.pm \
	  lib/Bio/Tools/Fgenesh.pm blib/lib/Bio/Tools/Fgenesh.pm \
	  lib/Bio/Tools/FootPrinter.pm blib/lib/Bio/Tools/FootPrinter.pm \
	  lib/Bio/Tools/GFF.pm blib/lib/Bio/Tools/GFF.pm \
	  lib/Bio/Tools/Geneid.pm blib/lib/Bio/Tools/Geneid.pm \
	  lib/Bio/Tools/Genemark.pm blib/lib/Bio/Tools/Genemark.pm \
	  lib/Bio/Tools/Genewise.pm blib/lib/Bio/Tools/Genewise.pm \
	  lib/Bio/Tools/Genomewise.pm blib/lib/Bio/Tools/Genomewise.pm \
	  lib/Bio/Tools/Genscan.pm blib/lib/Bio/Tools/Genscan.pm \
	  lib/Bio/Tools/Glimmer.pm blib/lib/Bio/Tools/Glimmer.pm \
	  lib/Bio/Tools/Grail.pm blib/lib/Bio/Tools/Grail.pm \
	  lib/Bio/Tools/GuessSeqFormat.pm blib/lib/Bio/Tools/GuessSeqFormat.pm \
	  lib/Bio/Tools/IUPAC.pm blib/lib/Bio/Tools/IUPAC.pm \
	  lib/Bio/Tools/Lucy.pm blib/lib/Bio/Tools/Lucy.pm \
	  lib/Bio/Tools/MZEF.pm blib/lib/Bio/Tools/MZEF.pm \
	  lib/Bio/Tools/Match.pm blib/lib/Bio/Tools/Match.pm \
	  lib/Bio/Tools/OddCodes.pm blib/lib/Bio/Tools/OddCodes.pm 
	$(NOECHO) $(ABSPERLRUN) -MExtUtils::Install -e 'pm_to_blib({@ARGV}, '\''$(INST_LIB)/auto'\'', q[$(PM_FILTER)], '\''$(PERM_DIR)'\'')' -- \
	  lib/Bio/Tools/Phylo/Gerp.pm blib/lib/Bio/Tools/Phylo/Gerp.pm \
	  lib/Bio/Tools/Phylo/Molphy.pm blib/lib/Bio/Tools/Phylo/Molphy.pm \
	  lib/Bio/Tools/Phylo/Molphy/Result.pm blib/lib/Bio/Tools/Phylo/Molphy/Result.pm \
	  lib/Bio/Tools/Phylo/Phylip/ProtDist.pm blib/lib/Bio/Tools/Phylo/Phylip/ProtDist.pm \
	  lib/Bio/Tools/Prediction/Exon.pm blib/lib/Bio/Tools/Prediction/Exon.pm \
	  lib/Bio/Tools/Prediction/Gene.pm blib/lib/Bio/Tools/Prediction/Gene.pm \
	  lib/Bio/Tools/Primer/Assessor/Base.pm blib/lib/Bio/Tools/Primer/Assessor/Base.pm \
	  lib/Bio/Tools/Primer/AssessorI.pm blib/lib/Bio/Tools/Primer/AssessorI.pm \
	  lib/Bio/Tools/Primer/Feature.pm blib/lib/Bio/Tools/Primer/Feature.pm \
	  lib/Bio/Tools/Primer/Pair.pm blib/lib/Bio/Tools/Primer/Pair.pm \
	  lib/Bio/Tools/Primer3.pm blib/lib/Bio/Tools/Primer3.pm \
	  lib/Bio/Tools/Prints.pm blib/lib/Bio/Tools/Prints.pm \
	  lib/Bio/Tools/Profile.pm blib/lib/Bio/Tools/Profile.pm \
	  lib/Bio/Tools/Promoterwise.pm blib/lib/Bio/Tools/Promoterwise.pm \
	  lib/Bio/Tools/PrositeScan.pm blib/lib/Bio/Tools/PrositeScan.pm \
	  lib/Bio/Tools/Pseudowise.pm blib/lib/Bio/Tools/Pseudowise.pm \
	  lib/Bio/Tools/QRNA.pm blib/lib/Bio/Tools/QRNA.pm \
	  lib/Bio/Tools/RandomDistFunctions.pm blib/lib/Bio/Tools/RandomDistFunctions.pm \
	  lib/Bio/Tools/RepeatMasker.pm blib/lib/Bio/Tools/RepeatMasker.pm \
	  lib/Bio/Tools/Run/Analysis.pm blib/lib/Bio/Tools/Run/Analysis.pm \
	  lib/Bio/Tools/Run/AnalysisFactory.pm blib/lib/Bio/Tools/Run/AnalysisFactory.pm \
	  lib/Bio/Tools/Run/GenericParameters.pm blib/lib/Bio/Tools/Run/GenericParameters.pm \
	  lib/Bio/Tools/Run/ParametersI.pm blib/lib/Bio/Tools/Run/ParametersI.pm \
	  lib/Bio/Tools/Run/Phylo/PhyloBase.pm blib/lib/Bio/Tools/Run/Phylo/PhyloBase.pm \
	  lib/Bio/Tools/Run/WrapperBase.pm blib/lib/Bio/Tools/Run/WrapperBase.pm \
	  lib/Bio/Tools/Run/WrapperBase/CommandExts.pm blib/lib/Bio/Tools/Run/WrapperBase/CommandExts.pm \
	  lib/Bio/Tools/Seg.pm blib/lib/Bio/Tools/Seg.pm \
	  lib/Bio/Tools/SeqPattern.pm blib/lib/Bio/Tools/SeqPattern.pm \
	  lib/Bio/Tools/SeqPattern/Backtranslate.pm blib/lib/Bio/Tools/SeqPattern/Backtranslate.pm \
	  lib/Bio/Tools/SeqStats.pm blib/lib/Bio/Tools/SeqStats.pm \
	  lib/Bio/Tools/SeqWords.pm blib/lib/Bio/Tools/SeqWords.pm \
	  lib/Bio/Tools/Sigcleave.pm blib/lib/Bio/Tools/Sigcleave.pm \
	  lib/Bio/Tools/Signalp.pm blib/lib/Bio/Tools/Signalp.pm \
	  lib/Bio/Tools/Signalp/ExtendedSignalp.pm blib/lib/Bio/Tools/Signalp/ExtendedSignalp.pm \
	  lib/Bio/Tools/Sim4/Exon.pm blib/lib/Bio/Tools/Sim4/Exon.pm \
	  lib/Bio/Tools/Sim4/Results.pm blib/lib/Bio/Tools/Sim4/Results.pm \
	  lib/Bio/Tools/Spidey/Exon.pm blib/lib/Bio/Tools/Spidey/Exon.pm \
	  lib/Bio/Tools/Spidey/Results.pm blib/lib/Bio/Tools/Spidey/Results.pm 
	$(NOECHO) $(ABSPERLRUN) -MExtUtils::Install -e 'pm_to_blib({@ARGV}, '\''$(INST_LIB)/auto'\'', q[$(PM_FILTER)], '\''$(PERM_DIR)'\'')' -- \
	  lib/Bio/Tools/TandemRepeatsFinder.pm blib/lib/Bio/Tools/TandemRepeatsFinder.pm \
	  lib/Bio/Tools/TargetP.pm blib/lib/Bio/Tools/TargetP.pm \
	  lib/Bio/Tools/Tmhmm.pm blib/lib/Bio/Tools/Tmhmm.pm \
	  lib/Bio/Tools/ipcress.pm blib/lib/Bio/Tools/ipcress.pm \
	  lib/Bio/Tools/isPcr.pm blib/lib/Bio/Tools/isPcr.pm \
	  lib/Bio/Tools/pICalculator.pm blib/lib/Bio/Tools/pICalculator.pm \
	  lib/Bio/Tools/tRNAscanSE.pm blib/lib/Bio/Tools/tRNAscanSE.pm \
	  lib/Bio/Tree/AnnotatableNode.pm blib/lib/Bio/Tree/AnnotatableNode.pm \
	  lib/Bio/Tree/Compatible.pm blib/lib/Bio/Tree/Compatible.pm \
	  lib/Bio/Tree/DistanceFactory.pm blib/lib/Bio/Tree/DistanceFactory.pm \
	  lib/Bio/Tree/Node.pm blib/lib/Bio/Tree/Node.pm \
	  lib/Bio/Tree/NodeI.pm blib/lib/Bio/Tree/NodeI.pm \
	  lib/Bio/Tree/NodeNHX.pm blib/lib/Bio/Tree/NodeNHX.pm \
	  lib/Bio/Tree/RandomFactory.pm blib/lib/Bio/Tree/RandomFactory.pm \
	  lib/Bio/Tree/Statistics.pm blib/lib/Bio/Tree/Statistics.pm \
	  lib/Bio/Tree/Tree.pm blib/lib/Bio/Tree/Tree.pm \
	  lib/Bio/Tree/TreeFunctionsI.pm blib/lib/Bio/Tree/TreeFunctionsI.pm \
	  lib/Bio/Tree/TreeI.pm blib/lib/Bio/Tree/TreeI.pm \
	  lib/Bio/TreeIO.pm blib/lib/Bio/TreeIO.pm \
	  lib/Bio/TreeIO/NewickParser.pm blib/lib/Bio/TreeIO/NewickParser.pm \
	  lib/Bio/TreeIO/TreeEventBuilder.pm blib/lib/Bio/TreeIO/TreeEventBuilder.pm \
	  lib/Bio/TreeIO/cluster.pm blib/lib/Bio/TreeIO/cluster.pm \
	  lib/Bio/TreeIO/lintree.pm blib/lib/Bio/TreeIO/lintree.pm \
	  lib/Bio/TreeIO/newick.pm blib/lib/Bio/TreeIO/newick.pm \
	  lib/Bio/TreeIO/nexus.pm blib/lib/Bio/TreeIO/nexus.pm \
	  lib/Bio/TreeIO/nhx.pm blib/lib/Bio/TreeIO/nhx.pm \
	  lib/Bio/TreeIO/pag.pm blib/lib/Bio/TreeIO/pag.pm \
	  lib/Bio/TreeIO/phyloxml.pm blib/lib/Bio/TreeIO/phyloxml.pm \
	  lib/Bio/TreeIO/tabtree.pm blib/lib/Bio/TreeIO/tabtree.pm \
	  lib/Bio/UpdateableSeqI.pm blib/lib/Bio/UpdateableSeqI.pm \
	  lib/Bio/WebAgent.pm blib/lib/Bio/WebAgent.pm \
	  lib/BioPerl.pm blib/lib/BioPerl.pm 
	$(NOECHO) $(TOUCH) pm_to_blib


# --- MakeMaker selfdocument section:


# --- MakeMaker postamble section:


# End.
